\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8635 & 40.47 & 40.61 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8593.1 & 1 & 1 & 1 & 1 & 88207 & 175000 & 175000 & 394.81 & 393.26 & 392.82 & 391.27 & 10088 & 10080 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 742.98 & 69.65 & 8614 & 8614 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8593.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8635 & 40.75 & 40.61 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8593.1 & 1 & 1 & 1 & 1 & 88577 & 175000 & 175000 & 392.49 & 393.26 & 389.84 & 391.27 & 10071 & 10080 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.932 & 1.000 & 0.00 & 668.86 & 70.41 & 8522 & 8522 & 8522 & 8522 & 8522 & 8522 & 8431.0 & 8410.5 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8635 & 40.53 & 40.61 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8593.1 & 1 & 1 & 1 & 1 & 95180 & 175000 & 175000 & 353.69 & 393.26 & 339.65 & 391.27 & 9703 & 10080 & 98.14 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 98.108 & 1.000 & 2820.99 & 1277.96 & 71.00 & 8451 & 8451 & 8451 & 8451 & 8451 & 8451 & 8291.1 & 8270.9 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8635 & 40.81 & 40.61 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8641.0 & 1 & 1 & 1 & 1 & 93950 & 175000 & 175000 & 361.52 & 393.26 & 350.91 & 391.27 & 9729 & 10080 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.781 & 1.003 & 0.00 & 1289.11 & 492.21 & 8533 & 8533 & 8533 & 8533 & 8533 & 8533 & 8429.3 & 8432.2 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
