\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
Total length ($\geq$ 1000 bp) & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
Total length ($\geq$ 5000 bp) & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
Total length & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
Reference length & 8651 & 8651 & 8651 & 8651 \\ \hline
GC (\%) & 40.70 & 40.69 & 40.68 & 41.05 \\ \hline
Reference GC (\%) & 40.70 & 40.70 & 40.70 & 40.70 \\ \hline
N50 & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
NG50 & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
N90 & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
NG90 & 8624 & 8624 & 8587 & {\bf 8638} \\ \hline
auN & 8624.0 & 8624.0 & 8587.0 & {\bf 8638.0} \\ \hline
auNG & 8597.1 & 8597.1 & 8523.5 & {\bf 8625.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 81748 & 81757 & 81903 & 89381 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 426.05} & 425.98 & 425.0 & 374.14 \\ \hline
Reference mapped (\%) & 424.98 & 424.98 & 424.98 & 424.98 \\ \hline
Properly paired (\%) & {\bf 423.95} & 423.87 & 422.67 & 357.75 \\ \hline
Reference properly paired (\%) & 422.89 & 422.89 & 422.89 & 422.89 \\ \hline
Avg. coverage depth & 10082 & 10082 & {\bf 10117} & 9599 \\ \hline
Reference avg. coverage depth & 10074 & 10074 & 10074 & 10074 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 99.29 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & {\bf 100.000} & 99.641 & 99.849 \\ \hline
Duplication ratio & 1.000 & 1.000 & {\bf 0.999} & 1.002 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 11.60 & 116.46 & 1147.83 \\ \hline
\# indels per 100 kbp & {\bf 81.17} & {\bf 81.17} & 93.16 & 371.01 \\ \hline
Largest alignment & 8624 & 8624 & 8587 & {\bf 8625} \\ \hline
Total aligned length & 8624 & 8624 & 8587 & {\bf 8625} \\ \hline
NA50 & 8624 & 8624 & 8587 & {\bf 8625} \\ \hline
NGA50 & 8624 & 8624 & 8587 & {\bf 8625} \\ \hline
NA90 & 8624 & 8624 & 8587 & {\bf 8625} \\ \hline
NGA90 & 8624 & 8624 & 8587 & {\bf 8625} \\ \hline
auNA & {\bf 8624.0} & {\bf 8624.0} & 8587.0 & 8612.0 \\ \hline
auNGA & 8597.1 & 8597.1 & 8523.5 & {\bf 8599.1} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
