\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
Total length & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
Reference length & 8673 & 8673 & 8673 & 8673 \\ \hline
GC (\%) & 40.57 & 40.54 & 40.90 & 40.77 \\ \hline
Reference GC (\%) & 40.54 & 40.54 & 40.54 & 40.54 \\ \hline
N50 & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
NG50 & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
N75 & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
NG75 & {\bf 8632} & 8631 & 8076 & 8619 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 99.61 & {\bf 99.62} & 47.44 & 90.25 \\ \hline
Reference mapped (\%) & 99.62 & 99.62 & 99.62 & 99.62 \\ \hline
Properly paired (\%) & 99.23 & {\bf 99.24} & 21.93 & 82.65 \\ \hline
Reference properly paired (\%) & 99.24 & 99.24 & 99.24 & 99.24 \\ \hline
Avg. coverage depth & {\bf 506} & {\bf 506} & 247 & 452 \\ \hline
Reference avg. coverage depth & 505 & 505 & 505 & 505 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 59.92 & 99.23 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 3 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 7 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & {\bf 100.000} & 55.565 & 87.862 \\ \hline
Duplication ratio & {\bf 1.001} & {\bf 1.001} & 1.685 & 1.137 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 40960.87 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 440.53} & 811.50 & 2169.83 & 3668.03 \\ \hline
\# indels per 100 kbp & 139.11 & {\bf 127.52} & 166.91 & 290.28 \\ \hline
Largest alignment & {\bf 8632} & 8631 & 4799 & 7580 \\ \hline
Total aligned length & {\bf 8632} & 8631 & 4799 & 7580 \\ \hline
NA50 & {\bf 8632} & 8631 & 4799 & 7580 \\ \hline
NGA50 & {\bf 8632} & 8631 & 4799 & 7580 \\ \hline
NA75 & {\bf 8632} & 8631 & - & 7580 \\ \hline
NGA75 & {\bf 8632} & 8631 & - & 7580 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
