\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
Total length ($\geq$ 1000 bp) & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
Total length ($\geq$ 5000 bp) & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
Total length & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
Reference length & 8673 & 8673 & 8673 & 8673 \\ \hline
GC (\%) & 40.34 & 40.54 & 39.88 & 40.73 \\ \hline
Reference GC (\%) & 40.32 & 40.32 & 40.32 & 40.32 \\ \hline
N50 & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
NG50 & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
N75 & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
NG75 & 8634 & 8633 & 8476 & {\bf 8652} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 350000 & 350000 & 350000 & 350000 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 99.6} & 99.53 & 91.48 & 93.5 \\ \hline
Reference mapped (\%) & 99.59 & 99.59 & 99.59 & 99.59 \\ \hline
Properly paired (\%) & {\bf 99.19} & 99.06 & 82.97 & 87.5 \\ \hline
Reference properly paired (\%) & 99.19 & 99.19 & 99.19 & 99.19 \\ \hline
Avg. coverage depth & {\bf 10068} & 10062 & 9351 & 9342 \\ \hline
Reference avg. coverage depth & 10058 & 10058 & 10058 & 10058 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 95.88 & 99.53 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.988 & 94.044 & 95.180 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 1.044 & 1.053 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 4907.98 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 382.43 & 1885.17 & 1132.21 \\ \hline
\# indels per 100 kbp & {\bf 150.64} & 173.83 & 160.18 & 426.10 \\ \hline
Largest alignment & {\bf 8634} & 8633 & 8117 & 8220 \\ \hline
Total aligned length & {\bf 8634} & 8633 & 8117 & 8220 \\ \hline
NA50 & {\bf 8634} & 8633 & 8117 & 8220 \\ \hline
NGA50 & {\bf 8634} & 8633 & 8117 & 8220 \\ \hline
NA75 & {\bf 8634} & 8633 & 8117 & 8220 \\ \hline
NGA75 & {\bf 8634} & 8633 & 8117 & 8220 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
