\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
Total length & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
Reference length & 8643 & 8643 & 8643 & 8643 \\ \hline
GC (\%) & - & 39.94 & 39.79 & 40.40 \\ \hline
Reference GC (\%) & 39.91 & 39.91 & 39.91 & 39.91 \\ \hline
N50 & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
NG50 & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
N90 & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
NG90 & {\bf 8624} & 8598 & 8601 & 8613 \\ \hline
auN & {\bf 8624.0} & 8598.0 & 8601.0 & 8613.0 \\ \hline
auNG & {\bf 8605.0} & 8553.2 & 8559.2 & 8583.1 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 350000 & 91133 & 91120 & 114439 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & 0.0 & 378.97 & {\bf 379.11} & 266.54 \\ \hline
Reference mapped (\%) & 377.82 & 377.82 & 377.82 & 377.82 \\ \hline
Properly paired (\%) & 0.0 & 373.88 & {\bf 374.11} & 232.54 \\ \hline
Reference properly paired (\%) & 372.83 & 372.83 & 372.83 & 372.83 \\ \hline
Avg. coverage depth & 0 & 10016 & {\bf 10017} & 8720 \\ \hline
Reference avg. coverage depth & 10000 & 10000 & 10000 & 10000 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 100.0 & 100.0 & 99.07 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 0 & 0 \\ \hline
\# structural variations & - & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 1 + 0 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & 8624 & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & - & 100.000 & 100.000 & 98.884 \\ \hline
Duplication ratio & - & 1.000 & 1.000 & 1.002 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & {\bf 0.00} & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 116.31 & 360.42 & 2348.24 \\ \hline
\# indels per 100 kbp & - & 116.31 & 104.64 & 727.96 \\ \hline
Largest alignment & - & 8598 & 8601 & 8517 \\ \hline
Total aligned length & - & 8598 & 8601 & 8517 \\ \hline
NA50 & - & 8598 & 8601 & 8517 \\ \hline
NGA50 & - & 8598 & 8601 & 8517 \\ \hline
NA90 & - & 8598 & 8601 & 8517 \\ \hline
NGA90 & - & 8598 & 8601 & 8517 \\ \hline
auNA & - & 8598.0 & 8601.0 & 8422.1 \\ \hline
auNGA & - & 8553.2 & 8559.2 & 8392.8 \\ \hline
LA50 & - & 1 & 1 & 1 \\ \hline
LGA50 & - & 1 & 1 & 1 \\ \hline
LA90 & - & 1 & 1 & 1 \\ \hline
LGA90 & - & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
