\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8643 & - & 39.91 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8605.0 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 0.0 & 377.82 & 0.0 & 372.83 & 0 & 10000 & 0.0 & 100.0 & - & - & - & - & - & - & - & - & 1 + 0 part & 8624 & - & - & 100000.00 & - & - & - & - & - & - & - & - & - & - & - & - & - & - \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8598 & 8598 & 8598 & 0 & 0 & 0 & 1 & 8598 & 8598 & 8643 & 39.94 & 39.91 & 8598 & 8598 & 8598 & 8598 & 8598.0 & 8553.2 & 1 & 1 & 1 & 1 & 91133 & 175000 & 175000 & 378.97 & 377.82 & 373.88 & 372.83 & 10016 & 10000 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 116.31 & 116.31 & 8598 & 8598 & 8598 & 8598 & 8598 & 8598 & 8598.0 & 8553.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8601 & 8601 & 8601 & 0 & 0 & 0 & 1 & 8601 & 8601 & 8643 & 39.79 & 39.91 & 8601 & 8601 & 8601 & 8601 & 8601.0 & 8559.2 & 1 & 1 & 1 & 1 & 91120 & 175000 & 175000 & 379.11 & 377.82 & 374.11 & 372.83 & 10017 & 10000 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 360.42 & 104.64 & 8601 & 8601 & 8601 & 8601 & 8601 & 8601 & 8601.0 & 8559.2 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8613 & 8613 & 8613 & 0 & 0 & 0 & 1 & 8613 & 8613 & 8643 & 40.40 & 39.91 & 8613 & 8613 & 8613 & 8613 & 8613.0 & 8583.1 & 1 & 1 & 1 & 1 & 114439 & 175000 & 175000 & 266.54 & 377.82 & 232.54 & 372.83 & 8720 & 10000 & 99.07 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.884 & 1.002 & 0.00 & 2348.24 & 727.96 & 8517 & 8517 & 8517 & 8517 & 8517 & 8517 & 8422.1 & 8392.8 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
