\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8716 & 40.71 & 40.71 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8656.1 & 1 & 1 & 1 & 1 & 16486 & 9500 & 9500 & 108.49 & 108.49 & 103.47 & 103.47 & 513 & 513 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 103.62 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8656.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8716 & 41.39 & 40.71 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8560.7 & 1 & 1 & 1 & 1 & 17091 & 9500 & 9500 & 58.84 & 108.49 & 42.96 & 103.47 & 283 & 513 & 63.08 & 100.0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 & 0 & 0 + 1 part & 3033 & 64.529 & 1.000 & 0.00 & 0.00 & 53.52 & 5605 & 5605 & 5605 & 5605 & - & - & 3637.0 & 3604.4 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7168 & 7168 & 7168 & 0 & 0 & 0 & 1 & 7168 & 7168 & 8716 & 41.10 & 40.71 & 7168 & 7168 & 7168 & - & 7168.0 & 5894.9 & 1 & 1 & 1 & - & 17237 & 9500 & 9500 & 58.64 & 108.49 & 46.32 & 103.47 & 347 & 513 & 71.64 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 60.707 & 1.000 & 26436.94 & 0.00 & 37.93 & 5273 & 5273 & 5273 & 5273 & - & - & 3879.0 & 3190.1 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8716 & 41.29 & 40.71 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8533.0 & 1 & 1 & 1 & 1 & 18087 & 9500 & 9500 & 20.46 & 108.49 & 9.09 & 103.47 & 101 & 513 & 54.82 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7682 & 10.845 & 1.000 & 0.00 & 2016.99 & 530.79 & 733 & 942 & - & - & - & - & 67.4 & 66.7 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
