\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8698 & 40.74 & 40.74 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8674.0 & 1 & 1 & 1 & 1 & 16645 & 9500 & 9500 & 107.86 & 107.87 & 103.03 & 103.03 & 515 & 515 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 46.05 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8674.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8698 & 41.35 & 40.74 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8578.4 & 1 & 1 & 1 & 1 & 17033 & 9500 & 9500 & 59.17 & 107.87 & 42.76 & 103.03 & 284 & 515 & 63.09 & 100.0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 & 0 & 0 + 1 part & 2866 & 66.452 & 1.000 & 0.00 & 0.00 & 69.30 & 5772 & 5772 & 5772 & 5772 & - & - & 3856.9 & 3830.3 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7168 & 7168 & 7168 & 0 & 0 & 0 & 1 & 7168 & 7168 & 8698 & 41.04 & 40.74 & 7168 & 7168 & 7168 & - & 7168.0 & 5907.1 & 1 & 1 & 1 & - & 17162 & 9500 & 9500 & 59.78 & 107.87 & 47.16 & 103.03 & 352 & 515 & 71.81 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 60.880 & 1.000 & 26227.68 & 0.00 & 37.82 & 5288 & 5288 & 5288 & 5288 & - & - & 3901.1 & 3214.9 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8698 & 41.34 & 40.74 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8550.6 & 1 & 1 & 1 & 1 & 18016 & 9500 & 9500 & 21.15 & 107.87 & 9.39 & 103.03 & 104 & 515 & 54.59 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7710 & 10.523 & 1.000 & 0.00 & 1750.55 & 547.05 & 705 & 914 & - & - & - & - & 62.7 & 62.2 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
