\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8720} & 8639 & 2997 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8720} & 8639 & 2997 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8720} & 8639 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8720} & 8639 & 2997 & 8625 \\ \hline
Total length & {\bf 8720} & 8639 & 2997 & 8625 \\ \hline
Reference length & 8750 & 8750 & 8750 & 8750 \\ \hline
GC (\%) & 40.77 & 41.36 & 43.68 & 41.31 \\ \hline
Reference GC (\%) & 40.79 & 40.79 & 40.79 & 40.79 \\ \hline
N50 & {\bf 8720} & 8639 & 2997 & 8625 \\ \hline
NG50 & {\bf 8720} & 8639 & - & 8625 \\ \hline
N75 & {\bf 8720} & 8639 & 2997 & 8625 \\ \hline
NG75 & {\bf 8720} & 8639 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 19000 & 19000 & 19000 & 19000 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 94.21} & 53.21 & 14.39 & 19.59 \\ \hline
Reference mapped (\%) & 94.21 & 94.21 & 94.21 & 94.21 \\ \hline
Properly paired (\%) & {\bf 90.16} & 38.66 & 6.23 & 8.88 \\ \hline
Reference properly paired (\%) & 90.16 & 90.16 & 90.16 & 90.16 \\ \hline
Avg. coverage depth & {\bf 511} & 284 & 220 & 102 \\ \hline
Reference avg. coverage depth & 511 & 511 & 511 & 511 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 63.27 & 49.38 & 56.64 \\ \hline
Reference coverage $\geq$ 1x (\%) & 99.99 & 99.99 & 99.99 & 99.99 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 7 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 2857 & {\bf 0} & 7686 \\ \hline
Genome fraction (\%) & {\bf 99.989} & 66.311 & 16.976 & 10.771 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 2.025 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 50650.65 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 11.47} & 51.89 & 202.70 & 2449.41 \\ \hline
\# indels per 100 kbp & 172.08 & 17.30 & {\bf 0.00} & 532.48 \\ \hline
Largest alignment & {\bf 8719} & 5782 & 1480 & 733 \\ \hline
Total aligned length & {\bf 8719} & 5782 & 1480 & 939 \\ \hline
NA50 & {\bf 8719} & 5782 & - & - \\ \hline
NGA50 & {\bf 8719} & 5782 & - & - \\ \hline
NA75 & {\bf 8719} & - & - & - \\ \hline
NGA75 & {\bf 8719} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
