\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
Total length ($\geq$ 1000 bp) & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
Total length ($\geq$ 5000 bp) & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
Total length & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
Reference length & 8636 & 8636 & 8636 & 8636 \\ \hline
GC (\%) & 40.62 & 40.60 & 40.57 & 40.62 \\ \hline
Reference GC (\%) & 40.56 & 40.56 & 40.56 & 40.56 \\ \hline
N50 & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
NG50 & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
N75 & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
NG75 & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 350000 & 350000 & 350000 & 350000 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & 99.55 & 99.55 & 99.55 & 99.55 \\ \hline
Reference mapped (\%) & 99.55 & 99.55 & 99.55 & 99.55 \\ \hline
Properly paired (\%) & 99.1 & 99.1 & 99.1 & 99.1 \\ \hline
Reference properly paired (\%) & 99.1 & 99.1 & 99.1 & 99.1 \\ \hline
Avg. coverage depth & 10092 & {\bf 10093} & 10090 & 10089 \\ \hline
Reference avg. coverage depth & 10083 & 10083 & 10083 & 10083 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 100.000 & 100.000 & 100.000 & 100.000 \\ \hline
Duplication ratio & 1.001 & {\bf 1.000} & {\bf 1.000} & 1.002 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 11.62 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 34.86 & 302.15 & {\bf 0.00} \\ \hline
\# indels per 100 kbp & 58.11 & 69.73 & 69.73 & {\bf 34.86} \\ \hline
Largest alignment & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
Total aligned length & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
NA50 & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
NGA50 & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
NA75 & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
NGA75 & 8615 & 8606 & 8609 & {\bf 8621} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
