\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8722} & 8639 & 4086 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8722} & 8639 & 4086 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8722} & 8639 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8722} & 8639 & 4086 & 8625 \\ \hline
Total length & {\bf 8722} & 8639 & 4086 & 8625 \\ \hline
Reference length & 8737 & 8737 & 8737 & 8737 \\ \hline
GC (\%) & 40.42 & 41.19 & 41.08 & 41.30 \\ \hline
Reference GC (\%) & 40.42 & 40.42 & 40.42 & 40.42 \\ \hline
N50 & {\bf 8722} & 8639 & 4086 & 8625 \\ \hline
NG50 & {\bf 8722} & 8639 & - & 8625 \\ \hline
N75 & {\bf 8722} & 8639 & 4086 & 8625 \\ \hline
NG75 & {\bf 8722} & 8639 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 380000 & 380000 & 380000 & 380000 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 94.34} & 54.21 & 25.53 & 17.84 \\ \hline
Reference mapped (\%) & 94.34 & 94.34 & 94.34 & 94.34 \\ \hline
Properly paired (\%) & {\bf 90.32} & 39.62 & 16.04 & 7.71 \\ \hline
Reference properly paired (\%) & 90.32 & 90.32 & 90.32 & 90.32 \\ \hline
Avg. coverage depth & {\bf 10243} & 5782 & 5773 & 1866 \\ \hline
Reference avg. coverage depth & 10236 & 10236 & 10236 & 10236 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 66.26 & 63.93 & 57.16 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 7 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 2807 & {\bf 0} & 7683 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 66.858 & 29.576 & 10.803 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 1.584 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 37738.62 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 11.47} & 34.31 & 620.40 & 2441.61 \\ \hline
\# indels per 100 kbp & 34.40 & {\bf 34.31} & 38.77 & 424.63 \\ \hline
Largest alignment & {\bf 8722} & 5832 & 2579 & 733 \\ \hline
Total aligned length & {\bf 8722} & 5832 & 2579 & 942 \\ \hline
NA50 & {\bf 8722} & 5832 & 2579 & - \\ \hline
NGA50 & {\bf 8722} & 5832 & - & - \\ \hline
NA75 & {\bf 8722} & - & - & - \\ \hline
NGA75 & {\bf 8722} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
