\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8618 & 8618 & 8618 & 0 & 0 & 0 & 1 & 8618 & 8618 & 8645 & 40.65 & 40.65 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8591.1 & 1 & 1 & 1 & 1 & 90755 & 175000 & 175000 & 383.73 & 382.6 & 381.81 & 380.68 & 10076 & 10072 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 0.00 & 11.60 & 92.83 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8591.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8621 & 8621 & 8621 & 0 & 0 & 0 & 1 & 8621 & 8621 & 8645 & 40.64 & 40.65 & 8621 & 8621 & 8621 & 8621 & 8621.0 & 8597.1 & 1 & 1 & 1 & 1 & 91113 & 175000 & 175000 & 381.65 & 382.6 & 379.15 & 380.68 & 10063 & 10072 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.000 & 0.00 & 232.34 & 81.32 & 8608 & 8608 & 8608 & 8608 & 8608 & 8608 & 8595.0 & 8571.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8618 & 8618 & 8618 & 0 & 0 & 0 & 1 & 8618 & 8618 & 8645 & 40.34 & 40.65 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8591.1 & 1 & 1 & 1 & 1 & 97813 & 175000 & 175000 & 339.03 & 382.6 & 320.23 & 380.68 & 9558 & 10072 & 96.86 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 96.695 & 1.000 & 3875.61 & 635.34 & 59.94 & 8342 & 8342 & 8342 & 8342 & 8342 & 8342 & 8074.8 & 8049.6 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8644 & 8644 & 8644 & 0 & 0 & 0 & 1 & 8644 & 8644 & 8645 & 40.96 & 40.65 & 8644 & 8644 & 8644 & 8644 & 8644.0 & 8643.0 & 1 & 1 & 1 & 1 & 97739 & 175000 & 175000 & 344.54 & 382.6 & 331.57 & 380.68 & 9649 & 10072 & 99.29 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.002 & 0.00 & 1239.72 & 393.93 & 8631 & 8631 & 8631 & 8631 & 8631 & 8631 & 8618.0 & 8617.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
