\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8616 & 8616 & 8616 & 0 & 0 & 0 & 1 & 8616 & 8616 & 8666 & 40.78 & 40.65 & 8616 & 8616 & 8616 & 8616 & 8616.0 & 8566.3 & 1 & 1 & 1 & 1 & 15504 & 8800 & 8800 & 113.0 & 112.93 & 112.49 & 112.41 & 505 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 34.82 & 139.28 & 8616 & 8616 & 8616 & 8616 & 8616 & 8616 & 8616.0 & 8566.3 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8627 & 8627 & 8627 & 0 & 0 & 0 & 1 & 8627 & 8627 & 8666 & 40.62 & 40.65 & 8627 & 8627 & 8627 & 8627 & 8627.0 & 8588.2 & 1 & 1 & 1 & 1 & 15507 & 8800 & 8800 & 112.96 & 112.93 & 112.41 & 112.41 & 506 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.988 & 1.001 & 0.00 & 243.45 & 81.15 & 8626 & 8626 & 8626 & 8626 & 8626 & 8626 & 8625.0 & 8586.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8157 & 8157 & 8157 & 0 & 0 & 0 & 1 & 8157 & 8157 & 8666 & 40.25 & 40.65 & 8157 & 8157 & 8157 & 8157 & 8157.0 & 7677.9 & 1 & 1 & 1 & 1 & 15061 & 8800 & 8800 & 71.24 & 112.93 & 42.45 & 112.41 & 318 & 505 & 72.66 & 100.0 & 0 & 0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 + 0 part & 0 & 68.747 & 1.000 & 28736.05 & 691.75 & 50.62 & 5927 & 5927 & 5927 & 5927 & - & - & 4306.6 & 4053.7 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8617 & 8617 & 8617 & 0 & 0 & 0 & 1 & 8617 & 8617 & 8666 & 41.01 & 40.65 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8568.3 & 1 & 1 & 1 & 1 & 15340 & 8800 & 8800 & 106.02 & 112.93 & 98.57 & 112.41 & 463 & 505 & 99.29 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.000 & 0.00 & 2417.48 & 662.48 & 8604 & 8604 & 8604 & 8604 & 8604 & 8604 & 8591.0 & 8542.4 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
