\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8617 & 8617 & 8617 & 0 & 0 & 0 & 1 & 8617 & 8617 & 8665 & 40.59 & 40.60 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8569.3 & 1 & 1 & 1 & 1 & 15448 & 8800 & 8800 & 113.3 & 113.25 & 112.66 & 112.62 & 506 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 11.60 & 127.65 & 8617 & 8617 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8569.3 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8665 & 40.59 & 40.60 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8563.3 & 1 & 1 & 1 & 1 & 15444 & 8800 & 8800 & 113.22 & 113.25 & 112.47 & 112.62 & 506 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.988 & 1.000 & 0.00 & 278.65 & 174.16 & 8613 & 8613 & 8613 & 8613 & 8613 & 8613 & 8612.0 & 8561.3 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8559 & 8559 & 8559 & 0 & 0 & 0 & 1 & 8559 & 8559 & 8665 & 39.99 & 40.60 & 8559 & 8559 & 8559 & 8559 & 8559.0 & 8454.3 & 1 & 1 & 1 & 1 & 15365 & 8800 & 8800 & 58.67 & 113.25 & 32.95 & 112.62 & 253 & 505 & 60.47 & 100.0 & 0 & 0 & 0 & 0 & 0 & 8 & 0 & 0 & 0 + 0 part & 0 & 60.866 & 1.000 & 38614.32 & 476.83 & 95.37 & 5243 & 5243 & 5243 & 5243 & - & - & 3211.7 & 3172.4 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8641 & 8641 & 8641 & 0 & 0 & 0 & 1 & 8641 & 8641 & 8665 & 40.92 & 40.60 & 8641 & 8641 & 8641 & 8641 & 8641.0 & 8617.1 & 1 & 1 & 1 & 1 & 15302 & 8800 & 8800 & 106.27 & 113.25 & 98.68 & 112.62 & 463 & 505 & 99.88 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.816 & 1.003 & 0.00 & 2564.70 & 761.21 & 8539 & 8539 & 8539 & 8539 & 8539 & 8539 & 8438.2 & 8414.8 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
