\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
Total length ($\geq$ 1000 bp) & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
Total length ($\geq$ 5000 bp) & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
Total length & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
Reference length & 8648 & 8648 & 8648 & 8648 \\ \hline
GC (\%) & 40.79 & 40.83 & 40.82 & 41.05 \\ \hline
Reference GC (\%) & 40.79 & 40.79 & 40.79 & 40.79 \\ \hline
N50 & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
NG50 & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
N90 & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
NG90 & 8624 & 8624 & 8616 & {\bf 8638} \\ \hline
auN & 8624.0 & 8624.0 & 8616.0 & {\bf 8638.0} \\ \hline
auNG & 8600.1 & 8600.1 & 8584.1 & {\bf 8628.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15365 & 15363 & 15365 & 15299 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 113.99} & 113.86 & {\bf 113.99} & 109.73 \\ \hline
Reference mapped (\%) & 113.98 & 113.98 & 113.98 & 113.98 \\ \hline
Properly paired (\%) & {\bf 113.44} & 113.17 & {\bf 113.44} & 105.03 \\ \hline
Reference properly paired (\%) & 113.42 & 113.42 & 113.42 & 113.42 \\ \hline
Avg. coverage depth & {\bf 507} & 506 & {\bf 507} & 480 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.93 & 100.0 & 99.19 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.849 & 99.977 & 99.849 \\ \hline
Duplication ratio & 1.000 & 1.000 & {\bf 0.999} & 1.002 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 23.21 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 162.58 & {\bf 0.00} & 1321.74 \\ \hline
\# indels per 100 kbp & {\bf 69.57} & 69.68 & 81.24 & 544.93 \\ \hline
Largest alignment & 8624 & 8611 & 8616 & {\bf 8625} \\ \hline
Total aligned length & 8624 & 8611 & 8616 & {\bf 8625} \\ \hline
NA50 & 8624 & 8611 & 8616 & {\bf 8625} \\ \hline
NGA50 & 8624 & 8611 & 8616 & {\bf 8625} \\ \hline
NA90 & 8624 & 8611 & 8616 & {\bf 8625} \\ \hline
NGA90 & 8624 & 8611 & 8616 & {\bf 8625} \\ \hline
auNA & {\bf 8624.0} & 8598.0 & 8616.0 & 8612.0 \\ \hline
auNGA & 8600.1 & 8574.2 & 8584.1 & {\bf 8602.1} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
