\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8624 & {\bf 8638} & 4878 & 8624 \\ \hline
Total length ($\geq$ 1000 bp) & 8624 & {\bf 8638} & 4878 & 8624 \\ \hline
Total length ($\geq$ 5000 bp) & 8624 & {\bf 8638} & 0 & 8624 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8624 & {\bf 8638} & 4878 & 8624 \\ \hline
Total length & 8624 & {\bf 8638} & 4878 & 8624 \\ \hline
Reference length & 8707 & 8707 & 8707 & 8707 \\ \hline
GC (\%) & - & 41.18 & 40.36 & 41.26 \\ \hline
Reference GC (\%) & 40.50 & 40.50 & 40.50 & 40.50 \\ \hline
N50 & 8624 & {\bf 8638} & 4878 & 8624 \\ \hline
NG50 & 8624 & {\bf 8638} & 4878 & 8624 \\ \hline
N90 & 8624 & {\bf 8638} & 4878 & 8624 \\ \hline
NG90 & 8624 & {\bf 8638} & - & 8624 \\ \hline
auN & 8624.0 & {\bf 8638.0} & 4878.0 & 8624.0 \\ \hline
auNG & 8541.8 & {\bf 8569.5} & 2732.8 & 8541.8 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 380000 & 217055 & 236357 & 323508 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & 0.0 & {\bf 92.36} & 75.42 & 20.51 \\ \hline
Reference mapped (\%) & 319.94 & 319.94 & 319.94 & 319.94 \\ \hline
Properly paired (\%) & 0.0 & {\bf 67.45} & 58.06 & 8.32 \\ \hline
Reference properly paired (\%) & 305.76 & 305.76 & 305.76 & 305.76 \\ \hline
Avg. coverage depth & 0 & 5648 & {\bf 8993} & 1824 \\ \hline
Reference avg. coverage depth & 10254 & 10254 & 10254 & 10254 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 63.65 & 93.5 & 57.88 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 6 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 2 & 0 \\ \hline
\# structural variations & - & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 1 \\ \hline
\# unaligned contigs & 1 + 0 part & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & 8624 & 3036 & {\bf 0} & 7682 \\ \hline
Genome fraction (\%) & - & 64.491 & 52.504 & 10.846 \\ \hline
Duplication ratio & - & 1.000 & 1.000 & 1.000 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & 6498.56 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 17.85 & 263.10 & 1592.36 \\ \hline
\# indels per 100 kbp & - & 53.55 & 43.85 & 530.79 \\ \hline
Largest alignment & - & 5602 & 4561 & 733 \\ \hline
Total aligned length & - & 5602 & 4561 & 942 \\ \hline
NA50 & - & 5602 & 4561 & - \\ \hline
NGA50 & - & 5602 & 4561 & - \\ \hline
NA90 & - & - & 4561 & - \\ \hline
NGA90 & - & - & - & - \\ \hline
auNA & - & 3633.1 & 4264.6 & 67.4 \\ \hline
auNGA & - & 3604.3 & 2389.2 & 66.7 \\ \hline
LA50 & - & 1 & 1 & - \\ \hline
LGA50 & - & 1 & 1 & - \\ \hline
LA90 & - & - & 1 & - \\ \hline
LGA90 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
