\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8707 & - & 40.50 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8541.8 & 1 & 1 & 1 & 1 & 380000 & 190000 & 190000 & 0.0 & 319.94 & 0.0 & 305.76 & 0 & 10254 & 0.0 & 100.0 & - & - & - & - & - & - & - & - & 1 + 0 part & 8624 & - & - & 100000.00 & - & - & - & - & - & - & - & - & - & - & - & - & - & - \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8707 & 41.18 & 40.50 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8569.5 & 1 & 1 & 1 & 1 & 217055 & 190000 & 190000 & 92.36 & 319.94 & 67.45 & 305.76 & 5648 & 10254 & 63.65 & 100.0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 & 0 & 0 + 1 part & 3036 & 64.491 & 1.000 & 0.00 & 17.85 & 53.55 & 5602 & 5602 & 5602 & 5602 & - & - & 3633.1 & 3604.3 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 0 & 0 & 0 & 0 & 4878 & 4878 & 0 & 0 & 0 & 0 & 1 & 4878 & 4878 & 8707 & 40.36 & 40.50 & 4878 & 4878 & 4878 & - & 4878.0 & 2732.8 & 1 & 1 & 1 & - & 236357 & 190000 & 190000 & 75.42 & 319.94 & 58.06 & 305.76 & 8993 & 10254 & 93.5 & 100.0 & 0 & 0 & 0 & 0 & 0 & 2 & 0 & 0 & 0 + 0 part & 0 & 52.504 & 1.000 & 6498.56 & 263.10 & 43.85 & 4561 & 4561 & 4561 & 4561 & 4561 & - & 4264.6 & 2389.2 & 1 & 1 & 1 & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8707 & 41.26 & 40.50 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8541.8 & 1 & 1 & 1 & 1 & 323508 & 190000 & 190000 & 20.51 & 319.94 & 8.32 & 305.76 & 1824 & 10254 & 57.88 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7682 & 10.846 & 1.000 & 0.00 & 1592.36 & 530.79 & 733 & 942 & - & - & - & - & 67.4 & 66.7 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
