\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
Total length ($\geq$ 1000 bp) & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
Total length ($\geq$ 5000 bp) & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
Total length & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
Reference length & 8661 & 8661 & 8661 & 8661 \\ \hline
GC (\%) & 40.48 & 40.55 & 40.38 & 40.77 \\ \hline
Reference GC (\%) & 40.50 & 40.50 & 40.50 & 40.50 \\ \hline
N50 & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
NG50 & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
N75 & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
NG75 & 8638 & 8638 & 8622 & {\bf 8639} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.56} & 99.36 & 82.8 & 93.94 \\ \hline
Reference mapped (\%) & 99.57 & 99.57 & 99.57 & 99.57 \\ \hline
Properly paired (\%) & {\bf 99.11} & 98.73 & 72.81 & 88.74 \\ \hline
Reference properly paired (\%) & 99.15 & 99.15 & 99.15 & 99.15 \\ \hline
Avg. coverage depth & {\bf 506} & 505 & 415 & 472 \\ \hline
Reference avg. coverage depth & 505 & 505 & 505 & 505 \\ \hline
Coverage $\geq$ 1x (\%) & 99.99 & 99.95 & 88.67 & 99.31 \\ \hline
Reference coverage $\geq$ 1x (\%) & 99.99 & 99.99 & 99.99 & 99.99 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 3 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 99.919} & 99.595 & 89.524 & 97.442 \\ \hline
Duplication ratio & {\bf 1.001} & 1.004 & 1.115 & 1.026 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 12073.76 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 46.34} & 255.70 & 258.60 & 1425.52 \\ \hline
\# indels per 100 kbp & {\bf 57.92} & 58.11 & 77.58 & 439.53 \\ \hline
Largest alignment & {\bf 8638} & 8605 & 7729 & 8430 \\ \hline
Total aligned length & {\bf 8638} & 8605 & 7729 & 8430 \\ \hline
NA50 & {\bf 8638} & 8605 & 7729 & 8430 \\ \hline
NGA50 & {\bf 8638} & 8605 & 7729 & 8430 \\ \hline
NA75 & {\bf 8638} & 8605 & 7729 & 8430 \\ \hline
NGA75 & {\bf 8638} & 8605 & 7729 & 8430 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
