\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
Total length ($\geq$ 1000 bp) & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
Total length ($\geq$ 5000 bp) & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
Total length & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
Reference length & 8637 & 8637 & 8637 & 8637 \\ \hline
GC (\%) & 40.28 & 40.17 & 39.92 & 40.72 \\ \hline
Reference GC (\%) & 40.16 & 40.16 & 40.16 & 40.16 \\ \hline
N50 & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
NG50 & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
N75 & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
NG75 & 8589 & 8574 & {\bf 8619} & 8606 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 350000 & 350000 & 350000 & 350000 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 98.7} & 98.15 & 82.53 & 90.86 \\ \hline
Reference mapped (\%) & 98.7 & 98.7 & 98.7 & 98.7 \\ \hline
Properly paired (\%) & {\bf 97.39} & 96.31 & 69.45 & 83.23 \\ \hline
Reference properly paired (\%) & 97.39 & 97.39 & 97.39 & 97.39 \\ \hline
Avg. coverage depth & {\bf 10015} & 9952 & 8229 & 9091 \\ \hline
Reference avg. coverage depth & 10004 & 10004 & 10004 & 10004 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 90.78 & 99.44 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 3 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & {\bf 100.000} & 92.908 & 98.744 \\ \hline
Duplication ratio & 0.999 & {\bf 0.997} & 1.079 & 1.013 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 9467.46 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 220.90} & 837.11 & 1977.22 & 2201.81 \\ \hline
\# indels per 100 kbp & 162.77 & 151.15 & {\bf 137.65} & 365.01 \\ \hline
Largest alignment & {\bf 8589} & 8574 & 8009 & 8499 \\ \hline
Total aligned length & {\bf 8589} & 8574 & 8009 & 8499 \\ \hline
NA50 & {\bf 8589} & 8574 & 8009 & 8499 \\ \hline
NGA50 & {\bf 8589} & 8574 & 8009 & 8499 \\ \hline
NA75 & {\bf 8589} & 8574 & 8009 & 8499 \\ \hline
NGA75 & {\bf 8589} & 8574 & 8009 & 8499 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
