\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{61}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N75 & NG75 & L50 & LG50 & L75 & LG75 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA75 & NGA75 & LA50 & LGA50 & LA75 & LGA75 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8589 & 8589 & 8589 & 0 & 0 & 0 & 1 & 8589 & 8589 & 8637 & 40.28 & 40.16 & 8589 & 8589 & 8589 & 8589 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 98.7 & 98.7 & 97.39 & 97.39 & 10015 & 10004 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 0.00 & 220.90 & 162.77 & 8589 & 8589 & 8589 & 8589 & 8589 & 8589 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8574 & 8574 & 8574 & 0 & 0 & 0 & 1 & 8574 & 8574 & 8637 & 40.17 & 40.16 & 8574 & 8574 & 8574 & 8574 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 98.15 & 98.7 & 96.31 & 97.39 & 9952 & 10004 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.997 & 0.00 & 837.11 & 151.15 & 8574 & 8574 & 8574 & 8574 & 8574 & 8574 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8619 & 8619 & 8619 & 0 & 0 & 0 & 1 & 8619 & 8619 & 8637 & 39.92 & 40.16 & 8619 & 8619 & 8619 & 8619 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 82.53 & 98.7 & 69.45 & 97.39 & 8229 & 10004 & 90.78 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 + 0 part & 0 & 92.908 & 1.079 & 9467.46 & 1977.22 & 137.65 & 8009 & 8009 & 8009 & 8009 & 8009 & 8009 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8606 & 8606 & 8606 & 0 & 0 & 0 & 1 & 8606 & 8606 & 8637 & 40.72 & 40.16 & 8606 & 8606 & 8606 & 8606 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 90.86 & 98.7 & 83.23 & 97.39 & 9091 & 10004 & 99.44 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.744 & 1.013 & 0.00 & 2201.81 & 365.01 & 8499 & 8499 & 8499 & 8499 & 8499 & 8499 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
