\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
Total length & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
Reference length & 8719 & 8719 & 8719 & 8719 \\ \hline
GC (\%) & 40.53 & 41.31 & - & 41.27 \\ \hline
Reference GC (\%) & 40.36 & 40.36 & 40.36 & 40.36 \\ \hline
N50 & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
NG50 & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
N90 & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
NG90 & {\bf 8677} & 8638 & 8624 & 8624 \\ \hline
auN & {\bf 8677.0} & 8638.0 & 8624.0 & 8624.0 \\ \hline
auNG & {\bf 8635.2} & 8557.8 & 8530.0 & 8530.0 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 16854 & 17103 & 19000 & 18297 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 105.5} & 58.26 & 0.0 & 15.77 \\ \hline
Reference mapped (\%) & 106.02 & 106.02 & 106.02 & 106.02 \\ \hline
Properly paired (\%) & {\bf 100.36} & 42.4 & 0.0 & 6.39 \\ \hline
Reference properly paired (\%) & 101.51 & 101.51 & 101.51 & 101.51 \\ \hline
Avg. coverage depth & {\bf 508} & 281 & 0 & 79 \\ \hline
Reference avg. coverage depth & 512 & 512 & 512 & 512 \\ \hline
Coverage $\geq$ 1x (\%) & 99.95 & 63.07 & 0.0 & 55.96 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & {\bf 0} & {\bf 0} & - & - \\ \hline
\# misassembled contigs & {\bf 0} & {\bf 0} & - & - \\ \hline
Misassembled contigs length & {\bf 0} & {\bf 0} & - & - \\ \hline
\# local misassemblies & {\bf 0} & 7 & - & - \\ \hline
\# scaffold gap ext. mis. & {\bf 0} & {\bf 0} & - & - \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & - & - \\ \hline
\# structural variations & 0 & 0 & - & - \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & - & - \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & 1 + 0 part & 1 + 0 part \\ \hline
Unaligned length & {\bf 0} & 2861 & 8624 & 8624 \\ \hline
Genome fraction (\%) & {\bf 99.896} & 66.502 & - & - \\ \hline
Duplication ratio & {\bf 0.999} & 1.000 & - & - \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 100000.00 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 323.06} & 346.20 & - & - \\ \hline
\# indels per 100 kbp & 184.61 & {\bf 51.93} & - & - \\ \hline
Largest alignment & {\bf 8667} & 5777 & - & - \\ \hline
Total aligned length & {\bf 8667} & 5777 & - & - \\ \hline
NA50 & {\bf 8667} & 5777 & - & - \\ \hline
NGA50 & {\bf 8667} & 5777 & - & - \\ \hline
NA90 & {\bf 8667} & - & - & - \\ \hline
NGA90 & {\bf 8667} & - & - & - \\ \hline
auNA & {\bf 8657.0} & 3863.6 & - & - \\ \hline
auNGA & {\bf 8615.3} & 3827.7 & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA90 & {\bf 1} & - & - & - \\ \hline
LGA90 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
