\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8677 & 8677 & 8677 & 0 & 0 & 0 & 1 & 8677 & 8677 & 8704 & 40.69 & 40.70 & 8677 & 8677 & 8677 & 8677 & 8677.0 & 8650.1 & 1 & 1 & 1 & 1 & 113875 & 190000 & 190000 & 313.79 & 313.01 & 300.13 & 299.38 & 10262 & 10256 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 103.72 & 8677 & 8677 & 8677 & 8677 & 8677 & 8677 & 8677.0 & 8650.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8704 & 41.24 & 40.70 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8572.5 & 1 & 1 & 1 & 1 & 217395 & 190000 & 190000 & 92.56 & 313.01 & 67.63 & 299.38 & 5673 & 10256 & 65.83 & 100.0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 & 0 & 0 + 1 part & 2859 & 66.598 & 1.000 & 0.00 & 138.43 & 51.91 & 5779 & 5779 & 5779 & 5779 & - & - & 3866.3 & 3837.0 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 0 & 0 & 0 & 0 & 4644 & 4644 & 0 & 0 & 0 & 0 & 1 & 4644 & 4644 & 8704 & 39.16 & 40.70 & 4644 & 4644 & 4644 & - & 4644.0 & 2477.8 & 1 & 1 & 1 & - & 274078 & 190000 & 190000 & 46.73 & 313.01 & 28.35 & 299.38 & 6664 & 10256 & 77.2 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 41.298 & 1.000 & 22803.62 & 892.86 & 83.71 & 3584 & 3584 & 3584 & - & - & - & 2765.9 & 1475.8 & 1 & - & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8704 & 41.28 & 40.70 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8544.7 & 1 & 1 & 1 & 1 & 334696 & 190000 & 190000 & 15.98 & 313.01 & 5.46 & 299.38 & 1462 & 10256 & 55.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7968 & 7.561 & 1.000 & 0.00 & 0.00 & 304.88 & 447 & 656 & - & - & - & - & 28.2 & 28.0 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
