\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8620 & 40.17 & 40.18 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8590.0 & 1 & 1 & 1 & 1 & 94395 & 175000 & 175000 & 368.95 & 368.21 & 367.11 & 366.38 & 10091 & 10089 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 58.11 & 8605 & 8605 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8590.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8620 & 40.08 & 40.18 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8590.0 & 1 & 1 & 1 & 1 & 94445 & 175000 & 175000 & 368.69 & 368.21 & 366.8 & 366.38 & 10090 & 10089 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 174.32 & 58.11 & 8605 & 8605 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8590.0 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8020 & 8020 & 8020 & 0 & 0 & 0 & 1 & 8020 & 8020 & 8620 & 39.18 & 40.18 & 8020 & 8020 & 8020 & 8020 & 8020.0 & 7461.8 & 1 & 1 & 1 & 1 & 114278 & 175000 & 175000 & 274.49 & 368.21 & 249.0 & 366.38 & 9677 & 10089 & 97.37 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 91.734 & 0.999 & 2643.39 & 1179.45 & 76.09 & 7885 & 7885 & 7885 & 7885 & 7885 & 7885 & 7752.3 & 7212.7 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8617 & 8617 & 8617 & 0 & 0 & 0 & 1 & 8617 & 8617 & 8620 & 40.16 & 40.18 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8614.0 & 1 & 1 & 1 & 1 & 94573 & 175000 & 175000 & 368.25 & 368.21 & 366.42 & 366.38 & 10090 & 10089 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.002 & 0.00 & 0.00 & 11.60 & 8617 & 8617 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8614.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
