\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
Total length ($\geq$ 1000 bp) & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
Total length ($\geq$ 5000 bp) & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
Total length & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
Reference length & 8644 & 8644 & 8644 & 8644 \\ \hline
GC (\%) & 40.76 & 40.84 & 40.75 & 41.03 \\ \hline
Reference GC (\%) & 40.75 & 40.75 & 40.75 & 40.75 \\ \hline
N50 & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
NG50 & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
N90 & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
NG90 & 8609 & 8614 & 8614 & {\bf 8641} \\ \hline
auN & 8609.0 & 8614.0 & 8614.0 & {\bf 8641.0} \\ \hline
auNG & 8574.1 & 8584.1 & 8584.1 & {\bf 8638.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15428 & 15427 & 15426 & 15350 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 113.44 & 113.24 & {\bf 113.45} & 109.97 \\ \hline
Reference mapped (\%) & 113.44 & 113.44 & 113.44 & 113.44 \\ \hline
Properly paired (\%) & 112.79 & 112.39 & {\bf 112.81} & 105.97 \\ \hline
Reference properly paired (\%) & 112.79 & 112.79 & 112.79 & 112.79 \\ \hline
Avg. coverage depth & {\bf 507} & 506 & {\bf 507} & 482 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.88 & 100.0 & 99.88 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.932 & {\bf 100.000} & 98.015 \\ \hline
Duplication ratio & {\bf 0.999} & 1.000 & 1.000 & 1.003 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & {\bf 0.00} & {\bf 0.00} & 1157.02 \\ \hline
\# indels per 100 kbp & 92.93 & {\bf 58.67} & 69.65 & 637.54 \\ \hline
Largest alignment & 8609 & 8522 & {\bf 8614} & 8470 \\ \hline
Total aligned length & 8609 & 8522 & {\bf 8614} & 8470 \\ \hline
NA50 & 8609 & 8522 & {\bf 8614} & 8470 \\ \hline
NGA50 & 8609 & 8522 & {\bf 8614} & 8470 \\ \hline
NA90 & 8609 & 8522 & {\bf 8614} & 8470 \\ \hline
NGA90 & 8609 & 8522 & {\bf 8614} & 8470 \\ \hline
auNA & 8609.0 & 8431.0 & {\bf 8614.0} & 8302.4 \\ \hline
auNGA & 8574.1 & 8401.7 & {\bf 8584.1} & 8299.5 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
