\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8682} & 8639 & 3002 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8682} & 8639 & 3002 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8682} & 8639 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8682} & 8639 & 3002 & 8625 \\ \hline
Total length & {\bf 8682} & 8639 & 3002 & 8625 \\ \hline
Reference length & 8738 & 8738 & 8738 & 8738 \\ \hline
GC (\%) & 40.80 & 41.32 & 43.00 & 41.30 \\ \hline
Reference GC (\%) & 40.77 & 40.77 & 40.77 & 40.77 \\ \hline
N50 & {\bf 8682} & 8639 & 3002 & 8625 \\ \hline
NG50 & {\bf 8682} & 8639 & - & 8625 \\ \hline
N75 & {\bf 8682} & 8639 & 3002 & 8625 \\ \hline
NG75 & {\bf 8682} & 8639 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 380000 & 380000 & 380000 & 380000 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 93.97} & 53.28 & 16.27 & 22.93 \\ \hline
Reference mapped (\%) & 94.5 & 94.5 & 94.5 & 94.5 \\ \hline
Properly paired (\%) & {\bf 89.46} & 38.76 & 7.06 & 10.04 \\ \hline
Reference properly paired (\%) & 90.53 & 90.53 & 90.53 & 90.53 \\ \hline
Avg. coverage depth & {\bf 10191} & 5702 & 4916 & 2423 \\ \hline
Reference avg. coverage depth & 10262 & 10262 & 10262 & 10262 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 65.66 & 59.26 & 60.16 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 7 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 2834 & {\bf 0} & 7294 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 66.571 & 20.401 & 15.264 \\ \hline
Duplication ratio & {\bf 0.996} & 1.000 & 1.687 & 1.000 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 40739.51 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & 45.87 & {\bf 0.00} & 112.42 & 300.53 \\ \hline
\# indels per 100 kbp & 114.68 & 86.13 & {\bf 56.21} & 375.66 \\ \hline
Largest alignment & {\bf 8682} & 5805 & 1779 & 851 \\ \hline
Total aligned length & {\bf 8682} & 5805 & 1779 & 1331 \\ \hline
NA50 & {\bf 8682} & 5805 & 1779 & - \\ \hline
NGA50 & {\bf 8682} & 5805 & - & - \\ \hline
NA75 & {\bf 8682} & - & - & - \\ \hline
NGA75 & {\bf 8682} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
