\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
Total length ($\geq$ 1000 bp) & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
Total length ($\geq$ 5000 bp) & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
Total length & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
Reference length & 8661 & 8661 & 8661 & 8661 \\ \hline
GC (\%) & 39.73 & 39.67 & 39.84 & 40.25 \\ \hline
Reference GC (\%) & 39.72 & 39.72 & 39.72 & 39.72 \\ \hline
N50 & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
NG50 & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
N90 & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
NG90 & 8598 & 8580 & 8583 & {\bf 8616} \\ \hline
auN & 8598.0 & 8580.0 & 8583.0 & {\bf 8616.0} \\ \hline
auNG & 8535.5 & 8499.8 & 8505.7 & {\bf 8571.2} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 96260 & 97392 & 99501 & 112430 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 358.63} & 352.87 & 337.79 & 278.45 \\ \hline
Reference mapped (\%) & 356.1 & 356.1 & 356.1 & 356.1 \\ \hline
Properly paired (\%) & {\bf 353.67} & 346.38 & 323.82 & 250.81 \\ \hline
Reference properly paired (\%) & 351.16 & 351.16 & 351.16 & 351.16 \\ \hline
Avg. coverage depth & {\bf 10006} & 9949 & 9723 & 8950 \\ \hline
Reference avg. coverage depth & 9978 & 9978 & 9978 & 9978 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 98.89 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & {\bf 100.000} & 99.826 & 99.453 \\ \hline
Duplication ratio & 1.000 & {\bf 0.998} & 1.000 & 1.002 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 2003.96 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 209.79 & 1316.56 & 1773.84 \\ \hline
\# indels per 100 kbp & {\bf 197.72} & 198.14 & 198.07 & 548.49 \\ \hline
Largest alignment & {\bf 8598} & 8580 & 8583 & 8569 \\ \hline
Total aligned length & {\bf 8598} & 8580 & 8583 & 8569 \\ \hline
NA50 & {\bf 8598} & 8580 & 8583 & 8569 \\ \hline
NGA50 & {\bf 8598} & 8580 & 8583 & 8569 \\ \hline
NA90 & {\bf 8598} & 8580 & 8583 & 8569 \\ \hline
NGA90 & {\bf 8598} & 8580 & 8583 & 8569 \\ \hline
auNA & {\bf 8598.0} & 8580.0 & 8583.0 & 8522.3 \\ \hline
auNGA & {\bf 8535.5} & 8499.8 & 8505.7 & 8478.0 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
