\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8595 & 8595 & 8595 & 0 & 0 & 0 & 1 & 8595 & 8595 & 8637 & 40.28 & 40.27 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8553.2 & 1 & 1 & 1 & 1 & 89382 & 175000 & 175000 & 386.41 & 384.75 & 381.25 & 379.6 & 10016 & 9997 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 11.63 & 139.62 & 8595 & 8595 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8553.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8574 & 8574 & 8574 & 0 & 0 & 0 & 1 & 8574 & 8574 & 8637 & 40.26 & 40.27 & 8574 & 8574 & 8574 & 8574 & 8574.0 & 8511.5 & 1 & 1 & 1 & 1 & 90663 & 175000 & 175000 & 378.28 & 384.75 & 370.51 & 379.6 & 9936 & 9997 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.882 & 0.998 & 0.00 & 318.47 & 141.54 & 8478 & 8478 & 8478 & 8478 & 8478 & 8478 & 8383.1 & 8321.9 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8569 & 8569 & 8569 & 0 & 0 & 0 & 1 & 8569 & 8569 & 8637 & 39.80 & 40.27 & 8569 & 8569 & 8569 & 8569 & 8569.0 & 8501.5 & 1 & 1 & 1 & 1 & 131806 & 175000 & 175000 & 215.3 & 384.75 & 195.0 & 379.6 & 8170 & 9997 & 86.78 & 100.0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 + 0 part & 0 & 87.590 & 1.001 & 14727.51 & 1965.73 & 119.54 & 7529 & 7529 & 7529 & 7529 & - & - & 6615.2 & 6563.1 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8610 & 8610 & 8610 & 0 & 0 & 0 & 1 & 8610 & 8610 & 8637 & 40.71 & 40.27 & 8610 & 8610 & 8610 & 8610 & 8610.0 & 8583.1 & 1 & 1 & 1 & 1 & 108984 & 175000 & 175000 & 288.22 & 384.75 & 260.27 & 379.6 & 8980 & 9997 & 99.55 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.068 & 1.002 & 0.00 & 1646.33 & 592.21 & 8443 & 8443 & 8443 & 8443 & 8443 & 8443 & 8279.2 & 8253.4 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
