\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
Total length ($\geq$ 1000 bp) & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
Total length ($\geq$ 5000 bp) & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
Total length & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
Reference length & 8634 & 8634 & 8634 & 8634 \\ \hline
GC (\%) & 40.28 & 40.28 & 40.11 & 40.63 \\ \hline
Reference GC (\%) & 40.33 & 40.33 & 40.33 & 40.33 \\ \hline
N50 & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
NG50 & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
N75 & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
NG75 & 8604 & 8579 & 8588 & {\bf 8635} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 350000 & 350000 & 350000 & 350000 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 98.7} & 98.07 & 83.77 & 93.63 \\ \hline
Reference mapped (\%) & 98.69 & 98.69 & 98.69 & 98.69 \\ \hline
Properly paired (\%) & {\bf 97.4} & 96.15 & 68.98 & 88.47 \\ \hline
Reference properly paired (\%) & 97.39 & 97.39 & 97.39 & 97.39 \\ \hline
Avg. coverage depth & {\bf 10013} & 9942 & 8408 & 9399 \\ \hline
Reference avg. coverage depth & 10002 & 10002 & 10002 & 10002 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 89.57 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 3 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.302 & 89.405 & 98.000 \\ \hline
Duplication ratio & {\bf 1.001} & 1.005 & 1.117 & 1.025 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 11737.31 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 46.52} & 374.80 & 1457.01 & 996.91 \\ \hline
\# indels per 100 kbp & 139.57 & {\bf 128.84} & 130.09 & 356.04 \\ \hline
Largest alignment & {\bf 8603} & 8520 & 7690 & 8464 \\ \hline
Total aligned length & {\bf 8603} & 8520 & 7690 & 8464 \\ \hline
NA50 & {\bf 8603} & 8520 & 7690 & 8464 \\ \hline
NGA50 & {\bf 8603} & 8520 & 7690 & 8464 \\ \hline
NA75 & {\bf 8603} & 8520 & 7690 & 8464 \\ \hline
NGA75 & {\bf 8603} & 8520 & 7690 & 8464 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
