\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8622 & 8622 & 8622 & 0 & 0 & 0 & 1 & 8622 & 8622 & 8642 & 40.62 & 40.62 & 8622 & 8622 & 8622 & 8622 & 8622.0 & 8602.0 & 1 & 1 & 1 & 1 & 80752 & 175000 & 175000 & 431.28 & 430.62 & 429.14 & 428.48 & 10082 & 10077 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 115.98 & 81.19 & 8622 & 8622 & 8622 & 8622 & 8622 & 8622 & 8622.0 & 8602.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8618 & 8618 & 8618 & 0 & 0 & 0 & 1 & 8618 & 8618 & 8642 & 40.61 & 40.62 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8594.1 & 1 & 1 & 1 & 1 & 80680 & 175000 & 175000 & 431.67 & 430.62 & 429.52 & 428.48 & 10085 & 10077 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 139.24 & 81.23 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8594.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8595 & 8595 & 8595 & 0 & 0 & 0 & 1 & 8595 & 8595 & 8642 & 40.65 & 40.62 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8548.3 & 1 & 1 & 1 & 1 & 80943 & 175000 & 175000 & 429.64 & 430.62 & 426.88 & 428.48 & 10098 & 10077 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.733 & 1.000 & 0.00 & 407.21 & 81.44 & 8595 & 8595 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8548.3 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8632 & 8632 & 8632 & 0 & 0 & 0 & 1 & 8632 & 8632 & 8642 & 41.04 & 40.62 & 8632 & 8632 & 8632 & 8632 & 8632.0 & 8622.0 & 1 & 1 & 1 & 1 & 90449 & 175000 & 175000 & 368.6 & 430.62 & 351.59 & 428.48 & 9576 & 10077 & 99.33 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.120 & 1.002 & 0.00 & 1286.89 & 389.61 & 8470 & 8470 & 8470 & 8470 & 8470 & 8470 & 8311.0 & 8301.4 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
