\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8720} & 8639 & 3001 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8720} & 8639 & 3001 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8720} & 8639 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8720} & 8639 & 3001 & 8625 \\ \hline
Total length & {\bf 8720} & 8639 & 3001 & 8625 \\ \hline
Reference length & 8723 & 8723 & 8723 & 8723 \\ \hline
GC (\%) & 40.76 & 41.34 & 43.13 & 41.33 \\ \hline
Reference GC (\%) & 40.76 & 40.76 & 40.76 & 40.76 \\ \hline
N50 & {\bf 8720} & 8639 & 3001 & 8625 \\ \hline
NG50 & {\bf 8720} & 8639 & - & 8625 \\ \hline
N75 & {\bf 8720} & 8639 & 3001 & 8625 \\ \hline
NG75 & {\bf 8720} & 8639 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 19000 & 19000 & 19000 & 19000 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 94.79} & 54.19 & 16.81 & 22.63 \\ \hline
Reference mapped (\%) & 94.78 & 94.78 & 94.78 & 94.78 \\ \hline
Properly paired (\%) & {\bf 90.88} & 39.28 & 7.38 & 9.59 \\ \hline
Reference properly paired (\%) & 90.85 & 90.85 & 90.85 & 90.85 \\ \hline
Avg. coverage depth & {\bf 514} & 288 & 253 & 118 \\ \hline
Reference avg. coverage depth & 514 & 514 & 514 & 514 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 65.57 & 59.18 & 54.24 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 7 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 2829 & {\bf 0} & 7417 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 66.651 & 20.374 & 13.858 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 1.690 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 40819.73 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & {\bf 0.00} & {\bf 0.00} & 1490.07 \\ \hline
\# indels per 100 kbp & 22.94 & 17.21 & {\bf 0.00} & 331.13 \\ \hline
Largest alignment & {\bf 8717} & 5810 & 1776 & 733 \\ \hline
Total aligned length & {\bf 8717} & 5810 & 1776 & 1208 \\ \hline
NA50 & {\bf 8717} & 5810 & 1776 & - \\ \hline
NGA50 & {\bf 8717} & 5810 & - & - \\ \hline
NA75 & {\bf 8717} & - & - & - \\ \hline
NGA75 & {\bf 8717} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
