\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8713 & 40.32 & 40.34 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8659.1 & 1 & 1 & 1 & 1 & 16720 & 9500 & 9500 & 107.1 & 107.07 & 102.6 & 102.55 & 513 & 512 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 115.13 & 80.59 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8659.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8713 & 41.11 & 40.34 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8563.6 & 1 & 1 & 1 & 1 & 17190 & 9500 & 9500 & 56.85 & 107.07 & 43.28 & 102.55 & 276 & 512 & 59.96 & 100.0 & 0 & 0 & 0 & 5 & 0 & 0 & 0 & 0 & 0 + 1 part & 3301 & 61.435 & 1.000 & 0.00 & 206.11 & 37.47 & 5337 & 5337 & 5337 & 5337 & - & - & 3297.5 & 3269.1 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 5272 & 5272 & 5272 & 0 & 0 & 0 & 1 & 5272 & 5272 & 8713 & 40.02 & 40.34 & 5272 & 5272 & 5272 & - & 5272.0 & 3189.9 & 1 & 1 & 1 & - & 17299 & 9500 & 9500 & 49.27 & 107.07 & 34.31 & 102.55 & 396 & 512 & 82.3 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 51.624 & 1.000 & 15876.33 & 1338.09 & 44.60 & 4484 & 4484 & 4484 & 4484 & - & - & 3813.8 & 2307.6 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8713 & 41.30 & 40.34 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8535.9 & 1 & 1 & 1 & 1 & 18213 & 9500 & 9500 & 16.57 & 107.07 & 6.8 & 102.55 & 83 & 512 & 53.21 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 1 part & 8205 & 4.813 & 1.002 & 0.00 & 715.99 & 0.00 & 419 & 419 & - & - & - & - & 20.4 & 20.1 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
