\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{61}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N75 & NG75 & L50 & LG50 & L75 & LG75 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA75 & NGA75 & LA50 & LGA50 & LA75 & LGA75 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8635 & 8635 & 8635 & 0 & 0 & 0 & 1 & 8635 & 8635 & 8671 & 40.28 & 40.26 & 8635 & 8635 & 8635 & 8635 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 99.62 & 99.62 & 99.24 & 99.23 & 10076 & 10065 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 23.16 & 104.24 & 8635 & 8635 & 8635 & 8635 & 8635 & 8635 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8635 & 8635 & 8635 & 0 & 0 & 0 & 1 & 8635 & 8635 & 8671 & 40.25 & 40.26 & 8635 & 8635 & 8635 & 8635 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 99.61 & 99.62 & 99.23 & 99.23 & 10075 & 10065 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 127.40 & 104.24 & 8635 & 8635 & 8635 & 8635 & 8635 & 8635 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8636 & 8636 & 8636 & 0 & 0 & 0 & 1 & 8636 & 8636 & 8671 & 40.34 & 40.26 & 8636 & 8636 & 8636 & 8636 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 99.62 & 99.62 & 99.24 & 99.23 & 10076 & 10065 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.977 & 1.000 & 81.06 & 405.47 & 92.68 & 8636 & 8636 & 8636 & 8636 & 8636 & 8636 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8644 & 8644 & 8644 & 0 & 0 & 0 & 1 & 8644 & 8644 & 8671 & 40.65 & 40.26 & 8644 & 8644 & 8644 & 8644 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 94.76 & 99.62 & 90.06 & 99.23 & 9474 & 10065 & 99.27 & 100.0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 & 0 + 0 part & 0 & 92.877 & 1.078 & 0.00 & 1621.15 & 99.76 & 8019 & 8019 & 8019 & 8019 & 8019 & 8019 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
