\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
Total length ($\geq$ 1000 bp) & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
Total length ($\geq$ 5000 bp) & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
Total length & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
Reference length & 8625 & 8625 & 8625 & 8625 \\ \hline
GC (\%) & 40.27 & 40.25 & 40.36 & 40.79 \\ \hline
Reference GC (\%) & 40.37 & 40.37 & 40.37 & 40.37 \\ \hline
N50 & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
NG50 & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
N90 & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
NG90 & 8569 & 8574 & 8587 & {\bf 8605} \\ \hline
auN & 8569.0 & 8574.0 & 8587.0 & {\bf 8605.0} \\ \hline
auNG & 8513.4 & 8523.3 & 8549.2 & {\bf 8585.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15527 & 15508 & 15513 & 15298 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 110.96 & 111.5 & {\bf 111.84} & 102.16 \\ \hline
Reference mapped (\%) & 111.9 & 111.9 & 111.9 & 111.9 \\ \hline
Properly paired (\%) & 109.25 & 109.5 & {\bf 110.22} & 91.32 \\ \hline
Reference properly paired (\%) & 110.38 & 110.38 & 110.38 & 110.38 \\ \hline
Avg. coverage depth & 499 & {\bf 501} & {\bf 501} & 447 \\ \hline
Reference avg. coverage depth & 503 & 503 & 503 & 503 \\ \hline
Coverage $\geq$ 1x (\%) & 99.9 & 100.0 & 100.0 & 99.27 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 99.930 & {\bf 100.000} & 99.907 & 98.010 \\ \hline
Duplication ratio & {\bf 0.997} & 0.998 & 1.000 & 1.002 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 1979.74 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 105.14} & 116.63 & 547.34 & 2181.65 \\ \hline
\# indels per 100 kbp & 151.87 & 128.29 & {\bf 116.46} & 616.55 \\ \hline
Largest alignment & 8560 & 8574 & {\bf 8587} & 8434 \\ \hline
Total aligned length & 8560 & 8574 & {\bf 8587} & 8434 \\ \hline
NA50 & 8560 & 8574 & {\bf 8587} & 8434 \\ \hline
NGA50 & 8560 & 8574 & {\bf 8587} & 8434 \\ \hline
NA90 & 8560 & 8574 & {\bf 8587} & 8434 \\ \hline
NGA90 & 8560 & 8574 & {\bf 8587} & 8434 \\ \hline
auNA & 8551.0 & 8574.0 & {\bf 8587.0} & 8266.4 \\ \hline
auNGA & 8495.5 & 8523.3 & {\bf 8549.2} & 8247.2 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
