\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
Total length ($\geq$ 1000 bp) & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
Total length ($\geq$ 5000 bp) & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
Total length & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
Reference length & 8692 & 8692 & 8692 & 8692 \\ \hline
GC (\%) & 40.47 & 40.56 & 39.04 & 40.98 \\ \hline
Reference GC (\%) & 40.43 & 40.43 & 40.43 & 40.43 \\ \hline
N50 & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
NG50 & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
N75 & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
NG75 & 8621 & {\bf 8627} & 8329 & 8622 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.55} & 99.28 & 46.47 & 89.02 \\ \hline
Reference mapped (\%) & 99.55 & 99.55 & 99.55 & 99.55 \\ \hline
Properly paired (\%) & {\bf 99.1} & 98.56 & 22.48 & 79.55 \\ \hline
Reference properly paired (\%) & 99.1 & 99.1 & 99.1 & 99.1 \\ \hline
Avg. coverage depth & {\bf 505} & 504 & 232 & 447 \\ \hline
Reference avg. coverage depth & 504 & 504 & 504 & 504 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.92 & 61.09 & 99.05 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 11 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.760 & 60.327 & 97.891 \\ \hline
Duplication ratio & {\bf 0.999} & 1.012 & 1.600 & 1.021 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 37723.62 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 34.77} & 211.24 & 1440.92 & 3113.90 \\ \hline
\# indels per 100 kbp & 220.21 & {\bf 164.30} & 172.91 & 686.72 \\ \hline
Largest alignment & {\bf 8621} & 8522 & 5224 & 8454 \\ \hline
Total aligned length & {\bf 8621} & 8522 & 5224 & 8454 \\ \hline
NA50 & {\bf 8621} & 8522 & 5224 & 8454 \\ \hline
NGA50 & {\bf 8621} & 8522 & 5224 & 8454 \\ \hline
NA75 & {\bf 8621} & 8522 & - & 8454 \\ \hline
NGA75 & {\bf 8621} & 8522 & - & 8454 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
