\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
Total length ($\geq$ 1000 bp) & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
Total length ($\geq$ 5000 bp) & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
Total length & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
Reference length & 8607 & 8607 & 8607 & 8607 \\ \hline
GC (\%) & 40.35 & 40.33 & 40.41 & 40.86 \\ \hline
Reference GC (\%) & 40.40 & 40.40 & 40.40 & 40.40 \\ \hline
N50 & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
NG50 & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
N90 & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
NG90 & 8571 & 8574 & 8595 & {\bf 8604} \\ \hline
auN & 8571.0 & 8574.0 & 8595.0 & {\bf 8604.0} \\ \hline
auNG & 8535.2 & 8541.1 & 8583.0 & {\bf 8601.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15462 & 15431 & 15417 & 15261 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 111.62 & {\bf 111.92} & 109.35 & 102.42 \\ \hline
Reference mapped (\%) & 112.43 & 112.43 & 112.43 & 112.43 \\ \hline
Properly paired (\%) & {\bf 110.08} & 109.79 & 104.55 & 91.58 \\ \hline
Reference properly paired (\%) & 111.12 & 111.12 & 111.12 & 111.12 \\ \hline
Avg. coverage depth & {\bf 500} & {\bf 500} & 487 & 447 \\ \hline
Reference avg. coverage depth & 504 & 504 & 504 & 504 \\ \hline
Coverage $\geq$ 1x (\%) & 99.9 & 100.0 & 98.27 & 99.73 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 99.930} & 99.453 & 98.243 & 98.010 \\ \hline
Duplication ratio & {\bf 0.997} & 0.998 & 1.000 & 1.001 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 1896.45 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 23.36} & 58.64 & 189.48 & 2051.46 \\ \hline
\# indels per 100 kbp & 70.08 & 46.91 & {\bf 35.53} & 545.48 \\ \hline
Largest alignment & {\bf 8562} & 8527 & 8444 & 8433 \\ \hline
Total aligned length & {\bf 8562} & 8527 & 8444 & 8433 \\ \hline
NA50 & {\bf 8562} & 8527 & 8444 & 8433 \\ \hline
NGA50 & {\bf 8562} & 8527 & 8444 & 8433 \\ \hline
NA90 & {\bf 8562} & 8527 & 8444 & 8433 \\ \hline
NGA90 & {\bf 8562} & 8527 & 8444 & 8433 \\ \hline
auNA & {\bf 8553.0} & 8480.3 & 8295.7 & 8265.4 \\ \hline
auNGA & {\bf 8517.2} & 8447.7 & 8284.1 & 8262.5 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
