\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8721 & 8721 & 8721 & 0 & 0 & 0 & 1 & 8721 & 8721 & 8763 & 40.75 & 40.72 & 8721 & 8721 & 8721 & 8721 & 8721.0 & 8679.2 & 1 & 1 & 1 & 1 & 119413 & 190000 & 190000 & 300.52 & 299.48 & 287.73 & 286.79 & 10239 & 10228 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 45.87 & 80.27 & 8721 & 8721 & 8721 & 8721 & 8721 & 8721 & 8721.0 & 8679.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8727 & 8727 & 8727 & 0 & 0 & 0 & 1 & 8727 & 8727 & 8763 & 40.70 & 40.72 & 8727 & 8727 & 8727 & 8727 & 8727.0 & 8691.1 & 1 & 1 & 1 & 1 & 119435 & 190000 & 190000 & 300.49 & 299.48 & 287.72 & 286.79 & 10240 & 10228 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 80.21 & 68.75 & 8727 & 8727 & 8727 & 8727 & 8727 & 8727 & 8727.0 & 8691.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8499 & 8499 & 8499 & 0 & 0 & 0 & 1 & 8499 & 8499 & 8763 & 40.28 & 40.72 & 8499 & 8499 & 8499 & 8499 & 8499.0 & 8243.0 & 1 & 1 & 1 & 1 & 136456 & 190000 & 190000 & 244.93 & 299.48 & 231.09 & 286.79 & 9781 & 10228 & 94.89 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 92.271 & 1.002 & 6212.50 & 731.56 & 49.60 & 8065 & 8065 & 8065 & 8065 & 8065 & 8065 & 7653.2 & 7422.6 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8754 & 8754 & 8754 & 0 & 0 & 0 & 1 & 8754 & 8754 & 8763 & 40.67 & 40.72 & 8754 & 8754 & 8754 & 8754 & 8754.0 & 8745.0 & 1 & 1 & 1 & 1 & 119816 & 190000 & 190000 & 299.56 & 299.48 & 286.86 & 286.79 & 10230 & 10228 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.004 & 0.00 & 125.66 & 22.85 & 8754 & 8754 & 8754 & 8754 & 8754 & 8754 & 8754.0 & 8745.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
