\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
Total length & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
Reference length & 8664 & 8664 & 8664 & 8664 \\ \hline
GC (\%) & 40.71 & 40.75 & 40.57 & 41.04 \\ \hline
Reference GC (\%) & 40.69 & 40.69 & 40.69 & 40.69 \\ \hline
N50 & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
NG50 & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
N75 & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
NG75 & {\bf 8624} & 8608 & 8412 & 8618 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.66} & 99.05 & 98.74 & 89.34 \\ \hline
Reference mapped (\%) & 99.56 & 99.56 & 99.56 & 99.56 \\ \hline
Properly paired (\%) & {\bf 99.33} & 98.1 & 97.49 & 80.41 \\ \hline
Reference properly paired (\%) & 99.12 & 99.12 & 99.12 & 99.12 \\ \hline
Avg. coverage depth & 507 & 503 & {\bf 515} & 448 \\ \hline
Reference avg. coverage depth & 505 & 505 & 505 & 505 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.9 & 100.0 & 98.58 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.737 & 97.393 & 98.656 \\ \hline
Duplication ratio & {\bf 0.999} & 1.010 & 1.001 & 1.012 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 11.89 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 35.20 & 309.30 & 3241.34 \\ \hline
\# indels per 100 kbp & 115.86 & 93.87 & {\bf 59.48} & 634.17 \\ \hline
Largest alignment & {\bf 8624} & 8501 & 8412 & 8518 \\ \hline
Total aligned length & {\bf 8624} & 8501 & 8412 & 8518 \\ \hline
NA50 & {\bf 8624} & 8501 & 8412 & 8518 \\ \hline
NGA50 & {\bf 8624} & 8501 & 8412 & 8518 \\ \hline
NA75 & {\bf 8624} & 8501 & 8412 & 8518 \\ \hline
NGA75 & {\bf 8624} & 8501 & 8412 & 8518 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
