\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8630 & 8630 & 8630 & 0 & 0 & 0 & 1 & 8630 & 8630 & 8654 & 40.58 & 40.52 & 8630 & 8630 & 8630 & 8630 & 8630.0 & 8606.1 & 1 & 1 & 1 & 1 & 15395 & 8800 & 8800 & 113.76 & 113.75 & 113.21 & 113.19 & 506 & 506 & 99.97 & 99.97 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 312.86 & 81.11 & 8630 & 8630 & 8630 & 8630 & 8630 & 8630 & 8630.0 & 8606.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8627 & 8627 & 8627 & 0 & 0 & 0 & 1 & 8627 & 8627 & 8654 & 40.58 & 40.52 & 8627 & 8627 & 8627 & 8627 & 8627.0 & 8600.1 & 1 & 1 & 1 & 1 & 15384 & 8800 & 8800 & 113.63 & 113.75 & 112.86 & 113.19 & 505 & 506 & 99.9 & 99.97 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.888 & 1.000 & 0.00 & 374.97 & 82.02 & 8534 & 8534 & 8534 & 8534 & 8534 & 8534 & 8442.0 & 8415.7 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8314 & 8314 & 8314 & 0 & 0 & 0 & 1 & 8314 & 8314 & 8654 & 39.72 & 40.52 & 8314 & 8314 & 8314 & 8314 & 8314.0 & 7987.4 & 1 & 1 & 1 & 1 & 15081 & 8800 & 8800 & 98.21 & 113.75 & 81.86 & 113.19 & 439 & 506 & 91.48 & 99.97 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 88.134 & 1.000 & 8539.81 & 328.69 & 65.74 & 7606 & 7606 & 7606 & 7606 & 7606 & - & 6958.3 & 6684.9 & 1 & 1 & 1 & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8635 & 8635 & 8635 & 0 & 0 & 0 & 1 & 8635 & 8635 & 8654 & 41.11 & 40.52 & 8635 & 8635 & 8635 & 8635 & 8635.0 & 8616.0 & 1 & 1 & 1 & 1 & 15303 & 8800 & 8800 & 107.56 & 113.75 & 101.33 & 113.19 & 470 & 506 & 99.21 & 99.97 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.065 & 1.001 & 0.00 & 2195.73 & 543.03 & 8471 & 8471 & 8471 & 8471 & 8471 & 8471 & 8310.1 & 8291.9 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
