\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
Total length ($\geq$ 1000 bp) & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
Total length ($\geq$ 5000 bp) & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
Total length & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
Reference length & 8629 & 8629 & 8629 & 8629 \\ \hline
GC (\%) & - & 40.84 & 40.78 & 41.04 \\ \hline
Reference GC (\%) & 40.75 & 40.75 & 40.75 & 40.75 \\ \hline
N50 & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
NG50 & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
N90 & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
NG90 & 8624 & 8614 & 8319 & {\bf 8638} \\ \hline
auN & 8624.0 & 8614.0 & 8319.0 & {\bf 8638.0} \\ \hline
auNG & 8619.0 & 8599.0 & 8020.1 & {\bf 8647.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 15409 & 15377 & 15315 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 0.0 & {\bf 113.3} & 112.29 & 109.7 \\ \hline
Reference mapped (\%) & 113.49 & 113.49 & 113.49 & 113.49 \\ \hline
Properly paired (\%) & 0.0 & {\bf 112.38} & 110.13 & 105.01 \\ \hline
Reference properly paired (\%) & 112.86 & 112.86 & 112.86 & 112.86 \\ \hline
Avg. coverage depth & 0 & 506 & {\bf 516} & 481 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 0 & 0 \\ \hline
\# structural variations & - & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 1 + 0 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & 8624 & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & - & 98.932 & 96.715 & 98.816 \\ \hline
Duplication ratio & - & 1.000 & 0.999 & 1.003 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & 84.14 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 0.00 & 60.10 & 1323.81 \\ \hline
\# indels per 100 kbp & - & 46.94 & 84.14 & 644.33 \\ \hline
Largest alignment & - & 8522 & 8319 & 8536 \\ \hline
Total aligned length & - & 8522 & 8319 & 8536 \\ \hline
NA50 & - & 8522 & 8319 & 8536 \\ \hline
NGA50 & - & 8522 & 8319 & 8536 \\ \hline
NA90 & - & 8522 & 8319 & 8536 \\ \hline
NGA90 & - & 8522 & 8319 & 8536 \\ \hline
auNA & - & 8431.0 & 8319.0 & 8435.2 \\ \hline
auNGA & - & 8416.3 & 8020.1 & 8444.0 \\ \hline
LA50 & - & 1 & 1 & 1 \\ \hline
LGA50 & - & 1 & 1 & 1 \\ \hline
LA90 & - & 1 & 1 & 1 \\ \hline
LGA90 & - & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
