\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{61}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N75 & NG75 & L50 & LG50 & L75 & LG75 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA75 & NGA75 & LA50 & LGA50 & LA75 & LGA75 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8663 & 8663 & 8663 & 0 & 0 & 0 & 1 & 8663 & 8663 & 8671 & 40.85 & 40.77 & 8663 & 8663 & 8663 & 8663 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 98.69 & 99.64 & 97.38 & 99.28 & 497 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.004 & 0.00 & 57.93 & 127.45 & 8663 & 8663 & 8663 & 8663 & 8663 & 8663 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8671 & 40.81 & 40.77 & 8638 & 8638 & 8638 & 8638 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 99.55 & 99.64 & 99.1 & 99.28 & 506 & 506 & 99.95 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.687 & 1.004 & 0.00 & 209.21 & 81.36 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8680 & 8680 & 8680 & 0 & 0 & 0 & 1 & 8680 & 8680 & 8671 & 40.83 & 40.77 & 8680 & 8680 & 8680 & 8680 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 94.92 & 99.64 & 89.94 & 99.28 & 477 & 506 & 97.22 & 100.0 & 0 & 0 & 0 & 0 & 0 & 2 & 0 & 0 + 0 part & 0 & 97.115 & 1.036 & 2776.50 & 334.05 & 95.44 & 8383 & 8383 & 8383 & 8383 & 8383 & 8383 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8635 & 8635 & 8635 & 0 & 0 & 0 & 1 & 8635 & 8635 & 8671 & 41.03 & 40.77 & 8635 & 8635 & 8635 & 8635 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 94.81 & 99.64 & 90.17 & 99.28 & 476 & 506 & 99.24 & 100.0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 + 0 part & 0 & 94.079 & 1.063 & 0.00 & 2672.41 & 197.04 & 8117 & 8117 & 8117 & 8117 & 8117 & 8117 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
