\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8583 & 8583 & 8583 & 0 & 0 & 0 & 1 & 8583 & 8583 & 8610 & 40.35 & 40.37 & 8583 & 8583 & 8583 & 8583 & 8583.0 & 8556.1 & 1 & 1 & 1 & 1 & 15623 & 8800 & 8800 & 111.24 & 111.2 & 109.83 & 109.78 & 504 & 504 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 0.00 & 46.60 & 81.56 & 8583 & 8583 & 8583 & 8583 & 8583 & 8583 & 8583.0 & 8556.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8574 & 8574 & 8574 & 0 & 0 & 0 & 1 & 8574 & 8574 & 8610 & 40.32 & 40.37 & 8574 & 8574 & 8574 & 8574 & 8574.0 & 8538.2 & 1 & 1 & 1 & 1 & 15611 & 8800 & 8800 & 110.75 & 111.2 & 108.76 & 109.78 & 501 & 504 & 99.86 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.917 & 0.998 & 0.00 & 165.07 & 47.16 & 8481 & 8481 & 8481 & 8481 & 8481 & 8481 & 8389.0 & 8353.9 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8165 & 8165 & 8165 & 0 & 0 & 0 & 1 & 8165 & 8165 & 8610 & 39.23 & 40.37 & 8165 & 8165 & 8165 & 8165 & 8165.0 & 7743.0 & 1 & 1 & 1 & 1 & 15324 & 8800 & 8800 & 73.55 & 111.2 & 49.57 & 109.78 & 334 & 504 & 78.11 & 100.0 & 0 & 0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 + 0 part & 0 & 75.276 & 1.000 & 22731.17 & 525.58 & 30.92 & 6469 & 6469 & 6469 & 6469 & - & - & 5125.3 & 4860.4 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8596 & 8596 & 8596 & 0 & 0 & 0 & 1 & 8596 & 8596 & 8610 & 40.73 & 40.37 & 8596 & 8596 & 8596 & 8596 & 8596.0 & 8582.0 & 1 & 1 & 1 & 1 & 15388 & 8800 & 8800 & 101.88 & 111.2 & 91.62 & 109.78 & 448 & 504 & 99.14 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.452 & 1.001 & 0.00 & 2185.99 & 602.62 & 8463 & 8463 & 8463 & 8463 & 8463 & 8463 & 8332.1 & 8318.5 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
