\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
Total length ($\geq$ 1000 bp) & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
Total length ($\geq$ 5000 bp) & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
Total length & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
Reference length & 8669 & 8669 & 8669 & 8669 \\ \hline
GC (\%) & 40.43 & 40.46 & 40.20 & 41.01 \\ \hline
Reference GC (\%) & 40.44 & 40.44 & 40.44 & 40.44 \\ \hline
N50 & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
NG50 & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
N75 & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
NG75 & 8645 & {\bf 8648} & 8440 & 8623 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 99.6 & {\bf 99.61} & 78.61 & 90.23 \\ \hline
Reference mapped (\%) & 99.6 & 99.6 & 99.6 & 99.6 \\ \hline
Properly paired (\%) & 99.2 & {\bf 99.22} & 62.81 & 81.84 \\ \hline
Reference properly paired (\%) & 99.2 & 99.2 & 99.2 & 99.2 \\ \hline
Avg. coverage depth & {\bf 505} & {\bf 505} & 399 & 453 \\ \hline
Reference avg. coverage depth & 505 & 505 & 505 & 505 \\ \hline
Coverage $\geq$ 1x (\%) & 99.99 & 99.99 & 88.09 & 99.26 \\ \hline
Reference coverage $\geq$ 1x (\%) & 99.99 & 99.99 & 99.99 & 99.99 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 4 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & {\bf 0} & 1109 \\ \hline
Genome fraction (\%) & {\bf 99.977} & 99.942 & 87.056 & 86.998 \\ \hline
Duplication ratio & 1.000 & 1.001 & 1.121 & {\bf 0.999} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 13933.65 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 80.99} & 289.35 & 2298.70 & 2632.63 \\ \hline
\# indels per 100 kbp & 92.56 & {\bf 81.02} & 119.59 & 172.85 \\ \hline
Largest alignment & {\bf 8643} & {\bf 8643} & 7521 & 7514 \\ \hline
Total aligned length & {\bf 8643} & {\bf 8643} & 7521 & 7514 \\ \hline
NA50 & {\bf 8643} & {\bf 8643} & 7521 & 7514 \\ \hline
NGA50 & {\bf 8643} & {\bf 8643} & 7521 & 7514 \\ \hline
NA75 & {\bf 8643} & {\bf 8643} & 7521 & 7514 \\ \hline
NGA75 & {\bf 8643} & {\bf 8643} & 7521 & 7514 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
