\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{61}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N75 & NG75 & L50 & LG50 & L75 & LG75 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA75 & NGA75 & LA50 & LGA50 & LA75 & LGA75 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8645 & 8645 & 8645 & 0 & 0 & 0 & 1 & 8645 & 8645 & 8669 & 40.43 & 40.44 & 8645 & 8645 & 8645 & 8645 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 99.6 & 99.6 & 99.2 & 99.2 & 505 & 505 & 99.99 & 99.99 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.977 & 1.000 & 0.00 & 80.99 & 92.56 & 8643 & 8643 & 8643 & 8643 & 8643 & 8643 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8648 & 8648 & 8648 & 0 & 0 & 0 & 1 & 8648 & 8648 & 8669 & 40.46 & 40.44 & 8648 & 8648 & 8648 & 8648 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 99.61 & 99.6 & 99.22 & 99.2 & 505 & 505 & 99.99 & 99.99 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.942 & 1.001 & 0.00 & 289.35 & 81.02 & 8643 & 8643 & 8643 & 8643 & 8643 & 8643 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8440 & 8440 & 8440 & 0 & 0 & 0 & 1 & 8440 & 8440 & 8669 & 40.20 & 40.44 & 8440 & 8440 & 8440 & 8440 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 78.61 & 99.6 & 62.81 & 99.2 & 399 & 505 & 88.09 & 99.99 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 + 0 part & 0 & 87.056 & 1.121 & 13933.65 & 2298.70 & 119.59 & 7521 & 7521 & 7521 & 7521 & 7521 & 7521 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8623 & 8623 & 8623 & 0 & 0 & 0 & 1 & 8623 & 8623 & 8669 & 41.01 & 40.44 & 8623 & 8623 & 8623 & 8623 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 90.23 & 99.6 & 81.84 & 99.2 & 453 & 505 & 99.26 & 99.99 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 + 1 part & 1109 & 86.998 & 0.999 & 0.00 & 2632.63 & 172.85 & 7514 & 7514 & 7514 & 7514 & 7514 & 7514 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
