\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
Total length ($\geq$ 1000 bp) & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
Total length ($\geq$ 5000 bp) & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
Total length & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
Reference length & 8722 & 8722 & 8722 & 8722 \\ \hline
GC (\%) & 40.27 & 40.27 & 40.22 & 40.27 \\ \hline
Reference GC (\%) & 40.28 & 40.28 & 40.28 & 40.28 \\ \hline
N50 & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
NG50 & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
N90 & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
NG90 & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
auN & 8687.0 & 8689.0 & 8692.0 & {\bf 8716.0} \\ \hline
auNG & 8652.1 & 8656.1 & 8662.1 & {\bf 8710.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 114872 & 114867 & 117846 & 115137 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & 311.65 & {\bf 311.72} & 299.26 & 310.93 \\ \hline
Reference mapped (\%) & 311.03 & 311.03 & 311.03 & 311.03 \\ \hline
Properly paired (\%) & 297.7 & {\bf 297.81} & 281.06 & 297.1 \\ \hline
Reference properly paired (\%) & 297.17 & 297.17 & 297.17 & 297.17 \\ \hline
Avg. coverage depth & {\bf 10262} & 10261 & 10069 & 10252 \\ \hline
Reference avg. coverage depth & 10252 & 10252 & 10252 & 10252 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 100.000 & 100.000 & 100.000 & 100.000 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & {\bf 1.000} & 1.003 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 1449.61 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 103.58 & 816.84 & 34.42 \\ \hline
\# indels per 100 kbp & 126.63 & 103.58 & 92.04 & {\bf 22.95} \\ \hline
Largest alignment & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
Total aligned length & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
NA50 & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
NGA50 & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
NA90 & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
NGA90 & 8687 & 8689 & 8692 & {\bf 8716} \\ \hline
auNA & 8687.0 & 8689.0 & 8692.0 & {\bf 8716.0} \\ \hline
auNGA & 8652.1 & 8656.1 & 8662.1 & {\bf 8710.0} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
