\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8621 & 8621 & 8621 & 0 & 0 & 0 & 1 & 8621 & 8621 & 8669 & 40.61 & 40.62 & 8621 & 8621 & 8621 & 8621 & 8621.0 & 8573.3 & 1 & 1 & 1 & 1 & 15618 & 8800 & 8800 & 112.11 & 111.99 & 111.54 & 111.41 & 506 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 23.20 & 127.60 & 8621 & 8621 & 8621 & 8621 & 8621 & 8621 & 8621.0 & 8573.3 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8621 & 8621 & 8621 & 0 & 0 & 0 & 1 & 8621 & 8621 & 8669 & 40.51 & 40.62 & 8621 & 8621 & 8621 & 8621 & 8621.0 & 8573.3 & 1 & 1 & 1 & 1 & 15617 & 8800 & 8800 & 112.08 & 111.99 & 111.46 & 111.41 & 506 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.884 & 1.000 & 0.00 & 325.17 & 127.74 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 8601.0 & 8553.4 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8669 & 40.65 & 40.62 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8559.3 & 1 & 1 & 1 & 1 & 15155 & 8800 & 8800 & 94.37 & 111.99 & 76.21 & 111.41 & 406 & 505 & 86.75 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 86.643 & 1.001 & 15172.97 & 1485.35 & 120.43 & 7473 & 7473 & 7473 & 7473 & - & - & 6483.1 & 6442.0 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8617 & 8617 & 8617 & 0 & 0 & 0 & 1 & 8617 & 8617 & 8669 & 40.76 & 40.62 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8565.3 & 1 & 1 & 1 & 1 & 15436 & 8800 & 8800 & 103.36 & 111.99 & 94.9 & 111.41 & 454 & 505 & 99.23 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 1 part & 826 & 90.263 & 1.002 & 0.00 & 2990.63 & 654.60 & 7791 & 7791 & 7791 & 7791 & 7791 & - & 7044.2 & 7001.9 & 1 & 1 & 1 & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
