\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
Total length ($\geq$ 1000 bp) & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
Total length ($\geq$ 5000 bp) & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
Total length & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
Reference length & 8628 & 8628 & 8628 & 8628 \\ \hline
GC (\%) & 40.33 & 40.25 & 40.54 & 40.64 \\ \hline
Reference GC (\%) & 40.27 & 40.27 & 40.27 & 40.27 \\ \hline
N50 & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
NG50 & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
N75 & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
NG75 & 8610 & 8582 & {\bf 8616} & 8615 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 98.72} & 98.27 & 85.12 & 88.24 \\ \hline
Reference mapped (\%) & 98.71 & 98.71 & 98.71 & 98.71 \\ \hline
Properly paired (\%) & {\bf 97.43} & 96.53 & 72.25 & 78.25 \\ \hline
Reference properly paired (\%) & 97.42 & 97.42 & 97.42 & 97.42 \\ \hline
Avg. coverage depth & {\bf 503} & 500 & 428 & 443 \\ \hline
Reference avg. coverage depth & 502 & 502 & 502 & 502 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 91.71 & 99.07 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 3 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & {\bf 0} & 2429 \\ \hline
Genome fraction (\%) & {\bf 99.965} & 99.349 & 91.739 & 71.837 \\ \hline
Duplication ratio & 1.001 & 1.004 & 1.091 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 10666.20 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 162.72} & 608.12 & 595.24 & 1310.04 \\ \hline
\# indels per 100 kbp & 58.11 & 81.86 & 50.66 & {\bf 48.52} \\ \hline
Largest alignment & {\bf 8607} & 8527 & 7899 & 6186 \\ \hline
Total aligned length & {\bf 8607} & 8527 & 7899 & 6186 \\ \hline
NA50 & {\bf 8607} & 8527 & 7899 & 6186 \\ \hline
NGA50 & {\bf 8607} & 8527 & 7899 & 6186 \\ \hline
NA75 & {\bf 8607} & 8527 & 7899 & - \\ \hline
NGA75 & {\bf 8607} & 8527 & 7899 & - \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA75 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
