\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8713} & 8643 & 4809 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8713} & 8643 & 4809 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8713} & 8643 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8713} & 8643 & 4809 & 8625 \\ \hline
Total length & {\bf 8713} & 8643 & 4809 & 8625 \\ \hline
Reference length & 8750 & 8750 & 8750 & 8750 \\ \hline
GC (\%) & 40.64 & 41.28 & 40.08 & 41.39 \\ \hline
Reference GC (\%) & 40.64 & 40.64 & 40.64 & 40.64 \\ \hline
N50 & {\bf 8713} & 8643 & 4809 & 8625 \\ \hline
NG50 & {\bf 8713} & 8643 & 4809 & 8625 \\ \hline
N75 & {\bf 8713} & 8643 & 4809 & 8625 \\ \hline
NG75 & {\bf 8713} & 8643 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 380000 & 380000 & 380000 & 380000 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 94.37} & 52.17 & 38.32 & 21.68 \\ \hline
Reference mapped (\%) & 94.36 & 94.36 & 94.36 & 94.36 \\ \hline
Properly paired (\%) & {\bf 90.39} & 37.49 & 20.35 & 8.68 \\ \hline
Reference properly paired (\%) & 90.39 & 90.39 & 90.39 & 90.39 \\ \hline
Avg. coverage depth & {\bf 10235} & 5610 & 7369 & 2258 \\ \hline
Reference avg. coverage depth & 10230 & 10230 & 10230 & 10230 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 61.6 & 80.99 & 56.39 \\ \hline
Reference coverage $\geq$ 1x (\%) & 99.99 & 99.99 & 99.99 & 99.99 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 5 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 3 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 3185 & {\bf 0} & 7129 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 62.577 & 44.629 & 17.156 \\ \hline
Duplication ratio & {\bf 1.000} & 1.001 & 1.237 & 1.001 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 19837.80 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 421.79 & 1182.82 & 2876.25 \\ \hline
\# indels per 100 kbp & 172.14 & {\bf 73.35} & 77.14 & 535.12 \\ \hline
Largest alignment & {\bf 8712} & 5458 & 3893 & 733 \\ \hline
Total aligned length & {\bf 8712} & 5458 & 3893 & 1496 \\ \hline
NA50 & {\bf 8712} & 5458 & 3893 & - \\ \hline
NGA50 & {\bf 8712} & 5458 & - & - \\ \hline
NA75 & {\bf 8712} & - & 3893 & - \\ \hline
NGA75 & {\bf 8712} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & {\bf 1} & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
