\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
Total length ($\geq$ 1000 bp) & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
Total length ($\geq$ 5000 bp) & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
Total length & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
Reference length & 8653 & 8653 & 8653 & 8653 \\ \hline
GC (\%) & 40.39 & 40.48 & 39.79 & 40.82 \\ \hline
Reference GC (\%) & 40.41 & 40.41 & 40.41 & 40.41 \\ \hline
N50 & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
NG50 & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
N75 & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
NG75 & 8626 & 8628 & 8463 & {\bf 8643} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 350000 & 350000 & 350000 & 350000 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 99.58} & 99.46 & 95.22 & 95.94 \\ \hline
Reference mapped (\%) & 99.58 & 99.58 & 99.58 & 99.58 \\ \hline
Properly paired (\%) & {\bf 99.16} & 98.91 & 90.45 & 92.46 \\ \hline
Reference properly paired (\%) & 99.16 & 99.16 & 99.16 & 99.16 \\ \hline
Avg. coverage depth & {\bf 10077} & 10065 & 9783 & 9606 \\ \hline
Reference avg. coverage depth & 10068 & 10068 & 10068 & 10068 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.95 & 97.84 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.281 & 96.058 & 98.690 \\ \hline
Duplication ratio & {\bf 1.000} & 1.008 & 1.022 & 1.016 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 2185.99 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 11.60} & 315.35 & 2317.72 & 1856.42 \\ \hline
\# indels per 100 kbp & 92.76 & {\bf 81.76} & 108.64 & 458.23 \\ \hline
Largest alignment & {\bf 8626} & 8567 & 8280 & 8542 \\ \hline
Total aligned length & {\bf 8626} & 8567 & 8280 & 8542 \\ \hline
NA50 & {\bf 8626} & 8567 & 8280 & 8542 \\ \hline
NGA50 & {\bf 8626} & 8567 & 8280 & 8542 \\ \hline
NA75 & {\bf 8626} & 8567 & 8280 & 8542 \\ \hline
NGA75 & {\bf 8626} & 8567 & 8280 & 8542 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
