\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8641 & 40.27 & 40.25 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8587.1 & 1 & 1 & 1 & 1 & 15545 & 8800 & 8800 & 112.6 & 112.61 & 111.98 & 111.99 & 506 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.965 & 1.000 & 0.00 & 34.84 & 92.90 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 8608.0 & 8581.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8641 & 40.45 & 40.25 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8587.1 & 1 & 1 & 1 & 1 & 15528 & 8800 & 8800 & 112.42 & 112.61 & 111.5 & 111.99 & 505 & 506 & 99.88 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.723 & 1.000 & 0.00 & 246.94 & 82.31 & 8504 & 8504 & 8504 & 8504 & 8504 & 8504 & 8395.4 & 8369.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8580 & 8580 & 8580 & 0 & 0 & 0 & 1 & 8580 & 8580 & 8641 & 39.73 & 40.25 & 8580 & 8580 & 8580 & 8580 & 8580.0 & 8519.4 & 1 & 1 & 1 & 1 & 15083 & 8800 & 8800 & 80.35 & 112.61 & 52.89 & 111.99 & 344 & 506 & 78.02 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 1 part & 2235 & 52.792 & 1.000 & 22610.72 & 1627.45 & 131.96 & 4547 & 4547 & 4547 & 4547 & - & - & 2409.7 & 2392.7 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8641 & 40.88 & 40.25 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8587.1 & 1 & 1 & 1 & 1 & 15330 & 8800 & 8800 & 103.44 & 112.61 & 93.66 & 111.99 & 453 & 506 & 98.6 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 97.980 & 1.000 & 0.00 & 2997.63 & 699.05 & 8440 & 8440 & 8440 & 8440 & 8440 & 8440 & 8269.5 & 8243.7 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
