\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 1085} & {\bf 1085} & 299 & {\bf 1085} \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 1085} & {\bf 1085} & 0 & {\bf 1085} \\ \hline
Total length ($\geq$ 5000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 1085} & {\bf 1085} & 299 & {\bf 1085} \\ \hline
Total length & {\bf 1085} & {\bf 1085} & 299 & {\bf 1085} \\ \hline
Reference length & 1085 & 1085 & 1085 & 1085 \\ \hline
GC (\%) & 44.42 & 43.23 & 42.14 & 42.67 \\ \hline
Reference GC (\%) & 44.45 & 44.45 & 44.45 & 44.45 \\ \hline
N50 & {\bf 1085} & {\bf 1085} & 299 & {\bf 1085} \\ \hline
NG50 & {\bf 1085} & {\bf 1085} & - & {\bf 1085} \\ \hline
N75 & {\bf 1085} & {\bf 1085} & 299 & {\bf 1085} \\ \hline
NG75 & {\bf 1085} & {\bf 1085} & - & {\bf 1085} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 380000 & 380000 & 380000 & 380000 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 10.94} & 6.58 & 2.19 & 2.92 \\ \hline
Reference mapped (\%) & 10.91 & 10.91 & 10.91 & 10.91 \\ \hline
Properly paired (\%) & {\bf 5.91} & 1.58 & 0.0 & 0.26 \\ \hline
Reference properly paired (\%) & 5.87 & 5.87 & 5.87 & 5.87 \\ \hline
Avg. coverage depth & {\bf 9359} & 5556 & 6176 & 2321 \\ \hline
Reference avg. coverage depth & 9355 & 9355 & 9355 & 9355 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 64.52 & 100.0 & 64.52 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# misassembled contigs & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
Misassembled contigs length & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# local misassemblies & {\bf 0} & 1 & - & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 1 + 0 part & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & 299 & 878 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 79.667 & - & 19.131 \\ \hline
Duplication ratio & 1.003 & 1.259 & - & {\bf 1.000} \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & {\bf 0.00} & - & {\bf 0.00} \\ \hline
\# indels per 100 kbp & 92.42 & 116.01 & - & {\bf 0.00} \\ \hline
Largest alignment & {\bf 1082} & 862 & - & 207 \\ \hline
Total aligned length & {\bf 1082} & 862 & - & 207 \\ \hline
NA50 & {\bf 1082} & 862 & - & - \\ \hline
NGA50 & {\bf 1082} & 862 & - & - \\ \hline
NA75 & {\bf 1082} & 862 & - & - \\ \hline
NGA75 & {\bf 1082} & 862 & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & {\bf 1} & - & - \\ \hline
LGA75 & {\bf 1} & {\bf 1} & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
