\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8707 & 40.55 & 40.51 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8665.1 & 1 & 1 & 1 & 1 & 101433 & 190000 & 190000 & 353.27 & 352.81 & 337.51 & 337.08 & 10282 & 10273 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 57.56 & 46.05 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8665.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8707 & 41.22 & 40.51 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8569.5 & 1 & 1 & 1 & 1 & 213096 & 190000 & 190000 & 95.12 & 352.81 & 68.61 & 337.08 & 5714 & 10273 & 63.87 & 100.0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 & 0 & 0 + 1 part & 2809 & 67.104 & 1.000 & 0.00 & 137.24 & 17.16 & 5829 & 5829 & 5829 & 5829 & - & - & 3933.5 & 3902.3 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 6782 & 6782 & 6782 & 0 & 0 & 0 & 1 & 6782 & 6782 & 8707 & 40.61 & 40.51 & 6782 & 6782 & 6782 & - & 6782.0 & 5282.6 & 1 & 1 & 1 & - & 216176 & 190000 & 190000 & 92.62 & 352.81 & 67.45 & 337.08 & 7212 & 10273 & 77.99 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 60.887 & 1.000 & 22014.16 & 264.70 & 37.81 & 5289 & 5289 & 5289 & 5289 & - & - & 4124.7 & 3212.8 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8707 & 41.27 & 40.51 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8541.8 & 1 & 1 & 1 & 1 & 309209 & 190000 & 190000 & 26.73 & 352.81 & 11.32 & 337.08 & 2295 & 10273 & 59.89 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7284 & 15.429 & 1.000 & 0.00 & 298.51 & 373.13 & 862 & 1340 & - & - & - & - & 99.6 & 98.7 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
