\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{61}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N75 & NG75 & L50 & LG50 & L75 & LG75 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA75 & NGA75 & LA50 & LGA50 & LA75 & LGA75 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8610 & 8610 & 8610 & 0 & 0 & 0 & 1 & 8610 & 8610 & 8630 & 40.52 & 40.43 & 8610 & 8610 & 8610 & 8610 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 99.38 & 99.44 & 98.76 & 98.89 & 504 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 255.75 & 232.50 & 8610 & 8610 & 8610 & 8610 & 8610 & 8610 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8603 & 8603 & 8603 & 0 & 0 & 0 & 1 & 8603 & 8603 & 8630 & 40.51 & 40.43 & 8603 & 8603 & 8603 & 8603 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 99.47 & 99.44 & 98.94 & 98.89 & 507 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 279.00 & 69.75 & 8603 & 8603 & 8603 & 8603 & 8603 & 8603 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8603 & 8603 & 8603 & 0 & 0 & 0 & 1 & 8603 & 8603 & 8630 & 40.73 & 40.43 & 8603 & 8603 & 8603 & 8603 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 79.97 & 99.44 & 61.76 & 98.89 & 401 & 506 & 86.57 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 + 0 part & 0 & 86.573 & 1.155 & 13902.13 & 483.42 & 80.57 & 7448 & 7448 & 7448 & 7448 & 7448 & 7448 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8612 & 8612 & 8612 & 0 & 0 & 0 & 1 & 8612 & 8612 & 8630 & 40.43 & 40.43 & 8612 & 8612 & 8612 & 8612 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 99.43 & 99.44 & 98.86 & 98.89 & 506 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 279.00 & 81.38 & 8612 & 8612 & 8612 & 8612 & 8612 & 8612 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
