\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{61}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N75 & NG75 & L50 & LG50 & L75 & LG75 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA75 & NGA75 & LA50 & LGA50 & LA75 & LGA75 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8641 & 40.09 & 40.10 & 8605 & 8605 & 8605 & 8605 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 98.17 & 98.62 & 96.55 & 97.25 & 499 & 501 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 209.21 & 116.23 & 8588 & 8588 & 8588 & 8588 & 8588 & 8588 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8580 & 8580 & 8580 & 0 & 0 & 0 & 1 & 8580 & 8580 & 8641 & 40.08 & 40.10 & 8580 & 8580 & 8580 & 8580 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 98.15 & 98.62 & 96.3 & 97.25 & 499 & 501 & 99.97 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.338 & 1.004 & 0.00 & 234.00 & 93.60 & 8524 & 8524 & 8524 & 8524 & 8524 & 8524 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8641 & 39.70 & 40.10 & 8605 & 8605 & 8605 & 8605 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 94.58 & 98.62 & 89.16 & 97.25 & 479 & 501 & 97.58 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 + 0 part & 0 & 97.583 & 1.025 & 3858.22 & 273.94 & 71.46 & 8397 & 8397 & 8397 & 8397 & 8397 & 8397 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8609 & 8609 & 8609 & 0 & 0 & 0 & 1 & 8609 & 8609 & 8641 & 40.41 & 40.10 & 8609 & 8609 & 8609 & 8609 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 88.29 & 98.62 & 78.61 & 97.25 & 443 & 501 & 98.21 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 97.896 & 1.022 & 0.00 & 2505.05 & 486.76 & 8434 & 8434 & 8434 & 8434 & 8434 & 8434 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
