\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8684 & 8684 & 8684 & 0 & 0 & 0 & 1 & 8684 & 8684 & 8707 & 40.73 & 40.71 & 8684 & 8684 & 8684 & 8684 & 8684.0 & 8661.1 & 1 & 1 & 1 & 1 & 109184 & 190000 & 190000 & 328.06 & 327.35 & 313.57 & 312.92 & 10277 & 10269 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 11.52 & 92.12 & 8684 & 8684 & 8684 & 8684 & 8684 & 8684 & 8684.0 & 8661.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8707 & 41.31 & 40.71 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8569.5 & 1 & 1 & 1 & 1 & 214037 & 190000 & 190000 & 94.44 & 327.35 & 68.41 & 312.92 & 5697 & 10269 & 63.51 & 100.0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 & 0 & 0 + 1 part & 2839 & 66.770 & 1.000 & 0.00 & 34.49 & 68.98 & 5799 & 5799 & 5799 & 5799 & - & - & 3893.1 & 3862.2 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7113 & 7113 & 7113 & 0 & 0 & 0 & 1 & 7113 & 7113 & 8707 & 40.71 & 40.71 & 7113 & 7113 & 7113 & - & 7113.0 & 5810.8 & 1 & 1 & 1 & - & 218027 & 190000 & 190000 & 92.74 & 327.35 & 74.82 & 312.92 & 6959 & 10269 & 73.4 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 61.670 & 1.001 & 25474.48 & 373.20 & 74.64 & 5359 & 5359 & 5359 & 5359 & - & - & 4037.5 & 3298.4 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8707 & 41.36 & 40.71 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8541.8 & 1 & 1 & 1 & 1 & 304167 & 190000 & 190000 & 29.02 & 327.35 & 12.81 & 312.92 & 2444 & 10269 & 57.37 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 6997 & 18.733 & 1.000 & 0.00 & 1044.87 & 614.63 & 862 & 1627 & - & - & - & - & 127.1 & 125.9 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
