\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8613 & 8613 & 8613 & 0 & 0 & 0 & 1 & 8613 & 8613 & 8662 & 40.64 & 40.58 & 8613 & 8613 & 8613 & 8613 & 8613.0 & 8564.3 & 1 & 1 & 1 & 1 & 15607 & 8800 & 8800 & 112.15 & 112.08 & 111.53 & 111.44 & 506 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 34.83 & 127.71 & 8613 & 8613 & 8613 & 8613 & 8613 & 8613 & 8613.0 & 8564.3 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8617 & 8617 & 8617 & 0 & 0 & 0 & 1 & 8617 & 8617 & 8662 & 40.72 & 40.58 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8572.2 & 1 & 1 & 1 & 1 & 15600 & 8800 & 8800 & 111.97 & 112.08 & 111.13 & 111.44 & 505 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.932 & 1.000 & 0.00 & 140.76 & 93.84 & 8525 & 8525 & 8525 & 8525 & 8525 & 8525 & 8434.0 & 8390.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8602 & 8602 & 8602 & 0 & 0 & 0 & 1 & 8602 & 8602 & 8662 & 40.80 & 40.58 & 8602 & 8602 & 8602 & 8602 & 8602.0 & 8542.4 & 1 & 1 & 1 & 1 & 15019 & 8800 & 8800 & 95.65 & 112.08 & 74.11 & 111.44 & 410 & 505 & 89.29 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 89.068 & 1.000 & 12810.97 & 482.08 & 78.18 & 7675 & 7675 & 7675 & 7675 & - & - & 6847.9 & 6800.5 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8617 & 8617 & 8617 & 0 & 0 & 0 & 1 & 8617 & 8617 & 8662 & 40.92 & 40.58 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8572.2 & 1 & 1 & 1 & 1 & 15375 & 8800 & 8800 & 105.2 & 112.08 & 97.31 & 111.44 & 461 & 505 & 99.13 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.016 & 1.000 & 0.00 & 2723.18 & 639.36 & 8446 & 8446 & 8446 & 8446 & 8446 & 8446 & 8278.4 & 8235.4 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
