\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
Total length ($\geq$ 1000 bp) & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
Total length ($\geq$ 5000 bp) & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
Total length & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
Reference length & 8610 & 8610 & 8610 & 8610 \\ \hline
GC (\%) & 40.21 & 40.24 & 40.24 & 40.58 \\ \hline
Reference GC (\%) & 40.19 & 40.19 & 40.19 & 40.19 \\ \hline
N50 & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
NG50 & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
N75 & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
NG75 & 8595 & 8594 & 8589 & {\bf 8605} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 98.84} & 98.73 & 97.27 & 91.25 \\ \hline
Reference mapped (\%) & 98.84 & 98.84 & 98.84 & 98.84 \\ \hline
Properly paired (\%) & {\bf 97.68} & 97.47 & 94.55 & 83.84 \\ \hline
Reference properly paired (\%) & 97.68 & 97.68 & 97.68 & 97.68 \\ \hline
Avg. coverage depth & {\bf 504} & {\bf 504} & 497 & 459 \\ \hline
Reference avg. coverage depth & 504 & 504 & 504 & 504 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 97.25 & 99.3 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.383 & 98.790 & 97.987 \\ \hline
Duplication ratio & {\bf 1.000} & 1.006 & 1.012 & 1.022 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 1210.85 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 162.90} & 245.87 & 200.24 & 2018.76 \\ \hline
\# indels per 100 kbp & {\bf 46.54} & 46.83 & 58.89 & 498.75 \\ \hline
Largest alignment & {\bf 8595} & 8542 & 8485 & 8434 \\ \hline
Total aligned length & {\bf 8595} & 8542 & 8485 & 8434 \\ \hline
NA50 & {\bf 8595} & 8542 & 8485 & 8434 \\ \hline
NGA50 & {\bf 8595} & 8542 & 8485 & 8434 \\ \hline
NA75 & {\bf 8595} & 8542 & 8485 & 8434 \\ \hline
NGA75 & {\bf 8595} & 8542 & 8485 & 8434 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
