\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8627 & 8627 & 8627 & 0 & 0 & 0 & 1 & 8627 & 8627 & 8669 & 40.38 & 40.39 & 8627 & 8627 & 8627 & 8627 & 8627.0 & 8585.2 & 1 & 1 & 1 & 1 & 97991 & 175000 & 175000 & 355.51 & 354.02 & 353.85 & 352.36 & 10073 & 10061 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 139.10 & 8627 & 8627 & 8627 & 8627 & 8627 & 8627 & 8627.0 & 8585.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8627 & 8627 & 8627 & 0 & 0 & 0 & 1 & 8627 & 8627 & 8669 & 40.40 & 40.39 & 8627 & 8627 & 8627 & 8627 & 8627.0 & 8585.2 & 1 & 1 & 1 & 1 & 98152 & 175000 & 175000 & 354.59 & 354.02 & 352.59 & 352.36 & 10064 & 10061 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.386 & 1.000 & 0.00 & 23.33 & 139.96 & 8574 & 8574 & 8574 & 8574 & 8574 & 8574 & 8521.3 & 8480.0 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8621 & 8621 & 8621 & 0 & 0 & 0 & 1 & 8621 & 8621 & 8669 & 40.14 & 40.39 & 8621 & 8621 & 8621 & 8621 & 8621.0 & 8573.3 & 1 & 1 & 1 & 1 & 107026 & 175000 & 175000 & 305.81 & 354.02 & 284.6 & 352.36 & 9383 & 10061 & 97.18 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 1 part & 2881 & 63.726 & 1.000 & 3839.46 & 1746.41 & 127.34 & 5497 & 5497 & 5497 & 5497 & - & - & 3505.0 & 3485.6 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8656 & 8656 & 8656 & 0 & 0 & 0 & 1 & 8656 & 8656 & 8669 & 40.69 & 40.39 & 8656 & 8656 & 8656 & 8656 & 8656.0 & 8643.0 & 1 & 1 & 1 & 1 & 107309 & 175000 & 175000 & 307.15 & 354.02 & 291.48 & 352.36 & 9415 & 10061 & 99.34 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.099 & 1.004 & 0.00 & 1519.08 & 635.89 & 8492 & 8492 & 8492 & 8492 & 8492 & 8492 & 8331.1 & 8318.6 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
