\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
Total length ($\geq$ 1000 bp) & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
Total length ($\geq$ 5000 bp) & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
Total length & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
Reference length & 8725 & 8725 & 8725 & 8725 \\ \hline
GC (\%) & - & 40.67 & 40.12 & 40.59 \\ \hline
Reference GC (\%) & 40.66 & 40.66 & 40.66 & 40.66 \\ \hline
N50 & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
NG50 & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
N90 & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
NG90 & 8624 & 8686 & {\bf 8701} & 8677 \\ \hline
auN & 8624.0 & 8686.0 & {\bf 8701.0} & 8677.0 \\ \hline
auNG & 8524.2 & 8647.2 & {\bf 8677.1} & 8629.3 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 19000 & 16896 & 16326 & 16899 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & 0.0 & {\bf 105.87} & 74.78 & 105.83 \\ \hline
Reference mapped (\%) & 105.65 & 105.65 & 105.65 & 105.65 \\ \hline
Properly paired (\%) & 0.0 & {\bf 101.1} & 44.47 & 101.07 \\ \hline
Reference properly paired (\%) & 101.01 & 101.01 & 101.01 & 101.01 \\ \hline
Avg. coverage depth & 0 & {\bf 512} & 340 & {\bf 512} \\ \hline
Reference avg. coverage depth & 511 & 511 & 511 & 511 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 100.0 & 77.69 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 5 & 0 \\ \hline
\# structural variations & - & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 1 + 0 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & 8624 & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & - & 100.000 & 79.770 & 100.000 \\ \hline
Duplication ratio & - & 1.000 & 1.000 & 0.999 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & 22192.85 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 57.56 & 721.71 & 138.30 \\ \hline
\# indels per 100 kbp & - & 115.13 & 129.91 & 126.77 \\ \hline
Largest alignment & - & 8686 & 6928 & 8677 \\ \hline
Total aligned length & - & 8686 & 6928 & 8677 \\ \hline
NA50 & - & 8686 & 6928 & 8677 \\ \hline
NGA50 & - & 8686 & 6928 & 8677 \\ \hline
NA90 & - & 8686 & - & 8677 \\ \hline
NGA90 & - & 8686 & - & 8677 \\ \hline
auNA & - & 8686.0 & 5516.3 & 8677.0 \\ \hline
auNGA & - & 8647.2 & 5501.1 & 8629.3 \\ \hline
LA50 & - & 1 & 1 & 1 \\ \hline
LGA50 & - & 1 & 1 & 1 \\ \hline
LA90 & - & 1 & - & 1 \\ \hline
LGA90 & - & 1 & - & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
