\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
Total length ($\geq$ 1000 bp) & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
Total length ($\geq$ 5000 bp) & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
Total length & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
Reference length & 8632 & 8632 & 8632 & 8632 \\ \hline
GC (\%) & 40.76 & 40.82 & 40.54 & 41.14 \\ \hline
Reference GC (\%) & 40.75 & 40.75 & 40.75 & 40.75 \\ \hline
N50 & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
NG50 & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
N90 & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
NG90 & 8622 & 8614 & 8610 & {\bf 8639} \\ \hline
auN & 8622.0 & 8614.0 & 8610.0 & {\bf 8639.0} \\ \hline
auNG & 8612.0 & 8596.0 & 8588.1 & {\bf 8646.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15364 & 15357 & 15214 & 15246 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 113.99} & 113.86 & 109.48 & 108.27 \\ \hline
Reference mapped (\%) & 113.97 & 113.97 & 113.97 & 113.97 \\ \hline
Properly paired (\%) & {\bf 113.42} & 113.12 & 103.29 & 101.64 \\ \hline
Reference properly paired (\%) & 113.41 & 113.41 & 113.41 & 113.41 \\ \hline
Avg. coverage depth & {\bf 506} & {\bf 506} & 480 & 472 \\ \hline
Reference avg. coverage depth & 507 & 507 & 507 & 507 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 98.12 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.932 & 99.954 & 98.015 \\ \hline
Duplication ratio & 1.001 & {\bf 1.000} & {\bf 1.000} & 1.003 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 1881.53 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 11.60} & 23.47 & 220.67 & 1936.70 \\ \hline
\# indels per 100 kbp & 104.38 & 58.67 & {\bf 58.07} & 602.27 \\ \hline
Largest alignment & {\bf 8622} & 8522 & 8610 & 8468 \\ \hline
Total aligned length & {\bf 8622} & 8522 & 8610 & 8468 \\ \hline
NA50 & {\bf 8622} & 8522 & 8610 & 8468 \\ \hline
NGA50 & {\bf 8622} & 8522 & 8610 & 8468 \\ \hline
NA90 & {\bf 8622} & 8522 & 8610 & 8468 \\ \hline
NGA90 & {\bf 8622} & 8522 & 8610 & 8468 \\ \hline
auNA & {\bf 8622.0} & 8431.0 & 8610.0 & 8300.4 \\ \hline
auNGA & {\bf 8612.0} & 8413.4 & 8588.1 & 8307.1 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
