\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8622 & 8622 & 8622 & 0 & 0 & 0 & 1 & 8622 & 8622 & 8632 & 40.76 & 40.75 & 8622 & 8622 & 8622 & 8622 & 8622.0 & 8612.0 & 1 & 1 & 1 & 1 & 15364 & 8800 & 8800 & 113.99 & 113.97 & 113.42 & 113.41 & 506 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 11.60 & 104.38 & 8622 & 8622 & 8622 & 8622 & 8622 & 8622 & 8622.0 & 8612.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8632 & 40.82 & 40.75 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8596.0 & 1 & 1 & 1 & 1 & 15357 & 8800 & 8800 & 113.86 & 113.97 & 113.12 & 113.41 & 506 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.932 & 1.000 & 0.00 & 23.47 & 58.67 & 8522 & 8522 & 8522 & 8522 & 8522 & 8522 & 8431.0 & 8413.4 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8610 & 8610 & 8610 & 0 & 0 & 0 & 1 & 8610 & 8610 & 8632 & 40.54 & 40.75 & 8610 & 8610 & 8610 & 8610 & 8610.0 & 8588.1 & 1 & 1 & 1 & 1 & 15214 & 8800 & 8800 & 109.48 & 113.97 & 103.29 & 113.41 & 480 & 507 & 98.12 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.954 & 1.000 & 1881.53 & 220.67 & 58.07 & 8610 & 8610 & 8610 & 8610 & 8610 & 8610 & 8610.0 & 8588.1 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8639 & 8639 & 8639 & 0 & 0 & 0 & 1 & 8639 & 8639 & 8632 & 41.14 & 40.75 & 8639 & 8639 & 8639 & 8639 & 8639.0 & 8646.0 & 1 & 1 & 1 & 1 & 15246 & 8800 & 8800 & 108.27 & 113.97 & 101.64 & 113.41 & 472 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.015 & 1.003 & 0.00 & 1936.70 & 602.27 & 8468 & 8468 & 8468 & 8468 & 8468 & 8468 & 8300.4 & 8307.1 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
