\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
Total length & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
Reference length & 8647 & 8647 & 8647 & 8647 \\ \hline
GC (\%) & 40.51 & 40.65 & 40.39 & 40.87 \\ \hline
Reference GC (\%) & 40.51 & 40.51 & 40.51 & 40.51 \\ \hline
N50 & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
NG50 & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
N75 & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
NG75 & {\bf 8629} & 8627 & 8449 & 8618 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.61} & 99.41 & 85.3 & 94.07 \\ \hline
Reference mapped (\%) & 99.61 & 99.61 & 99.61 & 99.61 \\ \hline
Properly paired (\%) & {\bf 99.22} & 98.83 & 72.32 & 88.91 \\ \hline
Reference properly paired (\%) & 99.22 & 99.22 & 99.22 & 99.22 \\ \hline
Avg. coverage depth & {\bf 507} & 506 & 437 & 472 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.92 & 92.16 & 99.87 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 5 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.760 & 89.963 & 97.844 \\ \hline
Duplication ratio & {\bf 1.000} & 1.012 & 1.089 & 1.021 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 8474.38 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 11.74 & 244.78 & 2428.33 \\ \hline
\# indels per 100 kbp & {\bf 69.54} & 70.41 & 77.30 & 497.51 \\ \hline
Largest alignment & {\bf 8629} & 8522 & 7762 & 8446 \\ \hline
Total aligned length & {\bf 8629} & 8522 & 7762 & 8446 \\ \hline
NA50 & {\bf 8629} & 8522 & 7762 & 8446 \\ \hline
NGA50 & {\bf 8629} & 8522 & 7762 & 8446 \\ \hline
NA75 & {\bf 8629} & 8522 & 7762 & 8446 \\ \hline
NGA75 & {\bf 8629} & 8522 & 7762 & 8446 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
