\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8718 & 8718 & 8718 & 0 & 0 & 0 & 1 & 8718 & 8718 & 8748 & 40.28 & 40.27 & 8718 & 8718 & 8718 & 8718 & 8718.0 & 8688.1 & 1 & 1 & 1 & 1 & 124103 & 190000 & 190000 & 288.2 & 287.86 & 276.69 & 276.36 & 10210 & 10208 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 0.00 & 0.00 & 91.76 & 8718 & 8718 & 8718 & 8718 & 8718 & 8718 & 8718.0 & 8688.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8727 & 8727 & 8727 & 0 & 0 & 0 & 1 & 8727 & 8727 & 8748 & 40.35 & 40.27 & 8727 & 8727 & 8727 & 8727 & 8727.0 & 8706.1 & 1 & 1 & 1 & 1 & 123952 & 190000 & 190000 & 288.83 & 287.86 & 277.2 & 276.36 & 10214 & 10208 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 412.51 & 68.75 & 8727 & 8727 & 8727 & 8727 & 8727 & 8727 & 8727.0 & 8706.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8870 & 8870 & 8870 & 0 & 0 & 0 & 1 & 8870 & 8870 & 8748 & 40.37 & 40.27 & 8870 & 8870 & 8870 & 8870 & 8870.0 & 8993.7 & 1 & 1 & 1 & 1 & 132948 & 190000 & 190000 & 260.7 & 287.86 & 244.27 & 276.36 & 9650 & 10208 & 98.76 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 96.446 & 1.001 & 3855.69 & 3288.61 & 59.36 & 8423 & 8423 & 8423 & 8423 & 8423 & 8423 & 7998.5 & 8110.1 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8742 & 8742 & 8742 & 0 & 0 & 0 & 1 & 8742 & 8742 & 8748 & 40.25 & 40.27 & 8742 & 8742 & 8742 & 8742 & 8742.0 & 8736.0 & 1 & 1 & 1 & 1 & 124281 & 190000 & 190000 & 287.98 & 287.86 & 276.48 & 276.36 & 10209 & 10208 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.002 & 0.00 & 22.88 & 22.88 & 8742 & 8742 & 8742 & 8742 & 8742 & 8742 & 8742.0 & 8736.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
