\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8731 & 40.58 & 40.49 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8641.2 & 1 & 1 & 1 & 1 & 122778 & 190000 & 190000 & 291.53 & 290.71 & 278.8 & 277.96 & 10260 & 10246 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 218.74 & 115.13 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8641.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8644 & 8644 & 8644 & 0 & 0 & 0 & 1 & 8644 & 8644 & 8731 & 41.21 & 40.49 & 8644 & 8644 & 8644 & 8644 & 8644.0 & 8557.9 & 1 & 1 & 1 & 1 & 214450 & 190000 & 190000 & 96.52 & 290.71 & 70.5 & 277.96 & 5815 & 10246 & 66.66 & 100.0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 & 0 & 0 + 1 part & 2796 & 67.312 & 1.001 & 0.00 & 376.20 & 102.60 & 5848 & 5848 & 5848 & 5848 & - & - & 3956.4 & 3917.0 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 5298 & 5298 & 5298 & 0 & 0 & 0 & 1 & 5298 & 5298 & 8731 & 40.48 & 40.49 & 5298 & 5298 & 5298 & - & 5298.0 & 3214.8 & 1 & 1 & 1 & - & 249666 & 190000 & 190000 & 64.37 & 290.71 & 42.51 & 277.96 & 7416 & 10246 & 80.11 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 48.854 & 1.001 & 18875.05 & 824.69 & 23.56 & 4244 & 4244 & 4244 & - & - & - & 3399.7 & 2062.9 & 1 & - & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8731 & 41.33 & 40.49 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8518.3 & 1 & 1 & 1 & 1 & 326842 & 190000 & 190000 & 19.28 & 290.71 & 8.52 & 277.96 & 1726 & 10246 & 57.11 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7682 & 10.819 & 1.003 & 0.00 & 2335.46 & 424.63 & 733 & 942 & - & - & - & - & 67.4 & 66.5 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
