\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
Total length ($\geq$ 1000 bp) & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
Total length ($\geq$ 5000 bp) & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
Total length & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
Reference length & 8732 & 8732 & 8732 & 8732 \\ \hline
GC (\%) & 40.78 & 40.73 & 40.72 & 40.73 \\ \hline
Reference GC (\%) & 40.74 & 40.74 & 40.74 & 40.74 \\ \hline
N50 & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
NG50 & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
N90 & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
NG90 & 8649 & 8720 & 8714 & {\bf 8729} \\ \hline
auN & 8649.0 & 8720.0 & 8714.0 & {\bf 8729.0} \\ \hline
auNG & 8566.8 & 8708.0 & 8696.0 & {\bf 8726.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 109718 & 106097 & 106081 & 106193 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & 320.2 & 338.17 & {\bf 338.18} & 337.86 \\ \hline
Reference mapped (\%) & 337.67 & 337.67 & 337.67 & 337.67 \\ \hline
Properly paired (\%) & 299.51 & {\bf 323.81} & 323.79 & 323.52 \\ \hline
Reference properly paired (\%) & 323.27 & 323.27 & 323.27 & 323.27 \\ \hline
Avg. coverage depth & 10098 & {\bf 10256} & 10254 & 10253 \\ \hline
Reference avg. coverage depth & 10251 & 10251 & 10251 & 10251 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 99.943 & 99.943 & 99.943 & 99.943 \\ \hline
Duplication ratio & {\bf 0.992} & 1.000 & 0.999 & 1.001 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & 69.44 & {\bf 0.00} & 310.13 & {\bf 0.00} \\ \hline
\# indels per 100 kbp & 81.01 & 34.44 & 45.95 & {\bf 11.47} \\ \hline
Largest alignment & 8641 & 8712 & 8706 & {\bf 8721} \\ \hline
Total aligned length & 8641 & 8712 & 8706 & {\bf 8721} \\ \hline
NA50 & 8641 & 8712 & 8706 & {\bf 8721} \\ \hline
NGA50 & 8641 & 8712 & 8706 & {\bf 8721} \\ \hline
NA90 & 8641 & 8712 & 8706 & {\bf 8721} \\ \hline
NGA90 & 8641 & 8712 & 8706 & {\bf 8721} \\ \hline
auNA & 8633.0 & 8704.0 & 8698.0 & {\bf 8713.0} \\ \hline
auNGA & 8550.9 & 8692.0 & 8680.1 & {\bf 8710.0} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
