\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8647 & - & 40.20 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8601.1 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 0.0 & 112.73 & 0.0 & 112.12 & 0 & 505 & 0.0 & 100.0 & - & - & - & - & - & - & - & - & 1 + 0 part & 8624 & - & - & 100000.00 & - & - & - & - & - & - & - & - & - & - & - & - & - & - \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8617 & 8617 & 8617 & 0 & 0 & 0 & 1 & 8617 & 8617 & 8647 & 40.26 & 40.20 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8587.1 & 1 & 1 & 1 & 1 & 15522 & 8800 & 8800 & 112.56 & 112.73 & 111.73 & 112.12 & 505 & 505 & 99.88 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.897 & 1.000 & 0.00 & 821.40 & 70.41 & 8522 & 8522 & 8522 & 8522 & 8522 & 8522 & 8428.0 & 8398.8 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8174 & 8174 & 8174 & 0 & 0 & 0 & 1 & 8174 & 8174 & 8647 & 39.60 & 40.20 & 8174 & 8174 & 8174 & 8174 & 8174.0 & 7726.9 & 1 & 1 & 1 & 1 & 15308 & 8800 & 8800 & 63.67 & 112.73 & 36.74 & 112.12 & 287 & 505 & 66.71 & 100.0 & 0 & 0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 + 0 part & 0 & 63.227 & 1.000 & 34560.80 & 1065.20 & 73.46 & 5445 & 5445 & 5445 & 5445 & - & - & 3627.1 & 3428.7 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8647 & 40.79 & 40.20 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8581.1 & 1 & 1 & 1 & 1 & 15355 & 8800 & 8800 & 102.86 & 112.73 & 92.58 & 112.12 & 451 & 505 & 99.3 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.015 & 1.000 & 0.00 & 3268.98 & 746.18 & 8443 & 8443 & 8443 & 8443 & 8443 & 8443 & 8275.4 & 8243.8 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
