\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
Total length & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
Reference length & 8653 & 8653 & 8653 & 8653 \\ \hline
GC (\%) & 40.45 & 40.48 & 40.17 & 40.80 \\ \hline
Reference GC (\%) & 40.43 & 40.43 & 40.43 & 40.43 \\ \hline
N50 & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
NG50 & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
N75 & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
NG75 & 8622 & {\bf 8627} & 8574 & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.57} & 99.43 & 67.01 & 89.7 \\ \hline
Reference mapped (\%) & 99.57 & 99.57 & 99.57 & 99.57 \\ \hline
Properly paired (\%) & {\bf 99.15} & 98.85 & 45.98 & 81.02 \\ \hline
Reference properly paired (\%) & 99.15 & 99.15 & 99.15 & 99.15 \\ \hline
Avg. coverage depth & {\bf 506} & 505 & 335 & 450 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.92 & 74.83 & 99.32 \\ \hline
Reference coverage $\geq$ 1x (\%) & 99.94 & 99.94 & 99.94 & 99.94 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 4 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 3 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & {\bf 0} & 1643 \\ \hline
Genome fraction (\%) & {\bf 99.907} & 99.106 & 76.221 & 80.863 \\ \hline
Duplication ratio & {\bf 1.002} & 1.010 & 1.305 & {\bf 1.002} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 26638.68 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 197.47} & 515.22 & 1233.25 & 2568.89 \\ \hline
\# indels per 100 kbp & 116.16 & 58.55 & 30.45 & {\bf 28.70} \\ \hline
Largest alignment & {\bf 8614} & 8549 & 6576 & 6982 \\ \hline
Total aligned length & {\bf 8614} & 8549 & 6576 & 6982 \\ \hline
NA50 & {\bf 8614} & 8549 & 6576 & 6982 \\ \hline
NGA50 & {\bf 8614} & 8549 & 6576 & 6982 \\ \hline
NA75 & {\bf 8614} & 8549 & 6576 & 6982 \\ \hline
NGA75 & {\bf 8614} & 8549 & 6576 & 6982 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
