\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8666 & 8666 & 8666 & 0 & 0 & 0 & 1 & 8666 & 8666 & 8701 & 40.60 & 40.60 & 8666 & 8666 & 8666 & 8666 & 8666.0 & 8631.1 & 1 & 1 & 1 & 1 & 16728 & 9500 & 9500 & 105.8 & 106.54 & 100.2 & 101.85 & 507 & 511 & 99.95 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.965 & 0.998 & 0.00 & 80.81 & 92.36 & 8662 & 8662 & 8662 & 8662 & 8662 & 8662 & 8658.0 & 8623.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8701 & 40.59 & 40.60 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8671.0 & 1 & 1 & 1 & 1 & 16751 & 9500 & 9500 & 106.55 & 106.54 & 101.87 & 101.85 & 511 & 511 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.942 & 1.000 & 0.00 & 207.35 & 46.08 & 8681 & 8681 & 8681 & 8681 & 8681 & 8681 & 8676.0 & 8661.0 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8262 & 8262 & 8262 & 0 & 0 & 0 & 1 & 8262 & 8262 & 8701 & 40.08 & 40.60 & 8262 & 8262 & 8262 & 8262 & 8262.0 & 7845.1 & 1 & 1 & 1 & 1 & 16437 & 9500 & 9500 & 78.58 & 106.54 & 55.1 & 101.85 & 382 & 511 & 82.39 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 79.498 & 1.000 & 18167.51 & 1288.92 & 43.45 & 6905 & 6905 & 6905 & 6905 & - & - & 5770.9 & 5479.7 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8683 & 8683 & 8683 & 0 & 0 & 0 & 1 & 8683 & 8683 & 8701 & 40.55 & 40.60 & 8683 & 8683 & 8683 & 8683 & 8683.0 & 8665.0 & 1 & 1 & 1 & 1 & 16746 & 9500 & 9500 & 106.63 & 106.54 & 101.89 & 101.85 & 512 & 511 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 932.86 & 57.58 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683.0 & 8665.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
