\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8657 & 40.71 & 40.77 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8591.1 & 1 & 1 & 1 & 1 & 15614 & 8800 & 8800 & 112.22 & 112.18 & 111.72 & 111.69 & 506 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 150.74 & 115.96 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8591.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8657 & 40.76 & 40.77 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8591.1 & 1 & 1 & 1 & 1 & 15608 & 8800 & 8800 & 112.09 & 112.18 & 111.42 & 111.69 & 506 & 506 & 99.93 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.000 & 0.00 & 418.07 & 116.13 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 8598.0 & 8565.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7457 & 7457 & 7457 & 0 & 0 & 0 & 1 & 7457 & 7457 & 8657 & 40.35 & 40.77 & 7457 & 7457 & 7457 & - & 7457.0 & 6423.3 & 1 & 1 & 1 & - & 15506 & 8800 & 8800 & 76.87 & 112.18 & 60.83 & 111.69 & 390 & 506 & 83.91 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 73.773 & 0.999 & 16843.23 & 723.73 & 78.67 & 6356 & 6356 & 6356 & 6356 & - & - & 5417.6 & 4666.6 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8641 & 8641 & 8641 & 0 & 0 & 0 & 1 & 8641 & 8641 & 8657 & 41.05 & 40.77 & 8641 & 8641 & 8641 & 8641 & 8641.0 & 8625.0 & 1 & 1 & 1 & 1 & 15463 & 8800 & 8800 & 105.65 & 112.18 & 98.74 & 111.69 & 466 & 506 & 99.25 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.002 & 0.00 & 2515.07 & 591.10 & 8628 & 8628 & 8628 & 8628 & 8628 & 8628 & 8615.0 & 8599.1 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
