\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
Total length & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
Reference length & 8625 & 8625 & 8625 & 8625 \\ \hline
GC (\%) & 40.44 & 40.41 & 40.31 & 40.82 \\ \hline
Reference GC (\%) & 40.42 & 40.42 & 40.42 & 40.42 \\ \hline
N50 & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
NG50 & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
N90 & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
NG90 & {\bf 8612} & 8574 & 8595 & 8598 \\ \hline
auN & {\bf 8612.0} & 8574.0 & 8595.0 & 8598.0 \\ \hline
auNG & {\bf 8599.0} & 8523.3 & 8565.1 & 8571.1 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15714 & 15702 & 15579 & 15478 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 110.45} & 110.06 & 107.53 & 100.81 \\ \hline
Reference mapped (\%) & 110.49 & 110.49 & 110.49 & 110.49 \\ \hline
Properly paired (\%) & {\bf 108.9} & 108.02 & 102.09 & 90.21 \\ \hline
Reference properly paired (\%) & 109.07 & 109.07 & 109.07 & 109.07 \\ \hline
Avg. coverage depth & {\bf 502} & 500 & 484 & 446 \\ \hline
Reference avg. coverage depth & 503 & 503 & 503 & 503 \\ \hline
Coverage $\geq$ 1x (\%) & 99.9 & 99.86 & 98.07 & 99.57 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 99.930} & 98.883 & 98.068 & 98.010 \\ \hline
Duplication ratio & 1.002 & {\bf 0.998} & 1.000 & 1.001 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 2187.32 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 34.87} & 94.36 & 355.91 & 2041.06 \\ \hline
\# indels per 100 kbp & 151.11 & 106.16 & {\bf 94.91} & 605.20 \\ \hline
Largest alignment & {\bf 8603} & 8478 & 8429 & 8427 \\ \hline
Total aligned length & {\bf 8603} & 8478 & 8429 & 8427 \\ \hline
NA50 & {\bf 8603} & 8478 & 8429 & 8427 \\ \hline
NGA50 & {\bf 8603} & 8478 & 8429 & 8427 \\ \hline
NA90 & {\bf 8603} & 8478 & 8429 & 8427 \\ \hline
NGA90 & {\bf 8603} & 8478 & 8429 & 8427 \\ \hline
auNA & {\bf 8594.0} & 8383.1 & 8266.2 & 8259.4 \\ \hline
auNGA & {\bf 8581.1} & 8333.5 & 8237.5 & 8233.5 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
