\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8685 & 8685 & 8685 & 0 & 0 & 0 & 1 & 8685 & 8685 & 8710 & 40.59 & 40.61 & 8685 & 8685 & 8685 & 8685 & 8685.0 & 8660.1 & 1 & 1 & 1 & 1 & 115818 & 190000 & 190000 & 308.6 & 307.96 & 294.7 & 294.09 & 10247 & 10241 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 23.03 & 80.60 & 8685 & 8685 & 8685 & 8685 & 8685 & 8685 & 8685.0 & 8660.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8710 & 41.20 & 40.61 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8566.6 & 1 & 1 & 1 & 1 & 220924 & 190000 & 190000 & 89.89 & 307.96 & 66.05 & 294.09 & 5596 & 10241 & 65.64 & 100.0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 & 0 & 0 + 1 part & 3066 & 64.130 & 1.002 & 0.00 & 771.72 & 89.73 & 5572 & 5572 & 5572 & 5572 & - & - & 3594.3 & 3564.5 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 0 & 0 & 0 & 0 & 4349 & 4349 & 0 & 0 & 0 & 0 & 1 & 4349 & 4349 & 8710 & 39.73 & 40.61 & 4349 & - & 4349 & - & 4349.0 & 2171.5 & 1 & - & 1 & - & 279073 & 190000 & 190000 & 44.3 & 307.96 & 31.29 & 294.09 & 6816 & 10241 & 79.86 & 100.0 & 0 & 0 & 0 & 0 & 0 & 2 & 0 & 0 & 0 + 0 part & 0 & 41.981 & 1.002 & 21522.19 & 1206.47 & 82.26 & 3647 & 3647 & 3647 & - & - & - & 3058.3 & 1527.1 & 1 & - & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8627 & 8627 & 8627 & 0 & 0 & 0 & 1 & 8627 & 8627 & 8710 & 41.31 & 40.61 & 8627 & 8627 & 8627 & 8627 & 8627.0 & 8544.8 & 1 & 1 & 1 & 1 & 328846 & 190000 & 190000 & 18.62 & 307.96 & 8.11 & 294.09 & 1686 & 10241 & 57.46 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7574 & 12.095 & 1.004 & 0.00 & 1899.34 & 474.83 & 861 & 1053 & - & - & - & - & 90.2 & 89.3 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
