\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
Total length ($\geq$ 1000 bp) & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
Total length ($\geq$ 5000 bp) & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
Total length & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
Reference length & 8636 & 8636 & 8636 & 8636 \\ \hline
GC (\%) & - & 40.20 & 39.97 & 40.36 \\ \hline
Reference GC (\%) & 40.27 & 40.27 & 40.27 & 40.27 \\ \hline
N50 & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
NG50 & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
N75 & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
NG75 & 8500 & {\bf 8606} & 8391 & {\bf 8606} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 0.0 & 99.53 & 89.83 & {\bf 99.58} \\ \hline
Reference mapped (\%) & 99.58 & 99.58 & 99.58 & 99.58 \\ \hline
Properly paired (\%) & 0.0 & 99.06 & 79.66 & {\bf 99.16} \\ \hline
Reference properly paired (\%) & 99.16 & 99.16 & 99.16 & 99.16 \\ \hline
Avg. coverage depth & 0 & {\bf 507} & 465 & {\bf 507} \\ \hline
Reference avg. coverage depth & 507 & 507 & 507 & 507 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 100.0 & 96.01 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 2 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 1 + 0 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & 8500 & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & - & 99.965 & 94.457 & 100.000 \\ \hline
Duplication ratio & - & 1.000 & 1.032 & 1.000 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & 4862.35 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 244.13 & 652.07 & 511.33 \\ \hline
\# indels per 100 kbp & - & 81.38 & 61.52 & 81.35 \\ \hline
Largest alignment & - & 8603 & 8138 & 8606 \\ \hline
Total aligned length & - & 8603 & 8138 & 8606 \\ \hline
NA50 & - & 8603 & 8138 & 8606 \\ \hline
NGA50 & - & 8603 & 8138 & 8606 \\ \hline
NA75 & - & 8603 & 8138 & 8606 \\ \hline
NGA75 & - & 8603 & 8138 & 8606 \\ \hline
LA50 & - & 1 & 1 & 1 \\ \hline
LGA50 & - & 1 & 1 & 1 \\ \hline
LA75 & - & 1 & 1 & 1 \\ \hline
LGA75 & - & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
