\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8642 & 40.61 & 40.56 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8606.0 & 1 & 1 & 1 & 1 & 15661 & 8800 & 8800 & 111.86 & 111.84 & 111.35 & 111.3 & 506 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 46.38 & 69.57 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8606.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8627 & 8627 & 8627 & 0 & 0 & 0 & 1 & 8627 & 8627 & 8642 & 40.45 & 40.56 & 8627 & 8627 & 8627 & 8627 & 8627.0 & 8612.0 & 1 & 1 & 1 & 1 & 15656 & 8800 & 8800 & 111.76 & 111.84 & 111.1 & 111.3 & 506 & 506 & 99.93 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.001 & 0.00 & 592.06 & 58.05 & 8614 & 8614 & 8614 & 8614 & 8614 & 8614 & 8601.0 & 8586.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8175 & 8175 & 8175 & 0 & 0 & 0 & 1 & 8175 & 8175 & 8642 & 39.93 & 40.56 & 8175 & 8175 & 8175 & 8175 & 8175.0 & 7733.2 & 1 & 1 & 1 & 1 & 15541 & 8800 & 8800 & 107.68 & 111.84 & 102.31 & 111.3 & 507 & 506 & 99.02 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 94.827 & 1.000 & 990.83 & 2801.22 & 61.16 & 8175 & 8175 & 8175 & 8175 & 8175 & 8175 & 8175.0 & 7733.2 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8623 & 8623 & 8623 & 0 & 0 & 0 & 1 & 8623 & 8623 & 8642 & 40.81 & 40.56 & 8623 & 8623 & 8623 & 8623 & 8623.0 & 8604.0 & 1 & 1 & 1 & 1 & 15504 & 8800 & 8800 & 103.43 & 111.84 & 95.28 & 111.3 & 456 & 506 & 99.2 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 1 part & 826 & 90.246 & 1.002 & 0.00 & 2693.34 & 602.80 & 7797 & 7797 & 7797 & 7797 & 7797 & 7797 & 7050.1 & 7034.6 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
