\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8722 & 8722 & 8722 & 0 & 0 & 0 & 1 & 8722 & 8722 & 8764 & 40.54 & 40.55 & 8722 & 8722 & 8722 & 8722 & 8722.0 & 8680.2 & 1 & 1 & 1 & 1 & 112441 & 190000 & 190000 & 319.55 & 318.62 & 306.38 & 305.49 & 10263 & 10245 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 34.40 & 114.65 & 8722 & 8722 & 8722 & 8722 & 8722 & 8722 & 8722.0 & 8680.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8720 & 8720 & 8720 & 0 & 0 & 0 & 1 & 8720 & 8720 & 8764 & 40.50 & 40.55 & 8720 & 8720 & 8720 & 8720 & 8720.0 & 8676.2 & 1 & 1 & 1 & 1 & 112468 & 190000 & 190000 & 319.44 & 318.62 & 306.23 & 305.49 & 10264 & 10245 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.977 & 1.000 & 0.00 & 217.89 & 114.68 & 8720 & 8720 & 8720 & 8720 & 8720 & 8720 & 8720.0 & 8676.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8724 & 8724 & 8724 & 0 & 0 & 0 & 1 & 8724 & 8724 & 8764 & 40.55 & 40.55 & 8724 & 8724 & 8724 & 8724 & 8724.0 & 8684.2 & 1 & 1 & 1 & 1 & 112744 & 190000 & 190000 & 318.41 & 318.62 & 305.3 & 305.49 & 10249 & 10245 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 309.49 & 882.83 & 114.65 & 8722 & 8722 & 8722 & 8722 & 8722 & 8722 & 8720.0 & 8680.2 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8745 & 8745 & 8745 & 0 & 0 & 0 & 1 & 8745 & 8745 & 8764 & 40.53 & 40.55 & 8745 & 8745 & 8745 & 8745 & 8745.0 & 8726.0 & 1 & 1 & 1 & 1 & 112631 & 190000 & 190000 & 319.0 & 318.62 & 305.81 & 305.49 & 10252 & 10245 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.989 & 1.003 & 0.00 & 240.14 & 57.18 & 8745 & 8745 & 8745 & 8745 & 8745 & 8745 & 8745.0 & 8726.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
