\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8611 & 8611 & 8611 & 0 & 0 & 0 & 1 & 8611 & 8611 & 8650 & 40.31 & 40.31 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8572.2 & 1 & 1 & 1 & 1 & 15607 & 8800 & 8800 & 112.3 & 112.3 & 111.82 & 111.82 & 507 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 139.36 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8572.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8611 & 8611 & 8611 & 0 & 0 & 0 & 1 & 8611 & 8611 & 8650 & 40.44 & 40.31 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8572.2 & 1 & 1 & 1 & 1 & 15593 & 8800 & 8800 & 112.15 & 112.3 & 111.42 & 111.82 & 506 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.815 & 1.000 & 0.00 & 94.02 & 141.03 & 8509 & 8509 & 8509 & 8509 & 8509 & 8509 & 8408.2 & 8370.3 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8456 & 8456 & 8456 & 0 & 0 & 0 & 1 & 8456 & 8456 & 8650 & 40.21 & 40.31 & 8456 & 8456 & 8456 & 8456 & 8456.0 & 8266.4 & 1 & 1 & 1 & 1 & 15501 & 8800 & 8800 & 109.7 & 112.3 & 105.85 & 111.82 & 498 & 507 & 99.07 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.165 & 1.000 & 2530.75 & 248.34 & 130.09 & 8456 & 8456 & 8456 & 8456 & 8456 & 8456 & 8456.0 & 8266.4 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8608 & 8608 & 8608 & 0 & 0 & 0 & 1 & 8608 & 8608 & 8650 & 40.68 & 40.31 & 8608 & 8608 & 8608 & 8608 & 8608.0 & 8566.2 & 1 & 1 & 1 & 1 & 15424 & 8800 & 8800 & 105.18 & 112.3 & 97.39 & 111.82 & 462 & 507 & 98.62 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.060 & 1.000 & 0.00 & 2322.00 & 722.66 & 8441 & 8441 & 8441 & 8441 & 8441 & 8441 & 8277.2 & 8237.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
