\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
Total length ($\geq$ 1000 bp) & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
Total length ($\geq$ 5000 bp) & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
Total length & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
Reference length & 8653 & 8653 & 8653 & 8653 \\ \hline
GC (\%) & 40.76 & 40.86 & 40.76 & 40.98 \\ \hline
Reference GC (\%) & 40.73 & 40.73 & 40.73 & 40.73 \\ \hline
N50 & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
NG50 & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
N75 & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
NG75 & 8626 & 8627 & 8626 & {\bf 8640} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 350000 & 350000 & 350000 & 350000 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 99.61} & 99.43 & {\bf 99.61} & 97.29 \\ \hline
Reference mapped (\%) & 99.61 & 99.61 & 99.61 & 99.61 \\ \hline
Properly paired (\%) & {\bf 99.23} & 98.86 & {\bf 99.23} & 94.59 \\ \hline
Reference properly paired (\%) & 99.23 & 99.23 & 99.23 & 99.23 \\ \hline
Avg. coverage depth & {\bf 10088} & 10074 & 10085 & 9784 \\ \hline
Reference avg. coverage depth & 10078 & 10078 & 10078 & 10078 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.97 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.760 & {\bf 100.000} & 98.644 \\ \hline
Duplication ratio & {\bf 1.000} & 1.012 & {\bf 1.000} & 1.015 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & 11.59 & {\bf 0.00} & {\bf 0.00} & 751.88 \\ \hline
\# indels per 100 kbp & 92.71 & 82.14 & {\bf 81.12} & 364.19 \\ \hline
Largest alignment & {\bf 8626} & 8522 & {\bf 8626} & 8536 \\ \hline
Total aligned length & {\bf 8626} & 8522 & {\bf 8626} & 8536 \\ \hline
NA50 & {\bf 8626} & 8522 & {\bf 8626} & 8536 \\ \hline
NGA50 & {\bf 8626} & 8522 & {\bf 8626} & 8536 \\ \hline
NA75 & {\bf 8626} & 8522 & {\bf 8626} & 8536 \\ \hline
NGA75 & {\bf 8626} & 8522 & {\bf 8626} & 8536 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
