\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
Total length ($\geq$ 1000 bp) & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
Total length ($\geq$ 5000 bp) & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
Total length & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
Reference length & 8738 & 8738 & 8738 & 8738 \\ \hline
GC (\%) & - & 40.33 & 39.99 & 40.26 \\ \hline
Reference GC (\%) & 40.33 & 40.33 & 40.33 & 40.33 \\ \hline
N50 & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
NG50 & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
N90 & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
NG90 & 8500 & 8717 & 8716 & {\bf 8718} \\ \hline
auN & 8500.0 & 8717.0 & 8716.0 & {\bf 8718.0} \\ \hline
auNG & 8268.5 & 8696.1 & 8694.1 & {\bf 8698.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 19000 & 16836 & 16655 & 16836 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & 0.0 & {\bf 106.78} & 99.67 & {\bf 106.78} \\ \hline
Reference mapped (\%) & 106.75 & 106.75 & 106.75 & 106.75 \\ \hline
Properly paired (\%) & 0.0 & {\bf 102.52} & 89.64 & {\bf 102.52} \\ \hline
Reference properly paired (\%) & 102.49 & 102.49 & 102.49 & 102.49 \\ \hline
Avg. coverage depth & 0 & {\bf 514} & 471 & 513 \\ \hline
Reference avg. coverage depth & 513 & 513 & 513 & 513 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 100.0 & 94.81 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 2 & 0 \\ \hline
\# structural variations & - & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 1 + 0 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & 8500 & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & - & 100.000 & 94.814 & 100.000 \\ \hline
Duplication ratio & - & 1.000 & 1.000 & 1.000 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & 5667.74 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 57.36 & 459.83 & 240.91 \\ \hline
\# indels per 100 kbp & - & 68.83 & 60.50 & 68.83 \\ \hline
Largest alignment & - & 8717 & 8264 & 8717 \\ \hline
Total aligned length & - & 8717 & 8264 & 8717 \\ \hline
NA50 & - & 8717 & 8264 & 8717 \\ \hline
NGA50 & - & 8717 & 8264 & 8717 \\ \hline
NA90 & - & 8717 & 8264 & 8717 \\ \hline
NGA90 & - & 8717 & 8264 & 8717 \\ \hline
auNA & - & 8717.0 & 7835.4 & 8716.0 \\ \hline
auNGA & - & 8696.1 & 7815.7 & 8696.1 \\ \hline
LA50 & - & 1 & 1 & 1 \\ \hline
LGA50 & - & 1 & 1 & 1 \\ \hline
LA90 & - & 1 & 1 & 1 \\ \hline
LGA90 & - & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
