\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
Total length ($\geq$ 1000 bp) & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
Total length ($\geq$ 5000 bp) & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
Total length & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
Reference length & 8656 & 8656 & 8656 & 8656 \\ \hline
GC (\%) & 40.71 & 40.78 & 40.51 & 41.01 \\ \hline
Reference GC (\%) & 40.70 & 40.70 & 40.70 & 40.70 \\ \hline
N50 & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
NG50 & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
N75 & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
NG75 & 8626 & 8627 & 8601 & {\bf 8636} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.61} & 99.4 & 95.64 & 95.23 \\ \hline
Reference mapped (\%) & 99.61 & 99.61 & 99.61 & 99.61 \\ \hline
Properly paired (\%) & {\bf 99.22} & 98.81 & 91.27 & 91.17 \\ \hline
Reference properly paired (\%) & 99.22 & 99.22 & 99.22 & 99.22 \\ \hline
Avg. coverage depth & {\bf 507} & 506 & 485 & 477 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.92 & 97.5 & 99.05 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.760 & 97.184 & 98.644 \\ \hline
Duplication ratio & {\bf 1.000} & 1.012 & 1.026 & 1.015 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 4080.92 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 34.77} & 187.75 & 870.50 & 2608.08 \\ \hline
\# indels per 100 kbp & 81.12 & {\bf 70.41} & 71.55 & 587.41 \\ \hline
Largest alignment & {\bf 8626} & 8522 & 8386 & 8533 \\ \hline
Total aligned length & {\bf 8626} & 8522 & 8386 & 8533 \\ \hline
NA50 & {\bf 8626} & 8522 & 8386 & 8533 \\ \hline
NGA50 & {\bf 8626} & 8522 & 8386 & 8533 \\ \hline
NA75 & {\bf 8626} & 8522 & 8386 & 8533 \\ \hline
NGA75 & {\bf 8626} & 8522 & 8386 & 8533 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
