\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8621 & 8621 & 8621 & 0 & 0 & 0 & 1 & 8621 & 8621 & 8639 & 40.04 & 40.03 & 8621 & 8621 & 8621 & 8621 & 8621.0 & 8603.0 & 1 & 1 & 1 & 1 & 15564 & 8800 & 8800 & 112.47 & 112.44 & 111.86 & 111.83 & 506 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 69.60 & 8621 & 8621 & 8621 & 8621 & 8621 & 8621 & 8621.0 & 8603.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8639 & 40.11 & 40.03 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8609.0 & 1 & 1 & 1 & 1 & 15562 & 8800 & 8800 & 112.41 & 112.44 & 111.73 & 111.83 & 505 & 506 & 99.93 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.455 & 1.001 & 0.00 & 291.48 & 58.30 & 8577 & 8577 & 8577 & 8577 & 8577 & 8577 & 8530.3 & 8515.4 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8458 & 8458 & 8458 & 0 & 0 & 0 & 1 & 8458 & 8458 & 8639 & 40.07 & 40.03 & 8458 & 8458 & 8458 & 8458 & 8458.0 & 8280.8 & 1 & 1 & 1 & 1 & 15373 & 8800 & 8800 & 85.27 & 112.44 & 68.3 & 111.83 & 379 & 506 & 83.25 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 82.260 & 1.000 & 17947.51 & 846.38 & 84.64 & 7089 & 7089 & 7089 & 7089 & - & - & 5941.6 & 5817.1 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8618 & 8618 & 8618 & 0 & 0 & 0 & 1 & 8618 & 8618 & 8639 & 40.57 & 40.03 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8597.1 & 1 & 1 & 1 & 1 & 15295 & 8800 & 8800 & 103.48 & 112.44 & 93.99 & 111.83 & 450 & 506 & 99.16 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 1 part & 2800 & 67.560 & 0.999 & 0.00 & 2096.94 & 343.76 & 5818 & 5818 & 5818 & 5818 & - & - & 3927.7 & 3918.2 & 1 & 1 & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
