\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8592 & 8592 & 8592 & 0 & 0 & 0 & 1 & 8592 & 8592 & 8628 & 40.47 & 40.48 & 8592 & 8592 & 8592 & 8592 & 8592.0 & 8556.2 & 1 & 1 & 1 & 1 & 15567 & 8800 & 8800 & 111.65 & 111.61 & 110.25 & 110.2 & 504 & 504 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 0.00 & 23.28 & 116.39 & 8592 & 8592 & 8592 & 8592 & 8592 & 8592 & 8592.0 & 8556.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8577 & 8577 & 8577 & 0 & 0 & 0 & 1 & 8577 & 8577 & 8628 & 40.48 & 40.48 & 8577 & 8577 & 8577 & 8577 & 8577.0 & 8526.3 & 1 & 1 & 1 & 1 & 15553 & 8800 & 8800 & 111.19 & 111.61 & 109.21 & 110.2 & 501 & 504 & 99.86 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.883 & 0.998 & 0.00 & 23.58 & 94.33 & 8481 & 8481 & 8481 & 8481 & 8481 & 8481 & 8386.1 & 8336.5 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8592 & 8592 & 8592 & 0 & 0 & 0 & 1 & 8592 & 8592 & 8628 & 40.45 & 40.48 & 8592 & 8592 & 8592 & 8592 & 8592.0 & 8556.2 & 1 & 1 & 1 & 1 & 15570 & 8800 & 8800 & 111.63 & 111.61 & 110.22 & 110.2 & 504 & 504 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 11.64 & 162.94 & 104.75 & 8592 & 8592 & 8592 & 8592 & 8592 & 8592 & 8592.0 & 8556.2 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8628 & 40.94 & 40.48 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8582.1 & 1 & 1 & 1 & 1 & 15437 & 8800 & 8800 & 105.09 & 111.61 & 97.61 & 110.2 & 463 & 504 & 99.58 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.011 & 1.001 & 0.00 & 1695.52 & 521.70 & 8434 & 8434 & 8434 & 8434 & 8434 & 8434 & 8266.4 & 8244.4 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
