\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
Total length ($\geq$ 1000 bp) & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
Total length ($\geq$ 5000 bp) & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
Total length & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
Reference length & 8704 & 8704 & 8704 & 8704 \\ \hline
GC (\%) & 40.23 & 40.20 & - & 40.29 \\ \hline
Reference GC (\%) & 40.26 & 40.26 & 40.26 & 40.26 \\ \hline
N50 & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
NG50 & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
N90 & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
NG90 & 8683 & 8686 & 8624 & {\bf 8701} \\ \hline
auN & 8683.0 & 8686.0 & 8624.0 & {\bf 8701.0} \\ \hline
auNG & 8662.1 & 8668.0 & 8544.7 & {\bf 8698.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 115253 & 115266 & 380000 & 115518 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 310.37} & 310.32 & 0.0 & 309.62 \\ \hline
Reference mapped (\%) & 309.67 & 309.67 & 309.67 & 309.67 \\ \hline
Properly paired (\%) & {\bf 296.65} & 296.62 & 0.0 & 295.98 \\ \hline
Reference properly paired (\%) & 296.02 & 296.02 & 296.02 & 296.02 \\ \hline
Avg. coverage depth & {\bf 10254} & 10252 & 0 & 10250 \\ \hline
Reference avg. coverage depth & 10249 & 10249 & 10249 & 10249 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 0.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# misassembled contigs & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
Misassembled contigs length & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & - & 0 \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 1 + 0 part & {\bf 0 + 0 part} \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & 8624 & {\bf 0} \\ \hline
Genome fraction (\%) & {\bf 100.000} & {\bf 100.000} & - & {\bf 100.000} \\ \hline
Duplication ratio & {\bf 1.001} & {\bf 1.001} & - & 1.003 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 100000.00 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 57.58} & 506.56 & - & 264.34 \\ \hline
\# indels per 100 kbp & 80.62 & 69.08 & - & {\bf 11.49} \\ \hline
Largest alignment & 8683 & 8686 & - & {\bf 8701} \\ \hline
Total aligned length & 8683 & 8686 & - & {\bf 8701} \\ \hline
NA50 & 8683 & 8686 & - & {\bf 8701} \\ \hline
NGA50 & 8683 & 8686 & - & {\bf 8701} \\ \hline
NA90 & 8683 & 8686 & - & {\bf 8701} \\ \hline
NGA90 & 8683 & 8686 & - & {\bf 8701} \\ \hline
auNA & 8683.0 & 8686.0 & - & {\bf 8701.0} \\ \hline
auNGA & 8662.1 & 8668.0 & - & {\bf 8698.0} \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LA90 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA90 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
