\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8730} & 8639 & 2998 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8730} & 8639 & 2998 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8730} & 8639 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8730} & 8639 & 2998 & 8625 \\ \hline
Total length & {\bf 8730} & 8639 & 2998 & 8625 \\ \hline
Reference length & 8759 & 8759 & 8759 & 8759 \\ \hline
GC (\%) & 40.77 & 41.27 & 43.48 & 41.33 \\ \hline
Reference GC (\%) & 40.72 & 40.72 & 40.72 & 40.72 \\ \hline
N50 & {\bf 8730} & 8639 & 2998 & 8625 \\ \hline
NG50 & {\bf 8730} & 8639 & - & 8625 \\ \hline
N75 & {\bf 8730} & 8639 & 2998 & 8625 \\ \hline
NG75 & {\bf 8730} & 8639 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 19000 & 19000 & 19000 & 19000 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 94.36} & 50.6 & 13.79 & 18.22 \\ \hline
Reference mapped (\%) & 94.52 & 94.52 & 94.52 & 94.52 \\ \hline
Properly paired (\%) & {\bf 90.11} & 37.92 & 5.8 & 7.28 \\ \hline
Reference properly paired (\%) & 90.41 & 90.41 & 90.41 & 90.41 \\ \hline
Avg. coverage depth & {\bf 509} & 271 & 210 & 95 \\ \hline
Reference avg. coverage depth & 512 & 512 & 512 & 512 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 60.03 & 49.63 & 56.48 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 6 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 3124 & {\bf 0} & 7721 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 63.248 & 17.068 & 10.369 \\ \hline
Duplication ratio & 1.001 & {\bf 1.000} & 2.015 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 50366.91 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & 252.35 & {\bf 126.95} & 268.82 & 2433.63 \\ \hline
\# indels per 100 kbp & 217.94 & {\bf 90.68} & 336.02 & 442.48 \\ \hline
Largest alignment & {\bf 8730} & 5515 & 1488 & 698 \\ \hline
Total aligned length & {\bf 8730} & 5515 & 1488 & 904 \\ \hline
NA50 & {\bf 8730} & 5515 & - & - \\ \hline
NGA50 & {\bf 8730} & 5515 & - & - \\ \hline
NA75 & {\bf 8730} & - & - & - \\ \hline
NGA75 & {\bf 8730} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
