\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
Total length ($\geq$ 1000 bp) & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
Total length ($\geq$ 5000 bp) & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
Total length & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
Reference length & 8631 & 8631 & 8631 & 8631 \\ \hline
GC (\%) & 40.28 & 40.19 & 40.06 & 40.74 \\ \hline
Reference GC (\%) & 40.29 & 40.29 & 40.29 & 40.29 \\ \hline
N50 & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
NG50 & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
N90 & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
NG90 & 8592 & 8571 & 8491 & {\bf 8625} \\ \hline
auN & 8592.0 & 8571.0 & 8491.0 & {\bf 8625.0} \\ \hline
auNG & 8553.2 & 8511.4 & 8353.3 & {\bf 8619.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 92917 & 93784 & 94163 & 108054 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 371.86} & 366.15 & 364.57 & 295.45 \\ \hline
Reference mapped (\%) & 370.38 & 370.38 & 370.38 & 370.38 \\ \hline
Properly paired (\%) & {\bf 367.04} & 359.1 & 357.44 & 271.9 \\ \hline
Reference properly paired (\%) & 365.56 & 365.56 & 365.56 & 365.56 \\ \hline
Avg. coverage depth & 10027 & 9956 & {\bf 10069} & 9140 \\ \hline
Reference avg. coverage depth & 10014 & 10014 & 10014 & 10014 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.453 & 98.789 & 98.056 \\ \hline
Duplication ratio & 1.000 & {\bf 0.998} & 1.001 & 1.004 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 565.30 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 35.19 & 341.54 & 1572.48 \\ \hline
\# indels per 100 kbp & {\bf 116.39} & 129.05 & 117.77 & 425.63 \\ \hline
Largest alignment & {\bf 8592} & 8524 & 8491 & 8458 \\ \hline
Total aligned length & {\bf 8592} & 8524 & 8491 & 8458 \\ \hline
NA50 & {\bf 8592} & 8524 & 8491 & 8458 \\ \hline
NGA50 & {\bf 8592} & 8524 & 8491 & 8458 \\ \hline
NA90 & {\bf 8592} & 8524 & 8491 & 8458 \\ \hline
NGA90 & {\bf 8592} & 8524 & 8491 & 8458 \\ \hline
auNA & {\bf 8592.0} & 8477.3 & 8491.0 & 8294.2 \\ \hline
auNGA & {\bf 8553.2} & 8418.3 & 8353.3 & 8288.5 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
