\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8615 & 8615 & 8615 & 0 & 0 & 0 & 1 & 8615 & 8615 & 8654 & 40.57 & 40.52 & 8615 & 8615 & 8615 & 8615 & 8615.0 & 8576.2 & 1 & 1 & 1 & 1 & 98520 & 175000 & 175000 & 353.48 & 351.92 & 351.7 & 350.16 & 10063 & 10059 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 23.22 & 127.68 & 8615 & 8615 & 8615 & 8615 & 8615 & 8615 & 8615.0 & 8576.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8618 & 8618 & 8618 & 0 & 0 & 0 & 1 & 8618 & 8618 & 8654 & 40.42 & 40.52 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8582.1 & 1 & 1 & 1 & 1 & 98736 & 175000 & 175000 & 352.67 & 351.92 & 350.85 & 350.16 & 10055 & 10059 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 881.88 & 104.43 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8582.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8608 & 8608 & 8608 & 0 & 0 & 0 & 1 & 8608 & 8608 & 8654 & 40.17 & 40.52 & 8608 & 8608 & 8608 & 8608 & 8608.0 & 8562.2 & 1 & 1 & 1 & 1 & 116595 & 175000 & 175000 & 265.33 & 351.92 & 230.47 & 350.16 & 8875 & 10059 & 93.48 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 93.316 & 1.001 & 7005.11 & 2348.70 & 99.42 & 8047 & 8047 & 8047 & 8047 & 8047 & 8047 & 7522.6 & 7482.6 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8626 & 8626 & 8626 & 0 & 0 & 0 & 1 & 8626 & 8626 & 8654 & 40.86 & 40.52 & 8626 & 8626 & 8626 & 8626 & 8626.0 & 8598.1 & 1 & 1 & 1 & 1 & 112499 & 175000 & 175000 & 291.1 & 351.92 & 274.64 & 350.16 & 9361 & 10059 & 99.29 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.001 & 0.00 & 1590.62 & 417.97 & 8613 & 8613 & 8613 & 8613 & 8613 & 8613 & 8600.0 & 8572.2 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
