\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8645 & 40.24 & 40.23 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8603.1 & 1 & 1 & 1 & 1 & 99388 & 175000 & 175000 & 350.39 & 349.35 & 348.62 & 347.58 & 10068 & 10065 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 23.19 & 69.57 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8603.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8645 & 40.26 & 40.23 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8603.1 & 1 & 1 & 1 & 1 & 99452 & 175000 & 175000 & 350.09 & 349.35 & 348.25 & 347.58 & 10066 & 10065 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 487.01 & 69.57 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8603.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8275 & 8275 & 8275 & 0 & 0 & 0 & 1 & 8275 & 8275 & 8645 & 40.41 & 40.23 & 8275 & 8275 & 8275 & 8275 & 8275.0 & 7920.8 & 1 & 1 & 1 & 1 & 110701 & 175000 & 175000 & 290.68 & 349.35 & 265.19 & 347.58 & 9640 & 10065 & 97.1 & 100.0 & 0 & 0 & 0 & 0 & 0 & 2 & 0 & 0 & 0 + 0 part & 0 & 93.123 & 1.000 & 2924.47 & 1879.51 & 62.24 & 8034 & 8034 & 8034 & 8034 & 8034 & 8034 & 7800.0 & 7466.2 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8641 & 8641 & 8641 & 0 & 0 & 0 & 1 & 8641 & 8641 & 8645 & 40.55 & 40.23 & 8641 & 8641 & 8641 & 8641 & 8641.0 & 8637.0 & 1 & 1 & 1 & 1 & 108019 & 175000 & 175000 & 307.04 & 349.35 & 292.72 & 347.58 & 9466 & 10065 & 99.27 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.655 & 1.002 & 0.00 & 1489.74 & 516.13 & 8525 & 8525 & 8525 & 8525 & 8525 & 8525 & 8410.6 & 8406.7 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
