\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8701 & 40.70 & 40.70 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8671.0 & 1 & 1 & 1 & 1 & 16546 & 9500 & 9500 & 108.44 & 108.44 & 103.63 & 103.63 & 515 & 514 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 46.05 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8671.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8701 & 41.35 & 40.70 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8575.5 & 1 & 1 & 1 & 1 & 17080 & 9500 & 9500 & 58.82 & 108.44 & 42.88 & 103.63 & 283 & 514 & 63.08 & 100.0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 & 0 & 0 + 1 part & 2866 & 66.452 & 1.000 & 0.00 & 0.00 & 17.33 & 5772 & 5772 & 5772 & 5772 & - & - & 3856.9 & 3829.0 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7169 & 7169 & 7169 & 0 & 0 & 0 & 1 & 7169 & 7169 & 8701 & 41.00 & 40.70 & 7169 & 7169 & 7169 & - & 7169.0 & 5906.7 & 1 & 1 & 1 & - & 17174 & 9500 & 9500 & 60.68 & 108.44 & 47.78 & 103.63 & 357 & 514 & 75.27 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 63.781 & 1.000 & 22736.78 & 0.00 & 18.05 & 5540 & 5540 & 5540 & 5540 & - & - & 4281.2 & 3527.4 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8701 & 41.29 & 40.70 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8547.7 & 1 & 1 & 1 & 1 & 17831 & 9500 & 9500 & 24.37 & 108.44 & 10.6 & 103.63 & 119 & 514 & 57.26 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7470 & 13.286 & 1.000 & 0.00 & 1646.45 & 346.62 & 733 & 1154 & - & - & - & - & 72.6 & 71.9 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
