\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8680 & 8680 & 8680 & 0 & 0 & 0 & 1 & 8680 & 8680 & 8707 & 40.68 & 40.70 & 8680 & 8680 & 8680 & 8680 & 8680.0 & 8653.1 & 1 & 1 & 1 & 1 & 16703 & 9500 & 9500 & 107.33 & 107.25 & 102.68 & 102.62 & 514 & 514 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 11.52 & 92.17 & 8680 & 8680 & 8680 & 8680 & 8680 & 8680 & 8680.0 & 8653.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8707 & 41.19 & 40.70 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8569.5 & 1 & 1 & 1 & 1 & 17195 & 9500 & 9500 & 57.14 & 107.25 & 43.05 & 102.62 & 277 & 514 & 60.05 & 100.0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 & 0 & 0 + 1 part & 3120 & 63.549 & 1.000 & 0.00 & 18.12 & 36.25 & 5518 & 5518 & 5518 & 5518 & - & - & 3524.9 & 3497.0 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 0 & 0 & 0 & 0 & 4901 & 4901 & 0 & 0 & 0 & 0 & 1 & 4901 & 4901 & 8707 & 40.35 & 40.70 & 4901 & 4901 & 4901 & - & 4901.0 & 2758.7 & 1 & 1 & 1 & - & 17456 & 9500 & 9500 & 54.84 & 107.25 & 46.31 & 102.62 & 483 & 514 & 96.57 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 54.509 & 1.000 & 3427.87 & 42.26 & 21.13 & 4733 & 4733 & 4733 & 4733 & 4733 & - & 4570.8 & 2572.8 & 1 & 1 & 1 & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8707 & 41.35 & 40.70 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8541.8 & 1 & 1 & 1 & 1 & 18044 & 9500 & 9500 & 20.9 & 107.25 & 9.51 & 102.62 & 104 & 514 & 56.81 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7573 & 12.104 & 1.000 & 0.00 & 1902.95 & 380.59 & 845 & 1051 & - & - & - & - & 87.7 & 86.9 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
