\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8612 & 8612 & 8612 & 0 & 0 & 0 & 1 & 8612 & 8612 & 8656 & 40.22 & 40.24 & 8612 & 8612 & 8612 & 8612 & 8612.0 & 8568.2 & 1 & 1 & 1 & 1 & 15579 & 8800 & 8800 & 112.43 & 112.28 & 111.88 & 111.67 & 506 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 46.45 & 116.12 & 8612 & 8612 & 8612 & 8612 & 8612 & 8612 & 8612.0 & 8568.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8656 & 40.36 & 40.24 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8572.2 & 1 & 1 & 1 & 1 & 15573 & 8800 & 8800 & 112.3 & 112.28 & 111.58 & 111.67 & 505 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.932 & 1.000 & 0.00 & 234.69 & 93.87 & 8522 & 8522 & 8522 & 8522 & 8522 & 8522 & 8431.0 & 8390.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8108 & 8108 & 8108 & 0 & 0 & 0 & 1 & 8108 & 8108 & 8656 & 40.38 & 40.24 & 8108 & 8108 & 8108 & 8108 & 8108.0 & 7594.7 & 1 & 1 & 1 & 1 & 15219 & 8800 & 8800 & 62.05 & 112.28 & 30.24 & 111.67 & 280 & 505 & 66.55 & 100.0 & 0 & 0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 + 1 part & 773 & 53.316 & 1.002 & 34324.12 & 2434.25 & 86.94 & 4601 & 4601 & 4601 & 4601 & - & - & 2610.9 & 2445.6 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8612 & 8612 & 8612 & 0 & 0 & 0 & 1 & 8612 & 8612 & 8656 & 40.65 & 40.24 & 8612 & 8612 & 8612 & 8612 & 8612.0 & 8568.2 & 1 & 1 & 1 & 1 & 15409 & 8800 & 8800 & 104.97 & 112.28 & 96.76 & 111.67 & 462 & 505 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.014 & 1.000 & 0.00 & 2381.23 & 663.43 & 8441 & 8441 & 8441 & 8441 & 8441 & 8441 & 8273.4 & 8231.3 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
