\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
Total length & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
Reference length & 8743 & 8743 & 8743 & 8743 \\ \hline
GC (\%) & 40.72 & 40.68 & 40.58 & 40.65 \\ \hline
Reference GC (\%) & 40.67 & 40.67 & 40.67 & 40.67 \\ \hline
N50 & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
NG50 & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
N90 & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
NG90 & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
auN & {\bf 8728.0} & 8720.0 & 8717.0 & 8721.0 \\ \hline
auNG & {\bf 8713.0} & 8697.1 & 8691.1 & 8699.1 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 16730 & 16736 & 16739 & 16737 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 107.51} & 107.47 & 107.41 & 107.47 \\ \hline
Reference mapped (\%) & 107.44 & 107.44 & 107.44 & 107.44 \\ \hline
Properly paired (\%) & {\bf 103.13} & 103.1 & 103.04 & 103.1 \\ \hline
Reference properly paired (\%) & 103.07 & 103.07 & 103.07 & 103.07 \\ \hline
Avg. coverage depth & 512 & {\bf 513} & {\bf 513} & {\bf 513} \\ \hline
Reference avg. coverage depth & 513 & 513 & 513 & 513 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 99.977 & 99.977 & 99.943 & {\bf 99.989} \\ \hline
Duplication ratio & 1.001 & {\bf 1.000} & {\bf 1.000} & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 34.42 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & 57.29 & {\bf 0.00} & 355.63 & 126.13 \\ \hline
\# indels per 100 kbp & 126.03 & 68.81 & 68.83 & {\bf 68.80} \\ \hline
Largest alignment & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
Total aligned length & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
NA50 & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
NGA50 & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
NA90 & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
NGA90 & {\bf 8728} & 8720 & 8717 & 8721 \\ \hline
auNA & {\bf 8728.0} & 8720.0 & 8717.0 & 8721.0 \\ \hline
auNGA & {\bf 8713.0} & 8697.1 & 8691.1 & 8699.1 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
