\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
Total length & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
Reference length & 8737 & 8737 & 8737 & 8737 \\ \hline
GC (\%) & 40.72 & 40.73 & 40.71 & 40.73 \\ \hline
Reference GC (\%) & 40.71 & 40.71 & 40.71 & 40.71 \\ \hline
N50 & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
NG50 & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
N90 & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
NG90 & {\bf 8688} & 8686 & 8680 & 8681 \\ \hline
auN & {\bf 8688.0} & 8686.0 & 8680.0 & 8681.0 \\ \hline
auNG & {\bf 8639.3} & 8635.3 & 8623.4 & 8625.4 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 16684 & 16683 & 16684 & 16690 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & 107.22 & {\bf 107.25} & 107.23 & 107.2 \\ \hline
Reference mapped (\%) & 107.18 & 107.18 & 107.18 & 107.18 \\ \hline
Properly paired (\%) & 102.61 & {\bf 102.64} & {\bf 102.64} & 102.6 \\ \hline
Reference properly paired (\%) & 102.51 & 102.51 & 102.51 & 102.51 \\ \hline
Avg. coverage depth & 513 & 513 & 513 & 513 \\ \hline
Reference avg. coverage depth & 511 & 511 & 511 & 511 \\ \hline
Coverage $\geq$ 1x (\%) & 99.95 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 99.965 & {\bf 100.000} & {\bf 100.000} & {\bf 100.000} \\ \hline
Duplication ratio & 1.000 & 1.000 & {\bf 0.999} & 1.000 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & 11.52 & {\bf 11.51} & 46.08 & 23.04 \\ \hline
\# indels per 100 kbp & 126.67 & {\bf 115.13} & 126.73 & 126.71 \\ \hline
Largest alignment & 8684 & {\bf 8686} & 8680 & 8681 \\ \hline
Total aligned length & 8684 & {\bf 8686} & 8680 & 8681 \\ \hline
NA50 & 8684 & {\bf 8686} & 8680 & 8681 \\ \hline
NGA50 & 8684 & {\bf 8686} & 8680 & 8681 \\ \hline
NA90 & 8684 & {\bf 8686} & 8680 & 8681 \\ \hline
NGA90 & 8684 & {\bf 8686} & 8680 & 8681 \\ \hline
auNA & 8680.0 & {\bf 8686.0} & 8680.0 & 8681.0 \\ \hline
auNGA & 8631.3 & {\bf 8635.3} & 8623.4 & 8625.4 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
