\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
Total length & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
Reference length & 8695 & 8695 & 8695 & 8695 \\ \hline
GC (\%) & 40.40 & 40.44 & 40.11 & 40.89 \\ \hline
Reference GC (\%) & 40.37 & 40.37 & 40.37 & 40.37 \\ \hline
N50 & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
NG50 & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
N75 & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
NG75 & {\bf 8638} & {\bf 8638} & 8482 & 8620 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.6} & 99.39 & 94.74 & 90.95 \\ \hline
Reference mapped (\%) & 99.6 & 99.6 & 99.6 & 99.6 \\ \hline
Properly paired (\%) & {\bf 99.2} & 98.78 & 89.48 & 83.42 \\ \hline
Reference properly paired (\%) & 99.19 & 99.19 & 99.19 & 99.19 \\ \hline
Avg. coverage depth & {\bf 506} & 505 & 487 & 457 \\ \hline
Reference avg. coverage depth & 505 & 505 & 505 & 505 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.92 & 96.16 & 99.23 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & {\bf 0} & 1129 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.807 & 95.635 & 86.801 \\ \hline
Duplication ratio & 1.000 & 1.012 & 1.027 & {\bf 0.999} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 3277.53 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 34.73} & 58.59 & 290.56 & 2561.02 \\ \hline
\# indels per 100 kbp & {\bf 173.67} & 175.77 & 181.60 & 280.11 \\ \hline
Largest alignment & {\bf 8638} & 8535 & 8258 & 7491 \\ \hline
Total aligned length & {\bf 8638} & 8535 & 8258 & 7491 \\ \hline
NA50 & {\bf 8638} & 8535 & 8258 & 7491 \\ \hline
NGA50 & {\bf 8638} & 8535 & 8258 & 7491 \\ \hline
NA75 & {\bf 8638} & 8535 & 8258 & 7491 \\ \hline
NGA75 & {\bf 8638} & 8535 & 8258 & 7491 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
