\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{61}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N75 & NG75 & L50 & LG50 & L75 & LG75 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA75 & NGA75 & LA50 & LGA50 & LA75 & LGA75 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8622 & 8622 & 8622 & 0 & 0 & 0 & 1 & 8622 & 8622 & 8633 & 40.40 & 40.39 & 8622 & 8622 & 8622 & 8622 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 99.65 & 99.61 & 99.3 & 99.22 & 507 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 11.60 & 58.02 & 8619 & 8619 & 8619 & 8619 & 8619 & 8619 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8621 & 8621 & 8621 & 0 & 0 & 0 & 1 & 8621 & 8621 & 8633 & 40.48 & 40.39 & 8621 & 8621 & 8621 & 8621 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 99.31 & 99.61 & 98.61 & 99.22 & 505 & 507 & 99.92 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.654 & 1.014 & 0.00 & 164.69 & 47.05 & 8505 & 8505 & 8505 & 8505 & 8505 & 8505 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8234 & 8234 & 8234 & 0 & 0 & 0 & 1 & 8234 & 8234 & 8633 & 40.08 & 40.39 & 8234 & 8234 & 8234 & 8234 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 91.32 & 99.61 & 83.81 & 99.22 & 481 & 507 & 97.51 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 + 0 part & 0 & 94.139 & 1.015 & 4481.42 & 887.57 & 49.31 & 8115 & 8115 & 8115 & 8115 & 8115 & 8115 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8613 & 8613 & 8613 & 0 & 0 & 0 & 1 & 8613 & 8613 & 8633 & 40.72 & 40.39 & 8613 & 8613 & 8613 & 8613 & 1 & 1 & 1 & 1 & 17600 & 8800 & 8800 & 92.09 & 99.61 & 85.51 & 99.22 & 462 & 507 & 99.87 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.491 & 1.015 & 0.00 & 2969.25 & 671.62 & 8495 & 8495 & 8495 & 8495 & 8495 & 8495 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
