\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
Total length & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
Reference length & 8704 & 8704 & 8704 & 8704 \\ \hline
GC (\%) & 40.75 & 40.69 & 40.31 & 40.64 \\ \hline
Reference GC (\%) & 40.69 & 40.69 & 40.69 & 40.69 \\ \hline
N50 & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
NG50 & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
N90 & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
NG90 & {\bf 8687} & 8686 & 8591 & 8683 \\ \hline
auN & {\bf 8687.0} & 8686.0 & 8591.0 & 8683.0 \\ \hline
auNG & {\bf 8670.0} & 8668.0 & 8479.5 & 8662.1 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 16635 & 16644 & 16438 & 16644 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & 107.75 & {\bf 107.82} & 98.84 & 107.8 \\ \hline
Reference mapped (\%) & 107.82 & 107.82 & 107.82 & 107.82 \\ \hline
Properly paired (\%) & 103.04 & {\bf 103.23} & 85.07 & 103.2 \\ \hline
Reference properly paired (\%) & 103.23 & 103.23 & 103.23 & 103.23 \\ \hline
Avg. coverage depth & 512 & {\bf 514} & 467 & {\bf 514} \\ \hline
Reference avg. coverage depth & 514 & 514 & 514 & 514 \\ \hline
Coverage $\geq$ 1x (\%) & 99.95 & 100.0 & 96.04 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 99.965 & {\bf 100.000} & 94.992 & {\bf 100.000} \\ \hline
Duplication ratio & 1.000 & 1.000 & 1.000 & 1.000 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 5133.28 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & 23.03 & {\bf 0.00} & 387.83 & 138.20 \\ \hline
\# indels per 100 kbp & 161.23 & 69.08 & {\bf 60.60} & 80.62 \\ \hline
Largest alignment & 8683 & {\bf 8686} & 8251 & 8683 \\ \hline
Total aligned length & 8683 & {\bf 8686} & 8251 & 8683 \\ \hline
NA50 & 8683 & {\bf 8686} & 8251 & 8683 \\ \hline
NGA50 & 8683 & {\bf 8686} & 8251 & 8683 \\ \hline
NA90 & 8683 & {\bf 8686} & 8251 & 8683 \\ \hline
NGA90 & 8683 & {\bf 8686} & 8251 & 8683 \\ \hline
auNA & 8679.0 & {\bf 8686.0} & 7924.5 & 8683.0 \\ \hline
auNGA & 8662.1 & {\bf 8668.0} & 7821.6 & 8662.1 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
