\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8687 & 8687 & 8687 & 0 & 0 & 0 & 1 & 8687 & 8687 & 8704 & 40.75 & 40.69 & 8687 & 8687 & 8687 & 8687 & 8687.0 & 8670.0 & 1 & 1 & 1 & 1 & 16635 & 9500 & 9500 & 107.75 & 107.82 & 103.04 & 103.23 & 512 & 514 & 99.95 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.965 & 1.000 & 0.00 & 23.03 & 161.23 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683 & 8679.0 & 8662.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8704 & 40.69 & 40.69 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8668.0 & 1 & 1 & 1 & 1 & 16644 & 9500 & 9500 & 107.82 & 107.82 & 103.23 & 103.23 & 514 & 514 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 69.08 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8668.0 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8591 & 8591 & 8591 & 0 & 0 & 0 & 1 & 8591 & 8591 & 8704 & 40.31 & 40.69 & 8591 & 8591 & 8591 & 8591 & 8591.0 & 8479.5 & 1 & 1 & 1 & 1 & 16438 & 9500 & 9500 & 98.84 & 107.82 & 85.07 & 103.23 & 467 & 514 & 96.04 & 100.0 & 0 & 0 & 0 & 0 & 0 & 2 & 0 & 0 & 0 + 0 part & 0 & 94.992 & 1.000 & 5133.28 & 387.83 & 60.60 & 8251 & 8251 & 8251 & 8251 & 8251 & 8251 & 7924.5 & 7821.6 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8683 & 8683 & 8683 & 0 & 0 & 0 & 1 & 8683 & 8683 & 8704 & 40.64 & 40.69 & 8683 & 8683 & 8683 & 8683 & 8683.0 & 8662.1 & 1 & 1 & 1 & 1 & 16644 & 9500 & 9500 & 107.8 & 107.82 & 103.2 & 103.23 & 514 & 514 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 138.20 & 80.62 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683.0 & 8662.1 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
