\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{61}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N75 & NG75 & L50 & LG50 & L75 & LG75 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA75 & NGA75 & LA50 & LGA50 & LA75 & LGA75 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8609 & 8609 & 8609 & 0 & 0 & 0 & 1 & 8609 & 8609 & 8630 & 40.09 & 40.08 & 8609 & 8609 & 8609 & 8609 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 99.53 & 99.53 & 99.05 & 99.05 & 10086 & 10086 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 11.63 & 116.25 & 8609 & 8609 & 8609 & 8609 & 8609 & 8609 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8603 & 8603 & 8603 & 0 & 0 & 0 & 1 & 8603 & 8603 & 8630 & 40.09 & 40.08 & 8603 & 8603 & 8603 & 8603 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 99.53 & 99.53 & 99.05 & 99.05 & 10089 & 10086 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 46.50 & 104.63 & 8603 & 8603 & 8603 & 8603 & 8603 & 8603 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8615 & 8615 & 8615 & 0 & 0 & 0 & 1 & 8615 & 8615 & 8630 & 40.20 & 40.08 & 8615 & 8615 & 8615 & 8615 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 93.32 & 99.53 & 89.77 & 99.05 & 9386 & 10086 & 96.96 & 100.0 & 0 & 0 & 0 & 0 & 0 & 2 & 0 & 0 + 0 part & 0 & 96.954 & 1.033 & 3981.43 & 1966.43 & 59.95 & 8353 & 8353 & 8353 & 8353 & 8353 & 8353 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8630 & 40.13 & 40.08 & 8624 & 8624 & 8624 & 8624 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 99.53 & 99.53 & 99.05 & 99.05 & 10087 & 10086 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.003 & 0.00 & 11.63 & 23.25 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
