\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8716 & 8716 & 8716 & 0 & 0 & 0 & 1 & 8716 & 8716 & 8743 & 40.52 & 40.52 & 8716 & 8716 & 8716 & 8716 & 8716.0 & 8689.1 & 1 & 1 & 1 & 1 & 109917 & 190000 & 190000 & 326.27 & 325.22 & 312.65 & 311.91 & 10242 & 10236 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 68.86 & 103.29 & 8713 & 8713 & 8713 & 8713 & 8713 & 8713 & 8710.0 & 8683.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8719 & 8719 & 8719 & 0 & 0 & 0 & 1 & 8719 & 8719 & 8743 & 40.56 & 40.52 & 8719 & 8719 & 8719 & 8719 & 8719.0 & 8695.1 & 1 & 1 & 1 & 1 & 109955 & 190000 & 190000 & 326.17 & 325.22 & 312.82 & 311.91 & 10242 & 10236 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 91.75 & 80.28 & 8719 & 8719 & 8719 & 8719 & 8719 & 8719 & 8719.0 & 8695.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8718 & 8718 & 8718 & 0 & 0 & 0 & 1 & 8718 & 8718 & 8743 & 40.49 & 40.52 & 8718 & 8718 & 8718 & 8718 & 8718.0 & 8693.1 & 1 & 1 & 1 & 1 & 109869 & 190000 & 190000 & 326.35 & 325.22 & 312.89 & 311.91 & 10214 & 10236 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.989 & 1.000 & 1158.52 & 1250.29 & 80.29 & 8718 & 8718 & 8718 & 8718 & 8718 & 8718 & 8718.0 & 8693.1 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8731 & 8731 & 8731 & 0 & 0 & 0 & 1 & 8731 & 8731 & 8743 & 40.52 & 40.52 & 8731 & 8731 & 8731 & 8731 & 8731.0 & 8719.0 & 1 & 1 & 1 & 1 & 110121 & 190000 & 190000 & 325.68 & 325.22 & 312.35 & 311.91 & 10239 & 10236 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.002 & 0.00 & 0.00 & 45.81 & 8731 & 8731 & 8731 & 8731 & 8731 & 8731 & 8731.0 & 8719.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
