\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8500 & {\bf 8720} & 4732 & {\bf 8720} \\ \hline
Total length ($\geq$ 1000 bp) & 8500 & {\bf 8720} & 4732 & {\bf 8720} \\ \hline
Total length ($\geq$ 5000 bp) & 8500 & {\bf 8720} & 0 & {\bf 8720} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8500 & {\bf 8720} & 4732 & {\bf 8720} \\ \hline
Total length & 8500 & {\bf 8720} & 4732 & {\bf 8720} \\ \hline
Reference length & 8753 & 8753 & 8753 & 8753 \\ \hline
GC (\%) & - & 40.77 & 42.13 & 40.64 \\ \hline
Reference GC (\%) & 40.75 & 40.75 & 40.75 & 40.75 \\ \hline
N50 & 8500 & {\bf 8720} & 4732 & {\bf 8720} \\ \hline
NG50 & 8500 & {\bf 8720} & 4732 & {\bf 8720} \\ \hline
N90 & 8500 & {\bf 8720} & 4732 & {\bf 8720} \\ \hline
NG90 & 8500 & {\bf 8720} & - & {\bf 8720} \\ \hline
auN & 8500.0 & {\bf 8720.0} & 4732.0 & {\bf 8720.0} \\ \hline
auNG & 8254.3 & {\bf 8687.1} & 2558.2 & {\bf 8687.1} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 19000 & 16782 & 17522 & 16782 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & 0.0 & {\bf 106.66} & 38.69 & {\bf 106.66} \\ \hline
Reference mapped (\%) & 106.57 & 106.57 & 106.57 & 106.57 \\ \hline
Properly paired (\%) & 0.0 & {\bf 101.87} & 24.59 & {\bf 101.87} \\ \hline
Reference properly paired (\%) & 101.78 & 101.78 & 101.78 & 101.78 \\ \hline
Avg. coverage depth & 0 & {\bf 511} & 350 & {\bf 511} \\ \hline
Reference avg. coverage depth & 510 & 510 & 510 & 510 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 100.0 & 80.14 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 2 & 0 \\ \hline
\# structural variations & - & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 1 + 0 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & 8500 & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & - & 100.000 & 41.312 & 100.000 \\ \hline
Duplication ratio & - & 1.000 & 1.000 & 1.000 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & 21555.37 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 11.47 & 222.10 & 126.15 \\ \hline
\# indels per 100 kbp & - & 91.74 & 55.52 & 91.74 \\ \hline
Largest alignment & - & 8720 & 3602 & 8720 \\ \hline
Total aligned length & - & 8720 & 3602 & 8720 \\ \hline
NA50 & - & 8720 & 3602 & 8720 \\ \hline
NGA50 & - & 8720 & - & 8720 \\ \hline
NA90 & - & 8720 & - & 8720 \\ \hline
NGA90 & - & 8720 & - & 8720 \\ \hline
auNA & - & 8720.0 & 2741.8 & 8720.0 \\ \hline
auNGA & - & 8687.1 & 1482.3 & 8687.1 \\ \hline
LA50 & - & 1 & 1 & 1 \\ \hline
LGA50 & - & 1 & - & 1 \\ \hline
LA90 & - & 1 & - & 1 \\ \hline
LGA90 & - & 1 & - & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
