\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8595 & 8595 & 8595 & 0 & 0 & 0 & 1 & 8595 & 8595 & 8637 & 40.27 & 40.27 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8553.2 & 1 & 1 & 1 & 1 & 94001 & 175000 & 175000 & 367.38 & 365.54 & 362.43 & 360.6 & 10012 & 9991 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 23.27 & 93.08 & 8595 & 8595 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8553.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8574 & 8574 & 8574 & 0 & 0 & 0 & 1 & 8574 & 8574 & 8637 & 40.14 & 40.27 & 8574 & 8574 & 8574 & 8574 & 8574.0 & 8511.5 & 1 & 1 & 1 & 1 & 95315 & 175000 & 175000 & 360.02 & 365.54 & 352.83 & 360.6 & 9938 & 9991 & 99.93 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.453 & 0.998 & 0.00 & 422.19 & 93.82 & 8527 & 8527 & 8527 & 8527 & 8527 & 8527 & 8480.3 & 8418.4 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8551 & 8551 & 8551 & 0 & 0 & 0 & 1 & 8551 & 8551 & 8637 & 40.21 & 40.27 & 8551 & 8551 & 8551 & 8551 & 8551.0 & 8465.9 & 1 & 1 & 1 & 1 & 119944 & 175000 & 175000 & 249.98 & 365.54 & 208.42 & 360.6 & 8615 & 9991 & 93.87 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 94.399 & 1.001 & 7168.75 & 1552.87 & 86.27 & 8114 & 8114 & 8114 & 8114 & 8114 & 8114 & 7699.3 & 7622.7 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8634 & 8634 & 8634 & 0 & 0 & 0 & 1 & 8634 & 8634 & 8637 & 40.76 & 40.27 & 8634 & 8634 & 8634 & 8634 & 8634.0 & 8631.0 & 1 & 1 & 1 & 1 & 107514 & 175000 & 175000 & 298.03 & 365.54 & 274.82 & 360.6 & 9166 & 9991 & 99.77 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.451 & 1.005 & 0.00 & 1458.65 & 435.24 & 8501 & 8501 & 8501 & 8501 & 8501 & 8501 & 8370.0 & 8367.1 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
