\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8717} & 8633 & 4559 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8717} & 8633 & 4559 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8717} & 8633 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8717} & 8633 & 4559 & 8625 \\ \hline
Total length & {\bf 8717} & 8633 & 4559 & 8625 \\ \hline
Reference length & 8741 & 8741 & 8741 & 8741 \\ \hline
GC (\%) & 40.40 & 41.23 & 39.24 & 41.25 \\ \hline
Reference GC (\%) & 40.44 & 40.44 & 40.44 & 40.44 \\ \hline
N50 & {\bf 8717} & 8633 & 4559 & 8625 \\ \hline
NG50 & {\bf 8717} & 8633 & 4559 & 8625 \\ \hline
N75 & {\bf 8717} & 8633 & 4559 & 8625 \\ \hline
NG75 & {\bf 8717} & 8633 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 19000 & 19000 & 19000 & 19000 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 94.18} & 50.73 & 29.53 & 18.91 \\ \hline
Reference mapped (\%) & 94.16 & 94.16 & 94.16 & 94.16 \\ \hline
Properly paired (\%) & {\bf 90.19} & 37.19 & 11.07 & 7.06 \\ \hline
Reference properly paired (\%) & 90.15 & 90.15 & 90.15 & 90.15 \\ \hline
Avg. coverage depth & {\bf 511} & 272 & 297 & 98 \\ \hline
Reference avg. coverage depth & 510 & 510 & 510 & 510 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 59.97 & 67.67 & 56.43 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 6 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 3 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 3138 & {\bf 0} & 7788 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 63.081 & 35.440 & 9.657 \\ \hline
Duplication ratio & 1.000 & 0.999 & 1.475 & {\bf 0.994} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 32309.72 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 68.82} & 72.73 & 647.25 & 593.82 \\ \hline
\# indels per 100 kbp & 91.75 & 90.91 & {\bf 64.72} & 237.53 \\ \hline
Largest alignment & {\bf 8715} & 5495 & 3085 & 422 \\ \hline
Total aligned length & {\bf 8715} & 5495 & 3085 & 837 \\ \hline
NA50 & {\bf 8715} & 5495 & 3085 & - \\ \hline
NGA50 & {\bf 8715} & 5495 & - & - \\ \hline
NA75 & {\bf 8715} & - & - & - \\ \hline
NGA75 & {\bf 8715} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
