\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
Total length ($\geq$ 1000 bp) & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
Total length ($\geq$ 5000 bp) & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
Total length & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
Reference length & 8710 & 8710 & 8710 & 8710 \\ \hline
GC (\%) & 40.68 & 40.66 & 40.46 & 40.72 \\ \hline
Reference GC (\%) & 40.69 & 40.69 & 40.69 & 40.69 \\ \hline
N50 & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
NG50 & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
N90 & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
NG90 & 8653 & {\bf 8686} & {\bf 8686} & {\bf 8686} \\ \hline
auN & 8653.0 & {\bf 8686.0} & {\bf 8686.0} & {\bf 8686.0} \\ \hline
auNG & 8596.4 & {\bf 8662.1} & {\bf 8662.1} & {\bf 8662.1} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 16796 & 16833 & 16523 & 16832 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & 104.92 & {\bf 105.67} & 93.7 & 105.66 \\ \hline
Reference mapped (\%) & 105.67 & 105.67 & 105.67 & 105.67 \\ \hline
Properly paired (\%) & 99.21 & {\bf 100.96} & 74.99 & 100.94 \\ \hline
Reference properly paired (\%) & 100.96 & 100.96 & 100.96 & 100.96 \\ \hline
Avg. coverage depth & 504 & {\bf 510} & 438 & {\bf 510} \\ \hline
Reference avg. coverage depth & 509 & 509 & 509 & 509 \\ \hline
Coverage $\geq$ 1x (\%) & 99.97 & 100.0 & 91.96 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 4 & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 99.965 & {\bf 100.000} & 91.927 & {\bf 100.000} \\ \hline
Duplication ratio & {\bf 0.996} & 1.000 & 1.000 & 1.000 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 8623.07 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & 46.25 & {\bf 46.05} & 563.63 & 184.20 \\ \hline
\# indels per 100 kbp & 173.43 & {\bf 92.10} & 125.25 & {\bf 92.10} \\ \hline
Largest alignment & 8649 & {\bf 8686} & 7984 & {\bf 8686} \\ \hline
Total aligned length & 8649 & {\bf 8686} & 7984 & {\bf 8686} \\ \hline
NA50 & 8649 & {\bf 8686} & 7984 & {\bf 8686} \\ \hline
NGA50 & 8649 & {\bf 8686} & 7984 & {\bf 8686} \\ \hline
NA90 & 8649 & {\bf 8686} & 7984 & {\bf 8686} \\ \hline
NGA90 & 8649 & {\bf 8686} & 7984 & {\bf 8686} \\ \hline
auNA & 8645.0 & {\bf 8686.0} & 7338.7 & {\bf 8686.0} \\ \hline
auNGA & 8588.4 & {\bf 8662.1} & 7318.5 & {\bf 8662.1} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
