\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8592 & 8592 & 8592 & 0 & 0 & 0 & 1 & 8592 & 8592 & 8610 & 40.20 & 40.21 & 8592 & 8592 & 8592 & 8592 & 8592.0 & 8574.0 & 1 & 1 & 1 & 1 & 15458 & 8800 & 8800 & 112.27 & 112.26 & 110.69 & 110.67 & 503 & 503 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 11.64 & 69.83 & 8592 & 8592 & 8592 & 8592 & 8592 & 8592 & 8592.0 & 8574.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8595 & 8595 & 8595 & 0 & 0 & 0 & 1 & 8595 & 8595 & 8610 & 40.22 & 40.21 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8580.0 & 1 & 1 & 1 & 1 & 15458 & 8800 & 8800 & 112.19 & 112.26 & 110.53 & 110.67 & 503 & 503 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.453 & 1.000 & 0.00 & 105.29 & 58.49 & 8548 & 8548 & 8548 & 8548 & 8548 & 8548 & 8501.3 & 8486.4 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8159 & 8159 & 8159 & 0 & 0 & 0 & 1 & 8159 & 8159 & 8610 & 39.53 & 40.21 & 8159 & 8159 & 8159 & 8159 & 8159.0 & 7731.6 & 1 & 1 & 1 & 1 & 15179 & 8800 & 8800 & 89.81 & 112.26 & 71.73 & 110.67 & 407 & 503 & 89.83 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 86.198 & 1.000 & 11949.99 & 2119.90 & 40.51 & 7406 & 7406 & 7406 & 7406 & 7406 & - & 6722.5 & 6370.4 & 1 & 1 & 1 & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8599 & 8599 & 8599 & 0 & 0 & 0 & 1 & 8599 & 8599 & 8610 & 40.71 & 40.21 & 8599 & 8599 & 8599 & 8599 & 8599.0 & 8588.0 & 1 & 1 & 1 & 1 & 15323 & 8800 & 8800 & 102.17 & 112.26 & 91.67 & 110.67 & 447 & 503 & 99.9 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.813 & 1.001 & 0.00 & 2165.47 & 611.98 & 8497 & 8497 & 8497 & 8497 & 8497 & 8497 & 8396.2 & 8385.5 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
