\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8657 & 40.34 & 40.35 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8591.1 & 1 & 1 & 1 & 1 & 97882 & 175000 & 175000 & 355.86 & 354.75 & 354.14 & 353.04 & 10069 & 10061 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 104.36 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8591.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8627 & 8627 & 8627 & 0 & 0 & 0 & 1 & 8627 & 8627 & 8657 & 40.42 & 40.35 & 8627 & 8627 & 8627 & 8627 & 8627.0 & 8597.1 & 1 & 1 & 1 & 1 & 98148 & 175000 & 175000 & 354.29 & 354.75 & 351.97 & 353.04 & 10053 & 10061 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.000 & 0.00 & 336.66 & 92.87 & 8614 & 8614 & 8614 & 8614 & 8614 & 8614 & 8601.0 & 8571.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8505 & 8505 & 8505 & 0 & 0 & 0 & 1 & 8505 & 8505 & 8657 & 40.54 & 40.35 & 8505 & 8505 & 8505 & 8505 & 8505.0 & 8355.7 & 1 & 1 & 1 & 1 & 103392 & 175000 & 175000 & 323.08 & 354.75 & 307.65 & 353.04 & 9732 & 10061 & 97.68 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 96.268 & 1.000 & 3656.67 & 2599.90 & 96.29 & 8308 & 8308 & 8308 & 8308 & 8308 & 8308 & 8115.6 & 7973.1 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8657 & 40.70 & 40.35 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8619.0 & 1 & 1 & 1 & 1 & 105064 & 175000 & 175000 & 316.72 & 354.75 & 301.37 & 353.04 & 9534 & 10061 & 99.29 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 97.659 & 1.001 & 0.00 & 1209.10 & 308.20 & 8436 & 8436 & 8436 & 8436 & 8436 & 8436 & 8238.7 & 8220.6 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
