\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
Total length & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
Reference length & 8619 & 8619 & 8619 & 8619 \\ \hline
GC (\%) & 40.44 & 40.40 & 40.39 & 40.86 \\ \hline
Reference GC (\%) & 40.36 & 40.36 & 40.36 & 40.36 \\ \hline
N50 & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
NG50 & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
N90 & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
NG90 & {\bf 8612} & 8595 & 8589 & 8604 \\ \hline
auN & {\bf 8612.0} & 8595.0 & 8589.0 & 8604.0 \\ \hline
auNG & {\bf 8605.0} & 8571.1 & 8559.1 & 8589.0 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15575 & 15574 & 15575 & 15376 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 111.65} & 111.51 & {\bf 111.65} & 102.15 \\ \hline
Reference mapped (\%) & 111.61 & 111.61 & 111.61 & 111.61 \\ \hline
Properly paired (\%) & {\bf 110.3} & 110.02 & {\bf 110.3} & 91.68 \\ \hline
Reference properly paired (\%) & 110.26 & 110.26 & 110.26 & 110.26 \\ \hline
Avg. coverage depth & {\bf 504} & {\bf 504} & {\bf 504} & 449 \\ \hline
Reference avg. coverage depth & 504 & 504 & 504 & 504 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 99.35 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.883 & {\bf 100.000} & 98.010 \\ \hline
Duplication ratio & 1.002 & 1.000 & {\bf 0.999} & 1.001 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 186.28 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 69.67} & 70.60 & 69.86 & 2181.90 \\ \hline
\# indels per 100 kbp & 139.34 & {\bf 82.36} & 93.14 & 545.48 \\ \hline
Largest alignment & {\bf 8612} & 8499 & 8589 & 8433 \\ \hline
Total aligned length & {\bf 8612} & 8499 & 8589 & 8433 \\ \hline
NA50 & {\bf 8612} & 8499 & 8589 & 8433 \\ \hline
NGA50 & {\bf 8612} & 8499 & 8589 & 8433 \\ \hline
NA90 & {\bf 8612} & 8499 & 8589 & 8433 \\ \hline
NGA90 & {\bf 8612} & 8499 & 8589 & 8433 \\ \hline
auNA & {\bf 8612.0} & 8404.1 & 8589.0 & 8265.4 \\ \hline
auNGA & {\bf 8605.0} & 8380.7 & 8559.1 & 8251.0 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
