\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
Total length ($\geq$ 1000 bp) & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
Total length ($\geq$ 5000 bp) & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
Total length & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
Reference length & 8610 & 8610 & 8610 & 8610 \\ \hline
GC (\%) & 39.72 & 39.83 & 39.45 & 40.49 \\ \hline
Reference GC (\%) & 39.72 & 39.72 & 39.72 & 39.72 \\ \hline
N50 & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
NG50 & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
N90 & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
NG90 & 8570 & 8592 & 8601 & {\bf 8603} \\ \hline
auN & 8570.0 & 8592.0 & 8601.0 & {\bf 8603.0} \\ \hline
auNG & 8530.2 & 8574.0 & 8592.0 & {\bf 8596.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15530 & 15514 & 15305 & 15345 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 110.9 & {\bf 111.6} & 100.64 & 102.29 \\ \hline
Reference mapped (\%) & 111.72 & 111.72 & 111.72 & 111.72 \\ \hline
Properly paired (\%) & 109.13 & {\bf 109.76} & 88.47 & 92.16 \\ \hline
Reference properly paired (\%) & 110.1 & 110.1 & 110.1 & 110.1 \\ \hline
Avg. coverage depth & 499 & {\bf 502} & 440 & 448 \\ \hline
Reference avg. coverage depth & 503 & 503 & 503 & 503 \\ \hline
Coverage $\geq$ 1x (\%) & 99.9 & 99.93 & 93.38 & 99.08 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 3 & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 99.930} & 98.883 & 93.412 & 97.905 \\ \hline
Duplication ratio & {\bf 0.997} & 1.000 & 1.001 & 1.001 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 8057.20 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 105.13} & 153.01 & 410.86 & 2683.13 \\ \hline
\# indels per 100 kbp & 70.09 & 47.08 & {\bf 37.35} & 700.46 \\ \hline
Largest alignment & {\bf 8561} & 8496 & 8032 & 8423 \\ \hline
Total aligned length & {\bf 8561} & 8496 & 8032 & 8423 \\ \hline
NA50 & {\bf 8561} & 8496 & 8032 & 8423 \\ \hline
NGA50 & {\bf 8561} & 8496 & 8032 & 8423 \\ \hline
NA90 & {\bf 8561} & 8496 & 8032 & 8423 \\ \hline
NGA90 & {\bf 8561} & 8496 & 8032 & 8423 \\ \hline
auNA & {\bf 8552.0} & 8401.1 & 7500.6 & 8246.8 \\ \hline
auNGA & {\bf 8512.3} & 8383.5 & 7492.8 & 8240.1 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
