\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
Total length ($\geq$ 1000 bp) & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
Total length ($\geq$ 5000 bp) & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
Total length & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
Reference length & 8632 & 8632 & 8632 & 8632 \\ \hline
GC (\%) & 40.74 & 40.78 & 40.74 & 40.98 \\ \hline
Reference GC (\%) & 40.74 & 40.74 & 40.74 & 40.74 \\ \hline
N50 & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
NG50 & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
N90 & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
NG90 & 8614 & 8614 & 8614 & {\bf 8636} \\ \hline
auN & 8614.0 & 8614.0 & 8614.0 & {\bf 8636.0} \\ \hline
auNG & 8596.0 & 8596.0 & 8596.0 & {\bf 8640.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 78009 & 78072 & 78009 & 82344 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 446.48} & 445.98 & {\bf 446.48} & 414.6 \\ \hline
Reference mapped (\%) & 446.16 & 446.16 & 446.16 & 446.16 \\ \hline
Properly paired (\%) & {\bf 444.29} & 443.66 & {\bf 444.29} & 404.16 \\ \hline
Reference properly paired (\%) & 443.98 & 443.98 & 443.98 & 443.98 \\ \hline
Avg. coverage depth & {\bf 10098} & 10096 & {\bf 10098} & 9822 \\ \hline
Reference avg. coverage depth & 10088 & 10088 & 10088 & 10088 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & {\bf 100.000} & {\bf 100.000} & 98.851 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & {\bf 1.000} & 1.003 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 58.05 & {\bf 0.00} & 550.54 \\ \hline
\# indels per 100 kbp & {\bf 34.83} & {\bf 34.83} & {\bf 34.83} & 433.41 \\ \hline
Largest alignment & {\bf 8614} & {\bf 8614} & {\bf 8614} & 8537 \\ \hline
Total aligned length & {\bf 8614} & {\bf 8614} & {\bf 8614} & 8537 \\ \hline
NA50 & {\bf 8614} & {\bf 8614} & {\bf 8614} & 8537 \\ \hline
NGA50 & {\bf 8614} & {\bf 8614} & {\bf 8614} & 8537 \\ \hline
NA90 & {\bf 8614} & {\bf 8614} & {\bf 8614} & 8537 \\ \hline
NGA90 & {\bf 8614} & {\bf 8614} & {\bf 8614} & 8537 \\ \hline
auNA & {\bf 8614.0} & {\bf 8614.0} & {\bf 8614.0} & 8439.1 \\ \hline
auNGA & {\bf 8596.0} & {\bf 8596.0} & {\bf 8596.0} & 8443.0 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
