\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
Total length ($\geq$ 1000 bp) & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
Total length ($\geq$ 5000 bp) & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
Total length & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
Reference length & 8658 & 8658 & 8658 & 8658 \\ \hline
GC (\%) & 40.61 & 40.59 & 39.87 & 40.87 \\ \hline
Reference GC (\%) & 40.63 & 40.63 & 40.63 & 40.63 \\ \hline
N50 & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
NG50 & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
N75 & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
NG75 & 8631 & {\bf 8635} & 8366 & 8615 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.47} & {\bf 99.47} & 73.29 & 93.99 \\ \hline
Reference mapped (\%) & 99.45 & 99.45 & 99.45 & 99.45 \\ \hline
Properly paired (\%) & {\bf 98.94} & 98.93 & 54.24 & 88.78 \\ \hline
Reference properly paired (\%) & 98.91 & 98.91 & 98.91 & 98.91 \\ \hline
Avg. coverage depth & {\bf 505} & {\bf 505} & 377 & 472 \\ \hline
Reference avg. coverage depth & 505 & 505 & 505 & 505 \\ \hline
Coverage $\geq$ 1x (\%) & 99.99 & 99.99 & 80.95 & 99.27 \\ \hline
Reference coverage $\geq$ 1x (\%) & 99.99 & 99.99 & 99.99 & 99.99 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 4 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & {\bf 0} & 1097 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.768 & 78.181 & 87.138 \\ \hline
Duplication ratio & {\bf 1.000} & 1.003 & 1.240 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 20666.99 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 34.76} & 301.97 & 1911.96 & 2074.47 \\ \hline
\# indels per 100 kbp & 92.70 & 69.69 & {\bf 59.29} & 252.66 \\ \hline
Largest alignment & {\bf 8631} & 8614 & 6757 & 7518 \\ \hline
Total aligned length & {\bf 8631} & 8614 & 6757 & 7518 \\ \hline
NA50 & {\bf 8631} & 8614 & 6757 & 7518 \\ \hline
NGA50 & {\bf 8631} & 8614 & 6757 & 7518 \\ \hline
NA75 & {\bf 8631} & 8614 & 6757 & 7518 \\ \hline
NGA75 & {\bf 8631} & 8614 & 6757 & 7518 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
