\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8726} & 8643 & 1556 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8726} & 8643 & 1556 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8726} & 8643 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8726} & 8643 & 1556 & 8625 \\ \hline
Total length & {\bf 8726} & 8643 & 1556 & 8625 \\ \hline
Reference length & 8762 & 8762 & 8762 & 8762 \\ \hline
GC (\%) & 40.58 & 41.24 & 39.14 & 41.25 \\ \hline
Reference GC (\%) & 40.56 & 40.56 & 40.56 & 40.56 \\ \hline
N50 & {\bf 8726} & 8643 & 1556 & 8625 \\ \hline
NG50 & {\bf 8726} & 8643 & - & 8625 \\ \hline
N75 & {\bf 8726} & 8643 & 1556 & 8625 \\ \hline
NG75 & {\bf 8726} & 8643 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 19000 & 19000 & 19000 & 19000 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 94.05} & 49.61 & 16.35 & 15.04 \\ \hline
Reference mapped (\%) & 93.95 & 93.95 & 93.95 & 93.95 \\ \hline
Properly paired (\%) & {\bf 90.31} & 36.97 & 11.42 & 5.73 \\ \hline
Reference properly paired (\%) & 90.45 & 90.45 & 90.45 & 90.45 \\ \hline
Avg. coverage depth & {\bf 510} & 267 & 491 & 77 \\ \hline
Reference avg. coverage depth & 508 & 508 & 508 & 508 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 57.72 & 100.0 & 54.47 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 4 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 3504 & {\bf 0} & 7850 \\ \hline
Genome fraction (\%) & {\bf 99.874} & 58.819 & 17.809 & 8.870 \\ \hline
Duplication ratio & 1.000 & 1.000 & 1.000 & 1.000 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 68.76} & 175.13 & 385.60 & 1290.32 \\ \hline
\# indels per 100 kbp & 68.76 & 58.38 & 64.27 & {\bf 0.00} \\ \hline
Largest alignment & {\bf 8726} & 5139 & 1556 & 339 \\ \hline
Total aligned length & {\bf 8726} & 5139 & 1556 & 775 \\ \hline
NA50 & {\bf 8726} & 5139 & 1556 & - \\ \hline
NGA50 & {\bf 8726} & 5139 & - & - \\ \hline
NA75 & {\bf 8726} & - & 1556 & - \\ \hline
NGA75 & {\bf 8726} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & {\bf 1} & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
