\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8621 & 8621 & 8621 & 0 & 0 & 0 & 1 & 8621 & 8621 & 8660 & 40.53 & 40.53 & 8621 & 8621 & 8621 & 8621 & 8621.0 & 8582.2 & 1 & 1 & 1 & 1 & 88518 & 175000 & 175000 & 393.46 & 391.08 & 391.51 & 389.13 & 10077 & 10063 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.988 & 1.000 & 0.00 & 23.20 & 116.01 & 8620 & 8620 & 8620 & 8620 & 8620 & 8620 & 8619.0 & 8580.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8660 & 40.57 & 40.53 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8588.1 & 1 & 1 & 1 & 1 & 88589 & 175000 & 175000 & 393.11 & 391.08 & 391.14 & 389.13 & 10076 & 10063 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.000 & 0.00 & 104.52 & 104.52 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 8598.0 & 8562.3 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8530 & 8530 & 8530 & 0 & 0 & 0 & 1 & 8530 & 8530 & 8660 & 40.45 & 40.53 & 8530 & 8530 & 8530 & 8530 & 8530.0 & 8402.0 & 1 & 1 & 1 & 1 & 99052 & 175000 & 175000 & 331.45 & 391.08 & 312.79 & 389.13 & 9552 & 10063 & 97.69 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 97.529 & 1.001 & 2379.84 & 617.80 & 95.05 & 8417 & 8417 & 8417 & 8417 & 8417 & 8417 & 8305.5 & 8180.8 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8632 & 8632 & 8632 & 0 & 0 & 0 & 1 & 8632 & 8632 & 8660 & 40.87 & 40.53 & 8632 & 8632 & 8632 & 8632 & 8632.0 & 8604.1 & 1 & 1 & 1 & 1 & 95353 & 175000 & 175000 & 351.82 & 391.08 & 337.0 & 389.13 & 9616 & 10063 & 99.37 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.838 & 1.001 & 0.00 & 1137.15 & 417.73 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618 & 8604.0 & 8576.2 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
