\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8640 & 8638 & 6406 & {\bf 8650} \\ \hline
Total length ($\geq$ 1000 bp) & 8640 & 8638 & 6406 & {\bf 8650} \\ \hline
Total length ($\geq$ 5000 bp) & 8640 & 8638 & 6406 & {\bf 8650} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8640 & 8638 & 6406 & {\bf 8650} \\ \hline
Total length & 8640 & 8638 & 6406 & {\bf 8650} \\ \hline
Reference length & 8671 & 8671 & 8671 & 8671 \\ \hline
GC (\%) & 40.72 & 40.66 & 37.88 & 40.91 \\ \hline
Reference GC (\%) & 40.72 & 40.72 & 40.72 & 40.72 \\ \hline
N50 & 8640 & 8638 & 6406 & {\bf 8650} \\ \hline
NG50 & 8640 & 8638 & 6406 & {\bf 8650} \\ \hline
N75 & 8640 & 8638 & 6406 & {\bf 8650} \\ \hline
NG75 & 8640 & 8638 & - & {\bf 8650} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.63} & 99.52 & 28.77 & 92.44 \\ \hline
Reference mapped (\%) & 99.62 & 99.62 & 99.62 & 99.62 \\ \hline
Properly paired (\%) & {\bf 99.26} & 99.03 & 7.4 & 86.17 \\ \hline
Reference properly paired (\%) & 99.25 & 99.25 & 99.25 & 99.25 \\ \hline
Avg. coverage depth & {\bf 506} & 505 & 186 & 465 \\ \hline
Reference avg. coverage depth & 505 & 505 & 505 & 505 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.91 & 45.82 & 99.27 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 3 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 5 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.687 & 33.978 & 91.164 \\ \hline
Duplication ratio & {\bf 1.001} & 1.003 & 2.183 & 1.099 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 54214.80 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 34.74} & 383.36 & 818.00 & 1130.59 \\ \hline
\# indels per 100 kbp & 92.65 & 92.94 & {\bf 68.17} & 190.55 \\ \hline
Largest alignment & {\bf 8640} & 8611 & 2935 & 7874 \\ \hline
Total aligned length & {\bf 8640} & 8611 & 2935 & 7874 \\ \hline
NA50 & {\bf 8640} & 8611 & - & 7874 \\ \hline
NGA50 & {\bf 8640} & 8611 & - & 7874 \\ \hline
NA75 & {\bf 8640} & 8611 & - & 7874 \\ \hline
NGA75 & {\bf 8640} & 8611 & - & 7874 \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LA75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
