\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8611 & 8611 & 8611 & 0 & 0 & 0 & 1 & 8611 & 8611 & 8641 & 40.56 & 40.60 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8581.1 & 1 & 1 & 1 & 1 & 15609 & 8800 & 8800 & 112.22 & 112.13 & 111.68 & 111.59 & 507 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 104.52 & 81.29 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8581.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8641 & 40.65 & 40.60 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8587.1 & 1 & 1 & 1 & 1 & 15610 & 8800 & 8800 & 111.94 & 112.13 & 111.13 & 111.59 & 506 & 506 & 99.88 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.932 & 1.000 & 0.00 & 246.42 & 58.67 & 8522 & 8522 & 8522 & 8522 & 8522 & 8522 & 8431.0 & 8404.6 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8595 & 8595 & 8595 & 0 & 0 & 0 & 1 & 8595 & 8595 & 8641 & 40.29 & 40.60 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8549.2 & 1 & 1 & 1 & 1 & 15103 & 8800 & 8800 & 97.56 & 112.13 & 79.52 & 111.59 & 421 & 506 & 90.41 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 90.212 & 1.000 & 10890.05 & 308.84 & 77.21 & 7771 & 7771 & 7771 & 7771 & 7771 & - & 7026.0 & 6988.6 & 1 & 1 & 1 & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8641 & 40.91 & 40.60 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8587.1 & 1 & 1 & 1 & 1 & 15501 & 8800 & 8800 & 106.55 & 112.13 & 100.29 & 111.59 & 470 & 506 & 99.88 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.015 & 1.000 & 0.00 & 2439.89 & 568.52 & 8443 & 8443 & 8443 & 8443 & 8443 & 8443 & 8275.4 & 8249.5 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
