\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
Total length & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
Reference length & 8649 & 8649 & 8649 & 8649 \\ \hline
GC (\%) & 40.05 & 39.99 & 39.46 & 40.46 \\ \hline
Reference GC (\%) & 39.97 & 39.97 & 39.97 & 39.97 \\ \hline
N50 & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
NG50 & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
N75 & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
NG75 & {\bf 8612} & 8572 & 8602 & 8591 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 98.9} & 98.3 & 85.04 & 87.4 \\ \hline
Reference mapped (\%) & 98.86 & 98.86 & 98.86 & 98.86 \\ \hline
Properly paired (\%) & {\bf 97.81} & 96.59 & 70.44 & 77.78 \\ \hline
Reference properly paired (\%) & 97.73 & 97.73 & 97.73 & 97.73 \\ \hline
Avg. coverage depth & {\bf 504} & 501 & 428 & 440 \\ \hline
Reference avg. coverage depth & 503 & 503 & 503 & 503 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.95 & 91.91 & 98.64 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 3 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.314 & 91.844 & 99.314 \\ \hline
Duplication ratio & {\bf 1.002} & 1.004 & 1.090 & 1.006 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 9090.91 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 46.54} & 199.16 & 2774.26 & 2741.33 \\ \hline
\# indels per 100 kbp & 221.06 & {\bf 175.73} & 177.35 & 620.90 \\ \hline
Largest alignment & {\bf 8612} & 8515 & 7891 & 8540 \\ \hline
Total aligned length & {\bf 8612} & 8515 & 7891 & 8540 \\ \hline
NA50 & {\bf 8612} & 8515 & 7891 & 8540 \\ \hline
NGA50 & {\bf 8612} & 8515 & 7891 & 8540 \\ \hline
NA75 & {\bf 8612} & 8515 & 7891 & 8540 \\ \hline
NGA75 & {\bf 8612} & 8515 & 7891 & 8540 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
