\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8667 & 8667 & 8667 & 0 & 0 & 0 & 1 & 8667 & 8667 & 8707 & 40.31 & 40.41 & 8667 & 8667 & 8667 & 8667 & 8667.0 & 8627.2 & 1 & 1 & 1 & 1 & 110078 & 190000 & 190000 & 325.12 & 324.48 & 311.03 & 310.38 & 10259 & 10258 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 0.00 & 357.68 & 126.92 & 8667 & 8667 & 8667 & 8667 & 8667 & 8667 & 8667.0 & 8627.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8707 & 41.10 & 40.41 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8569.5 & 1 & 1 & 1 & 1 & 224138 & 190000 & 190000 & 86.17 & 324.48 & 63.17 & 310.38 & 5469 & 10258 & 60.55 & 100.0 & 0 & 0 & 0 & 5 & 0 & 0 & 0 & 0 & 0 + 1 part & 3290 & 61.570 & 1.001 & 0.00 & 729.24 & 93.49 & 5348 & 5348 & 5348 & 5348 & - & - & 3311.1 & 3284.8 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 0 & 0 & 0 & 0 & 3620 & 3620 & 0 & 0 & 0 & 0 & 1 & 3620 & 3620 & 8707 & 39.68 & 40.41 & 3620 & - & 3620 & - & 3620.0 & 1505.0 & 1 & - & 1 & - & 268589 & 190000 & 190000 & 51.6 & 324.48 & 41.13 & 310.38 & 9388 & 10258 & 97.15 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 40.332 & 1.001 & 4613.26 & 2083.93 & 57.09 & 3503 & 3503 & 3503 & - & 3503 & - & 3389.8 & 1409.3 & 1 & - & 1 & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8707 & 41.28 & 40.41 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8541.8 & 1 & 1 & 1 & 1 & 325485 & 190000 & 190000 & 19.72 & 324.48 & 8.38 & 310.38 & 1768 & 10258 & 56.54 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7567 & 12.187 & 1.000 & 0.00 & 2459.79 & 378.43 & 851 & 1057 & - & - & - & - & 88.9 & 88.0 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
