\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8639 & 40.78 & 40.78 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8609.0 & 1 & 1 & 1 & 1 & 81454 & 175000 & 175000 & 427.72 & 427.32 & 425.74 & 425.35 & 10086 & 10077 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 46.38 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8609.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8639 & 40.83 & 40.78 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8609.0 & 1 & 1 & 1 & 1 & 81739 & 175000 & 175000 & 425.66 & 427.32 & 423.13 & 425.35 & 10073 & 10077 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.000 & 0.00 & 220.65 & 46.45 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 8598.0 & 8583.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8623 & 8623 & 8623 & 0 & 0 & 0 & 1 & 8623 & 8623 & 8639 & 40.67 & 40.78 & 8623 & 8623 & 8623 & 8623 & 8623.0 & 8607.0 & 1 & 1 & 1 & 1 & 81472 & 175000 & 175000 & 427.59 & 427.32 & 425.59 & 425.35 & 10086 & 10077 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.988 & 1.000 & 0.00 & 301.52 & 46.39 & 8623 & 8623 & 8623 & 8623 & 8623 & 8623 & 8623.0 & 8607.0 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8639 & 41.02 & 40.78 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8637.0 & 1 & 1 & 1 & 1 & 90608 & 175000 & 175000 & 368.93 & 427.32 & 352.68 & 425.35 & 9608 & 10077 & 99.37 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.002 & 0.00 & 1089.86 & 382.61 & 8625 & 8625 & 8625 & 8625 & 8625 & 8625 & 8612.0 & 8611.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
