\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
Total length & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
Reference length & 8643 & 8643 & 8643 & 8643 \\ \hline
GC (\%) & 39.98 & 40.01 & 39.86 & 40.51 \\ \hline
Reference GC (\%) & 39.92 & 39.92 & 39.92 & 39.92 \\ \hline
N50 & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
NG50 & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
N75 & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
NG75 & {\bf 8607} & 8585 & 8571 & 8596 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 98.73} & 98.07 & 89.02 & 87.34 \\ \hline
Reference mapped (\%) & 98.73 & 98.73 & 98.73 & 98.73 \\ \hline
Properly paired (\%) & {\bf 97.45} & 96.14 & 78.05 & 76.43 \\ \hline
Reference properly paired (\%) & 97.45 & 97.45 & 97.45 & 97.45 \\ \hline
Avg. coverage depth & {\bf 503} & 499 & 451 & 438 \\ \hline
Reference avg. coverage depth & 502 & 502 & 502 & 502 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 94.32 & 98.67 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 4 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & {\bf 0} & 2534 \\ \hline
Genome fraction (\%) & {\bf 99.861} & 98.815 & 94.933 & 70.444 \\ \hline
Duplication ratio & 1.002 & 1.010 & 1.049 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 6043.64 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 46.55} & 105.86 & 171.40 & 758.95 \\ \hline
\# indels per 100 kbp & 139.66 & 129.38 & {\bf 122.43} & 148.49 \\ \hline
Largest alignment & {\bf 8591} & 8484 & 8171 & 6062 \\ \hline
Total aligned length & {\bf 8591} & 8484 & 8171 & 6062 \\ \hline
NA50 & {\bf 8591} & 8484 & 8171 & 6062 \\ \hline
NGA50 & {\bf 8591} & 8484 & 8171 & 6062 \\ \hline
NA75 & {\bf 8591} & 8484 & 8171 & - \\ \hline
NGA75 & {\bf 8591} & 8484 & 8171 & - \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA75 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
