\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8720} & 8639 & 3002 & 8628 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8720} & 8639 & 3002 & 8628 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8720} & 8639 & 0 & 8628 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8720} & 8639 & 3002 & 8628 \\ \hline
Total length & {\bf 8720} & 8639 & 3002 & 8628 \\ \hline
Reference length & 8749 & 8749 & 8749 & 8749 \\ \hline
GC (\%) & 40.71 & 41.28 & 43.28 & 41.43 \\ \hline
Reference GC (\%) & 40.72 & 40.72 & 40.72 & 40.72 \\ \hline
N50 & {\bf 8720} & 8639 & 3002 & 8628 \\ \hline
NG50 & {\bf 8720} & 8639 & - & 8628 \\ \hline
N75 & {\bf 8720} & 8639 & 3002 & 8628 \\ \hline
NG75 & {\bf 8720} & 8639 & - & 8628 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 380000 & 380000 & 380000 & 380000 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 94.38} & 54.29 & 12.13 & 22.46 \\ \hline
Reference mapped (\%) & 94.4 & 94.4 & 94.4 & 94.4 \\ \hline
Properly paired (\%) & {\bf 90.33} & 39.82 & 4.23 & 9.9 \\ \hline
Reference properly paired (\%) & 90.37 & 90.37 & 90.37 & 90.37 \\ \hline
Avg. coverage depth & {\bf 10246} & 5794 & 3688 & 2353 \\ \hline
Reference avg. coverage depth & 10237 & 10237 & 10237 & 10237 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 66.14 & 43.4 & 60.1 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 7 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 2789 & {\bf 0} & 7003 \\ \hline
Genome fraction (\%) & {\bf 99.943} & 67.049 & 14.939 & 18.585 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 2.304 & 1.002 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 56595.60 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 68.40 & 844.21 & 1665.64 \\ \hline
\# indels per 100 kbp & 80.30 & {\bf 68.40} & 153.49 & 308.45 \\ \hline
Largest alignment & {\bf 8720} & 5850 & 1303 & 861 \\ \hline
Total aligned length & {\bf 8720} & 5850 & 1303 & 1625 \\ \hline
NA50 & {\bf 8720} & 5850 & - & - \\ \hline
NGA50 & {\bf 8720} & 5850 & - & - \\ \hline
NA75 & {\bf 8720} & - & - & - \\ \hline
NGA75 & {\bf 8720} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
