\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8717} & 8639 & 2996 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8717} & 8639 & 2996 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8717} & 8639 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8717} & 8639 & 2996 & 8625 \\ \hline
Total length & {\bf 8717} & 8639 & 2996 & 8625 \\ \hline
Reference length & 8729 & 8729 & 8729 & 8729 \\ \hline
GC (\%) & 40.75 & 41.44 & 43.72 & 41.29 \\ \hline
Reference GC (\%) & 40.77 & 40.77 & 40.77 & 40.77 \\ \hline
N50 & {\bf 8717} & 8639 & 2996 & 8625 \\ \hline
NG50 & {\bf 8717} & 8639 & - & 8625 \\ \hline
N75 & {\bf 8717} & 8639 & 2996 & 8625 \\ \hline
NG75 & {\bf 8717} & 8639 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 19000 & 19000 & 19000 & 19000 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 94.2} & 52.66 & 14.31 & 20.24 \\ \hline
Reference mapped (\%) & 94.2 & 94.2 & 94.2 & 94.2 \\ \hline
Properly paired (\%) & {\bf 89.96} & 39.03 & 6.44 & 9.38 \\ \hline
Reference properly paired (\%) & 89.96 & 89.96 & 89.96 & 89.96 \\ \hline
Avg. coverage depth & {\bf 511} & 281 & 219 & 105 \\ \hline
Reference avg. coverage depth & 511 & 511 & 511 & 511 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 63.11 & 50.77 & 56.96 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 6 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 3094 & {\bf 0} & 7568 \\ \hline
Genome fraction (\%) & {\bf 99.989} & 63.589 & 17.443 & 12.122 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 1.970 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 49232.31 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & {\bf 0.00} & {\bf 0.00} & 1797.54 \\ \hline
\# indels per 100 kbp & 45.88 & 36.07 & {\bf 0.00} & 473.04 \\ \hline
Largest alignment & {\bf 8717} & 5545 & 1521 & 851 \\ \hline
Total aligned length & {\bf 8717} & 5545 & 1521 & 1057 \\ \hline
NA50 & {\bf 8717} & 5545 & 1521 & - \\ \hline
NGA50 & {\bf 8717} & 5545 & - & - \\ \hline
NA75 & {\bf 8717} & - & - & - \\ \hline
NGA75 & {\bf 8717} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
