\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
Total length ($\geq$ 1000 bp) & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
Total length ($\geq$ 5000 bp) & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
Total length & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
Reference length & 8635 & 8635 & 8635 & 8635 \\ \hline
GC (\%) & 40.57 & 40.57 & 40.53 & 40.62 \\ \hline
Reference GC (\%) & 40.57 & 40.57 & 40.57 & 40.57 \\ \hline
N50 & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
NG50 & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
N90 & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
NG90 & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
auN & 8605.0 & 8605.0 & 8608.0 & {\bf 8614.0} \\ \hline
auNG & 8575.1 & 8575.1 & 8581.1 & {\bf 8593.1} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 87059 & 87059 & 87110 & 87187 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 400.21} & 400.2 & 399.96 & 399.62 \\ \hline
Reference mapped (\%) & 398.82 & 398.82 & 398.82 & 398.82 \\ \hline
Properly paired (\%) & {\bf 398.39} & 398.37 & 398.14 & 397.8 \\ \hline
Reference properly paired (\%) & 397.01 & 397.01 & 397.01 & 397.01 \\ \hline
Avg. coverage depth & {\bf 10101} & {\bf 10101} & 10100 & 10099 \\ \hline
Reference avg. coverage depth & 10095 & 10095 & 10095 & 10095 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 100.000 & 100.000 & 100.000 & 100.000 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & {\bf 1.000} & 1.001 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 23.24 & 127.79 & 69.65 \\ \hline
\# indels per 100 kbp & 116.21 & 116.21 & 104.55 & {\bf 81.26} \\ \hline
Largest alignment & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
Total aligned length & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
NA50 & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
NGA50 & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
NA90 & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
NGA90 & 8605 & 8605 & 8608 & {\bf 8614} \\ \hline
auNA & 8605.0 & 8605.0 & 8608.0 & {\bf 8614.0} \\ \hline
auNGA & 8575.1 & 8575.1 & 8581.1 & {\bf 8593.1} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
