\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
Total length ($\geq$ 1000 bp) & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
Total length ($\geq$ 5000 bp) & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
Total length & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
Reference length & 8704 & 8704 & 8704 & 8704 \\ \hline
GC (\%) & 40.38 & 41.18 & - & 41.25 \\ \hline
Reference GC (\%) & 40.27 & 40.27 & 40.27 & 40.27 \\ \hline
N50 & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
NG50 & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
N90 & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
NG90 & 8623 & {\bf 8638} & 8624 & 8624 \\ \hline
auN & 8623.0 & {\bf 8638.0} & 8624.0 & 8624.0 \\ \hline
auNG & 8542.8 & {\bf 8572.5} & 8544.7 & 8544.7 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 16790 & 17185 & 19000 & 18279 \\ \hline
\# left & 9500 & 9500 & 9500 & 9500 \\ \hline
\# right & 9500 & 9500 & 9500 & 9500 \\ \hline
Mapped (\%) & {\bf 103.78} & 56.39 & 0.0 & 15.28 \\ \hline
Reference mapped (\%) & 105.72 & 105.72 & 105.72 & 105.72 \\ \hline
Properly paired (\%) & {\bf 96.72} & 40.91 & 0.0 & 5.69 \\ \hline
Reference properly paired (\%) & 101.13 & 101.13 & 101.13 & 101.13 \\ \hline
Avg. coverage depth & {\bf 502} & 274 & 0 & 76 \\ \hline
Reference avg. coverage depth & 511 & 511 & 511 & 511 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 60.69 & 0.0 & 52.76 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & {\bf 0} & {\bf 0} & - & - \\ \hline
\# misassembled contigs & {\bf 0} & {\bf 0} & - & - \\ \hline
Misassembled contigs length & {\bf 0} & {\bf 0} & - & - \\ \hline
\# local misassemblies & {\bf 0} & 5 & - & - \\ \hline
\# scaffold gap ext. mis. & {\bf 0} & {\bf 0} & - & - \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & - & - \\ \hline
\# structural variations & 0 & 0 & - & - \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & - & - \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & 1 + 0 part & 1 + 0 part \\ \hline
Unaligned length & {\bf 0} & 3240 & 8624 & 8624 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 62.150 & - & - \\ \hline
Duplication ratio & {\bf 0.994} & 1.001 & - & - \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 100000.00 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 104.37} & 611.34 & - & - \\ \hline
\# indels per 100 kbp & 81.18 & {\bf 74.10} & - & - \\ \hline
Largest alignment & {\bf 8623} & 5398 & - & - \\ \hline
Total aligned length & {\bf 8623} & 5398 & - & - \\ \hline
NA50 & {\bf 8623} & 5398 & - & - \\ \hline
NGA50 & {\bf 8623} & 5398 & - & - \\ \hline
NA90 & {\bf 8623} & - & - & - \\ \hline
NGA90 & {\bf 8623} & - & - & - \\ \hline
auNA & {\bf 8623.0} & 3373.3 & - & - \\ \hline
auNGA & {\bf 8542.8} & 3347.7 & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA90 & {\bf 1} & - & - & - \\ \hline
LGA90 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
