\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8623 & 8623 & 8623 & 0 & 0 & 0 & 1 & 8623 & 8623 & 8704 & 40.38 & 40.27 & 8623 & 8623 & 8623 & 8623 & 8623.0 & 8542.8 & 1 & 1 & 1 & 1 & 16790 & 9500 & 9500 & 103.78 & 105.72 & 96.72 & 101.13 & 502 & 511 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.994 & 0.00 & 104.37 & 81.18 & 8623 & 8623 & 8623 & 8623 & 8623 & 8623 & 8623.0 & 8542.8 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8704 & 41.18 & 40.27 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8572.5 & 1 & 1 & 1 & 1 & 17185 & 9500 & 9500 & 56.39 & 105.72 & 40.91 & 101.13 & 274 & 511 & 60.69 & 100.0 & 0 & 0 & 0 & 5 & 0 & 0 & 0 & 0 & 0 + 1 part & 3240 & 62.150 & 1.001 & 0.00 & 611.34 & 74.10 & 5398 & 5398 & 5398 & 5398 & - & - & 3373.3 & 3347.7 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8704 & - & 40.27 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8544.7 & 1 & 1 & 1 & 1 & 19000 & 9500 & 9500 & 0.0 & 105.72 & 0.0 & 101.13 & 0 & 511 & 0.0 & 100.0 & - & - & - & - & - & - & - & - & 1 + 0 part & 8624 & - & - & 100000.00 & - & - & - & - & - & - & - & - & - & - & - & - & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8704 & 41.25 & 40.27 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8544.7 & 1 & 1 & 1 & 1 & 18279 & 9500 & 9500 & 15.28 & 105.72 & 5.69 & 101.13 & 76 & 511 & 52.76 & 100.0 & - & - & - & - & - & - & - & - & 1 + 0 part & 8624 & - & - & 0.00 & - & - & - & - & - & - & - & - & - & - & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
