\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
Total length ($\geq$ 1000 bp) & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
Total length ($\geq$ 5000 bp) & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
Total length & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
Reference length & 8624 & 8624 & 8624 & 8624 \\ \hline
GC (\%) & 40.80 & 40.84 & 40.81 & 41.18 \\ \hline
Reference GC (\%) & 40.80 & 40.80 & 40.80 & 40.80 \\ \hline
N50 & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
NG50 & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
N90 & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
NG90 & 8624 & 8624 & 8622 & {\bf 8632} \\ \hline
auN & 8624.0 & 8624.0 & 8622.0 & {\bf 8632.0} \\ \hline
auNG & 8624.0 & 8624.0 & 8620.0 & {\bf 8640.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15357 & 15354 & 15357 & 15285 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 114.04} & 113.89 & {\bf 114.04} & 109.7 \\ \hline
Reference mapped (\%) & 114.04 & 114.04 & 114.04 & 114.04 \\ \hline
Properly paired (\%) & {\bf 113.47} & 113.16 & {\bf 113.47} & 105.07 \\ \hline
Reference properly paired (\%) & 113.47 & 113.47 & 113.47 & 113.47 \\ \hline
Avg. coverage depth & {\bf 507} & 506 & {\bf 507} & 480 \\ \hline
Reference avg. coverage depth & 507 & 507 & 507 & 507 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.93 & 100.0 & 99.19 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.849 & 99.977 & 99.849 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & {\bf 1.000} & 1.001 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 162.58 & {\bf 0.00} & 1496.69 \\ \hline
\# indels per 100 kbp & {\bf 0.00} & {\bf 0.00} & {\bf 0.00} & 406.08 \\ \hline
Largest alignment & {\bf 8624} & 8611 & 8622 & 8619 \\ \hline
Total aligned length & {\bf 8624} & 8611 & 8622 & 8619 \\ \hline
NA50 & {\bf 8624} & 8611 & 8622 & 8619 \\ \hline
NGA50 & {\bf 8624} & 8611 & 8622 & 8619 \\ \hline
NA90 & {\bf 8624} & 8611 & 8622 & 8619 \\ \hline
NGA90 & {\bf 8624} & 8611 & 8622 & 8619 \\ \hline
auNA & {\bf 8624.0} & 8598.0 & 8622.0 & 8606.0 \\ \hline
auNGA & {\bf 8624.0} & 8598.0 & 8620.0 & 8614.0 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
