\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
Total length ($\geq$ 1000 bp) & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
Total length ($\geq$ 5000 bp) & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
Total length & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
Reference length & 8635 & 8635 & 8635 & 8635 \\ \hline
GC (\%) & 40.34 & 40.34 & - & 40.47 \\ \hline
Reference GC (\%) & 40.30 & 40.30 & 40.30 & 40.30 \\ \hline
N50 & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
NG50 & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
N90 & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
NG90 & 8617 & 8617 & 8624 & {\bf 8643} \\ \hline
auN & 8617.0 & 8617.0 & 8624.0 & {\bf 8643.0} \\ \hline
auNG & 8599.0 & 8599.0 & 8613.0 & {\bf 8651.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 98665 & 98608 & 350000 & 103229 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 352.92} & 352.7 & 0.0 & 328.49 \\ \hline
Reference mapped (\%) & 352.32 & 352.32 & 352.32 & 352.32 \\ \hline
Properly paired (\%) & {\bf 351.11} & 350.46 & 0.0 & 318.56 \\ \hline
Reference properly paired (\%) & 350.5 & 350.5 & 350.5 & 350.5 \\ \hline
Avg. coverage depth & {\bf 10087} & 10074 & 0 & 9723 \\ \hline
Reference avg. coverage depth & 10079 & 10079 & 10079 & 10079 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 0.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# misassembled contigs & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
Misassembled contigs length & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & - & 0 \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 1 + 0 part & {\bf 0 + 0 part} \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & 8624 & {\bf 0} \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.932 & - & 98.061 \\ \hline
Duplication ratio & {\bf 1.001} & {\bf 1.001} & - & 1.004 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 100000.00 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 34.81} & 222.87 & - & 731.48 \\ \hline
\# indels per 100 kbp & 69.63 & {\bf 58.65} & - & 519.11 \\ \hline
Largest alignment & {\bf 8617} & 8525 & - & 8476 \\ \hline
Total aligned length & {\bf 8617} & 8525 & - & 8476 \\ \hline
NA50 & {\bf 8617} & 8525 & - & 8476 \\ \hline
NGA50 & {\bf 8617} & 8525 & - & 8476 \\ \hline
NA90 & {\bf 8617} & 8525 & - & 8476 \\ \hline
NGA90 & {\bf 8617} & 8525 & - & 8476 \\ \hline
auNA & {\bf 8617.0} & 8434.0 & - & 8312.2 \\ \hline
auNGA & {\bf 8599.0} & 8416.4 & - & 8319.9 \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LA90 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA90 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
