\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8685 & 8685 & 8685 & 0 & 0 & 0 & 1 & 8685 & 8685 & 8707 & 40.61 & 40.62 & 8685 & 8685 & 8685 & 8685 & 8685.0 & 8663.1 & 1 & 1 & 1 & 1 & 16634 & 9500 & 9500 & 107.57 & 107.71 & 103.15 & 103.08 & 513 & 513 & 99.95 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.965 & 1.000 & 0.00 & 0.00 & 57.60 & 8681 & 8681 & 8681 & 8681 & 8681 & 8681 & 8677.0 & 8655.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8707 & 41.29 & 40.62 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8569.5 & 1 & 1 & 1 & 1 & 17158 & 9500 & 9500 & 56.36 & 107.71 & 42.73 & 103.08 & 275 & 513 & 57.65 & 100.0 & 0 & 0 & 0 & 5 & 0 & 0 & 0 & 0 & 0 + 1 part & 3333 & 61.096 & 1.000 & 0.00 & 18.85 & 0.00 & 5305 & 5305 & 5305 & 5305 & - & - & 3258.0 & 3232.2 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 0 & 0 & 0 & 0 & 4892 & 4892 & 0 & 0 & 0 & 0 & 1 & 4892 & 4892 & 8707 & 40.43 & 40.62 & 4892 & 4892 & 4892 & - & 4892.0 & 2748.6 & 1 & 1 & 1 & - & 17373 & 9500 & 9500 & 56.12 & 107.71 & 47.98 & 103.08 & 493 & 513 & 96.63 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 54.440 & 1.000 & 3372.85 & 232.71 & 21.16 & 4727 & 4727 & 4727 & 4727 & 4727 & - & 4567.6 & 2566.3 & 1 & 1 & 1 & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8707 & 41.29 & 40.62 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8541.8 & 1 & 1 & 1 & 1 & 18056 & 9500 & 9500 & 19.35 & 107.71 & 7.8 & 103.08 & 96 & 513 & 55.53 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 1 part & 8415 & 2.407 & 1.000 & 0.00 & 1435.41 & 0.00 & 209 & 209 & - & - & - & - & 5.1 & 5.0 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
