\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
Total length ($\geq$ 1000 bp) & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
Total length ($\geq$ 5000 bp) & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
Total length & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
Reference length & 8662 & 8662 & 8662 & 8662 \\ \hline
GC (\%) & 40.69 & 40.77 & 40.39 & 40.91 \\ \hline
Reference GC (\%) & 40.67 & 40.67 & 40.67 & 40.67 \\ \hline
N50 & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
NG50 & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
N75 & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
NG75 & 8637 & {\bf 8642} & 8533 & 8635 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.6} & {\bf 99.6} & 78.32 & 94.49 \\ \hline
Reference mapped (\%) & 99.6 & 99.6 & 99.6 & 99.6 \\ \hline
Properly paired (\%) & {\bf 99.19} & {\bf 99.19} & 62.76 & 89.6 \\ \hline
Reference properly paired (\%) & 99.19 & 99.19 & 99.19 & 99.19 \\ \hline
Avg. coverage depth & {\bf 506} & {\bf 506} & 394 & 474 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 86.23 & 99.24 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 6 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & {\bf 100.000} & 85.076 & 99.687 \\ \hline
Duplication ratio & {\bf 1.000} & 1.001 & 1.161 & 1.003 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 14520.10 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 138.94} & 219.98 & 517.15 & 2485.48 \\ \hline
\# indels per 100 kbp & 92.62 & 69.47 & {\bf 68.05} & 418.12 \\ \hline
Largest alignment & 8637 & {\bf 8641} & 7352 & 8619 \\ \hline
Total aligned length & 8637 & {\bf 8641} & 7352 & 8619 \\ \hline
NA50 & 8637 & {\bf 8641} & 7352 & 8619 \\ \hline
NGA50 & 8637 & {\bf 8641} & 7352 & 8619 \\ \hline
NA75 & 8637 & {\bf 8641} & 7352 & 8619 \\ \hline
NGA75 & 8637 & {\bf 8641} & 7352 & 8619 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
