\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8614 & 40.52 & 40.53 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8596.0 & 1 & 1 & 1 & 1 & 15289 & 8800 & 8800 & 114.61 & 114.58 & 114.11 & 114.08 & 507 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 92.97 & 34.86 & 8605 & 8605 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8596.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8614 & 40.51 & 40.53 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8596.0 & 1 & 1 & 1 & 1 & 15291 & 8800 & 8800 & 114.6 & 114.58 & 114.11 & 114.08 & 507 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 220.80 & 34.86 & 8605 & 8605 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8596.0 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8504 & 8504 & 8504 & 0 & 0 & 0 & 1 & 8504 & 8504 & 8614 & 40.15 & 40.53 & 8504 & 8504 & 8504 & 8504 & 8504.0 & 8395.4 & 1 & 1 & 1 & 1 & 15436 & 8800 & 8800 & 87.76 & 114.58 & 77.17 & 114.08 & 390 & 507 & 81.69 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 83.657 & 1.000 & 17650.52 & 638.98 & 41.67 & 7199 & 7199 & 7199 & 7199 & - & - & 6094.3 & 6016.4 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8614 & 40.56 & 40.53 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8596.0 & 1 & 1 & 1 & 1 & 15289 & 8800 & 8800 & 114.61 & 114.58 & 114.1 & 114.08 & 507 & 507 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.988 & 1.000 & 0.00 & 743.84 & 34.87 & 8604 & 8604 & 8604 & 8604 & 8604 & 8604 & 8603.0 & 8594.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
