\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
Total length ($\geq$ 1000 bp) & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
Total length ($\geq$ 5000 bp) & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
Total length & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
Reference length & 8604 & 8604 & 8604 & 8604 \\ \hline
GC (\%) & 40.40 & 40.41 & 40.41 & 40.87 \\ \hline
Reference GC (\%) & 40.40 & 40.40 & 40.40 & 40.40 \\ \hline
N50 & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
NG50 & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
N90 & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
NG90 & 8595 & 8574 & 8589 & {\bf 8602} \\ \hline
auN & 8595.0 & 8574.0 & 8589.0 & {\bf 8602.0} \\ \hline
auNG & 8586.0 & 8544.1 & 8574.0 & {\bf 8600.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15343 & 15326 & 15343 & 15208 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 113.44} & 113.02 & {\bf 113.44} & 105.37 \\ \hline
Reference mapped (\%) & 113.44 & 113.44 & 113.44 & 113.44 \\ \hline
Properly paired (\%) & {\bf 112.17} & 111.2 & {\bf 112.17} & 96.84 \\ \hline
Reference properly paired (\%) & 112.17 & 112.17 & 112.17 & 112.17 \\ \hline
Avg. coverage depth & {\bf 505} & 502 & {\bf 505} & 457 \\ \hline
Reference avg. coverage depth & 505 & 505 & 505 & 505 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.93 & 100.0 & 99.2 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.883 & {\bf 100.000} & 98.010 \\ \hline
Duplication ratio & 1.000 & {\bf 0.998} & 0.999 & 1.001 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & {\bf 0.00} & 11.64 & 1767.29 \\ \hline
\# indels per 100 kbp & {\bf 34.90} & 47.18 & 46.57 & 474.44 \\ \hline
Largest alignment & {\bf 8595} & 8478 & 8589 & 8431 \\ \hline
Total aligned length & {\bf 8595} & 8478 & 8589 & 8431 \\ \hline
NA50 & {\bf 8595} & 8478 & 8589 & 8431 \\ \hline
NGA50 & {\bf 8595} & 8478 & 8589 & 8431 \\ \hline
NA90 & {\bf 8595} & 8478 & 8589 & 8431 \\ \hline
NGA90 & {\bf 8595} & 8478 & 8589 & 8431 \\ \hline
auNA & {\bf 8595.0} & 8383.1 & 8589.0 & 8263.4 \\ \hline
auNGA & {\bf 8586.0} & 8353.8 & 8574.0 & 8261.5 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
