\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8623 & 8623 & 8623 & 0 & 0 & 0 & 1 & 8623 & 8623 & 8651 & 40.73 & 40.73 & 8623 & 8623 & 8623 & 8623 & 8623.0 & 8595.1 & 1 & 1 & 1 & 1 & 92386 & 175000 & 175000 & 376.96 & 375.78 & 375.08 & 373.9 & 10074 & 10068 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 11.60 & 92.78 & 8623 & 8623 & 8623 & 8623 & 8623 & 8623 & 8623.0 & 8595.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8618 & 8618 & 8618 & 0 & 0 & 0 & 1 & 8618 & 8618 & 8651 & 40.66 & 40.73 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8585.1 & 1 & 1 & 1 & 1 & 92347 & 175000 & 175000 & 377.07 & 375.78 & 375.13 & 373.9 & 10072 & 10068 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 185.66 & 92.83 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618 & 8618.0 & 8585.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8304 & 8304 & 8304 & 0 & 0 & 0 & 1 & 8304 & 8304 & 8651 & 40.48 & 40.73 & 8304 & 8304 & 8304 & 8304 & 8304.0 & 7970.9 & 1 & 1 & 1 & 1 & 123749 & 175000 & 175000 & 233.26 & 375.78 & 186.5 & 373.9 & 8535 & 10068 & 91.34 & 100.0 & 0 & 0 & 0 & 0 & 0 & 5 & 0 & 0 & 0 + 0 part & 0 & 87.891 & 1.001 & 9658.00 & 975.61 & 65.92 & 7585 & 7585 & 7585 & 7585 & 7585 & - & 6928.3 & 6650.4 & 1 & 1 & 1 & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8644 & 8644 & 8644 & 0 & 0 & 0 & 1 & 8644 & 8644 & 8651 & 41.03 & 40.73 & 8644 & 8644 & 8644 & 8644 & 8644.0 & 8637.0 & 1 & 1 & 1 & 1 & 99920 & 175000 & 175000 & 336.2 & 375.78 & 322.73 & 373.9 & 9623 & 10068 & 99.35 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.849 & 1.003 & 0.00 & 1112.27 & 382.34 & 8631 & 8631 & 8631 & 8631 & 8631 & 8631 & 8618.0 & 8611.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
