\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8710 & 40.47 & 40.47 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8662.1 & 1 & 1 & 1 & 1 & 16807 & 9500 & 9500 & 106.7 & 106.69 & 102.09 & 102.09 & 514 & 514 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 80.59 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8662.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8710 & 41.29 & 40.47 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8566.6 & 1 & 1 & 1 & 1 & 17177 & 9500 & 9500 & 57.71 & 106.69 & 42.06 & 102.09 & 280 & 514 & 60.81 & 100.0 & 0 & 0 & 0 & 5 & 0 & 0 & 0 & 0 & 0 + 1 part & 3231 & 62.250 & 1.000 & 0.00 & 110.97 & 73.98 & 5407 & 5407 & 5407 & 5407 & - & - & 3384.5 & 3356.6 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 0 & 0 & 0 & 0 & 4864 & 4864 & 0 & 0 & 0 & 0 & 1 & 4864 & 4864 & 8710 & 40.16 & 40.47 & 4864 & 4864 & 4864 & - & 4864.0 & 2716.2 & 1 & 1 & 1 & - & 17359 & 9500 & 9500 & 45.2 & 106.69 & 27.47 & 102.09 & 393 & 514 & 84.4 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 47.260 & 1.000 & 15625.00 & 487.21 & 121.80 & 4105 & 4105 & 4105 & - & - & - & 3464.4 & 1934.7 & 1 & - & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8710 & 41.40 & 40.47 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8538.8 & 1 & 1 & 1 & 1 & 18285 & 9500 & 9500 & 16.26 & 106.69 & 6.55 & 102.09 & 82 & 514 & 56.62 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 1 part & 7902 & 8.312 & 1.000 & 0.00 & 3462.60 & 277.01 & 722 & 722 & - & - & - & - & 60.4 & 59.8 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
