\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
Total length ($\geq$ 1000 bp) & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
Total length ($\geq$ 5000 bp) & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
Total length & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
Reference length & 8639 & 8639 & 8639 & 8639 \\ \hline
GC (\%) & 40.61 & 40.61 & 40.30 & 41.07 \\ \hline
Reference GC (\%) & 40.50 & 40.50 & 40.50 & 40.50 \\ \hline
N50 & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
NG50 & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
N90 & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
NG90 & 8612 & {\bf 8621} & 8610 & 8614 \\ \hline
auN & 8612.0 & {\bf 8621.0} & 8610.0 & 8614.0 \\ \hline
auNG & 8585.1 & {\bf 8603.0} & 8581.1 & 8589.1 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15387 & 15382 & 15342 & 15283 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 113.78} & 113.63 & 111.71 & 105.3 \\ \hline
Reference mapped (\%) & 113.76 & 113.76 & 113.76 & 113.76 \\ \hline
Properly paired (\%) & {\bf 113.17} & 112.85 & 108.71 & 96.77 \\ \hline
Reference properly paired (\%) & 113.16 & 113.16 & 113.16 & 113.16 \\ \hline
Avg. coverage depth & 505 & {\bf 506} & 493 & 460 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 98.82 & 99.27 \\ \hline
Reference coverage $\geq$ 1x (\%) & 99.98 & 99.98 & 99.98 & 99.98 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 99.965} & 99.814 & 98.690 & 99.814 \\ \hline
Duplication ratio & {\bf 0.999} & 1.000 & 1.000 & {\bf 0.999} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 2636.47 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 92.93} & 395.12 & 481.90 & 2884.39 \\ \hline
\# indels per 100 kbp & 92.93 & {\bf 58.11} & 58.77 & 558.27 \\ \hline
Largest alignment & {\bf 8609} & 8605 & 8508 & 8598 \\ \hline
Total aligned length & {\bf 8609} & 8605 & 8508 & 8598 \\ \hline
NA50 & {\bf 8609} & 8605 & 8508 & 8598 \\ \hline
NGA50 & {\bf 8609} & 8605 & 8508 & 8598 \\ \hline
NA90 & {\bf 8609} & 8605 & 8508 & 8598 \\ \hline
NGA90 & {\bf 8609} & 8605 & 8508 & 8598 \\ \hline
auNA & {\bf 8606.0} & 8589.0 & 8407.2 & 8582.0 \\ \hline
auNGA & {\bf 8579.1} & 8571.1 & 8379.0 & 8557.2 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
