\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8589 & 8589 & 8589 & 0 & 0 & 0 & 1 & 8589 & 8589 & 8631 & 40.18 & 40.17 & 8589 & 8589 & 8589 & 8589 & 8589.0 & 8547.2 & 1 & 1 & 1 & 1 & 15708 & 8800 & 8800 & 110.45 & 110.36 & 108.86 & 108.77 & 503 & 502 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 23.29 & 139.71 & 8589 & 8589 & 8589 & 8589 & 8589 & 8589 & 8589.0 & 8547.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8589 & 8589 & 8589 & 0 & 0 & 0 & 1 & 8589 & 8589 & 8631 & 40.23 & 40.17 & 8589 & 8589 & 8589 & 8589 & 8589.0 & 8547.2 & 1 & 1 & 1 & 1 & 15702 & 8800 & 8800 & 110.21 & 110.36 & 108.33 & 108.77 & 501 & 502 & 99.86 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.894 & 1.000 & 0.00 & 94.20 & 141.29 & 8493 & 8493 & 8493 & 8493 & 8493 & 8493 & 8398.1 & 8357.2 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8460 & 8460 & 8460 & 0 & 0 & 0 & 1 & 8460 & 8460 & 8631 & 39.58 & 40.17 & 8460 & 8460 & 8460 & 8460 & 8460.0 & 8292.4 & 1 & 1 & 1 & 1 & 15508 & 8800 & 8800 & 74.73 & 110.36 & 55.24 & 108.77 & 331 & 502 & 78.17 & 100.0 & 0 & 0 & 0 & 0 & 0 & 5 & 0 & 0 & 0 + 0 part & 0 & 78.991 & 1.002 & 23723.40 & 1397.68 & 88.27 & 6797 & 6797 & 6797 & 6797 & - & - & 5460.9 & 5352.7 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8617 & 8617 & 8617 & 0 & 0 & 0 & 1 & 8617 & 8617 & 8631 & 40.50 & 40.17 & 8617 & 8617 & 8617 & 8617 & 8617.0 & 8603.0 & 1 & 1 & 1 & 1 & 15498 & 8800 & 8800 & 102.91 & 110.36 & 93.75 & 108.77 & 456 & 502 & 99.86 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.020 & 1.003 & 0.00 & 1811.51 & 816.95 & 8446 & 8446 & 8446 & 8446 & 8446 & 8446 & 8278.4 & 8265.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
