\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
Total length ($\geq$ 1000 bp) & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
Total length ($\geq$ 5000 bp) & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
Total length & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
Reference length & 8630 & 8630 & 8630 & 8630 \\ \hline
GC (\%) & 40.78 & 40.82 & 40.77 & 41.08 \\ \hline
Reference GC (\%) & 40.77 & 40.77 & 40.77 & 40.77 \\ \hline
N50 & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
NG50 & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
N90 & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
NG90 & 8624 & 8624 & 8615 & {\bf 8632} \\ \hline
auN & 8624.0 & 8624.0 & 8615.0 & {\bf 8632.0} \\ \hline
auNG & 8618.0 & 8618.0 & 8600.0 & {\bf 8634.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15218 & 15218 & 15218 & 15151 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 115.15} & 114.97 & {\bf 115.15} & 111.02 \\ \hline
Reference mapped (\%) & 115.15 & 115.15 & 115.15 & 115.15 \\ \hline
Properly paired (\%) & {\bf 114.65} & 114.29 & {\bf 114.65} & 106.2 \\ \hline
Reference properly paired (\%) & 114.65 & 114.65 & 114.65 & 114.65 \\ \hline
Avg. coverage depth & {\bf 507} & 506 & {\bf 507} & 482 \\ \hline
Reference avg. coverage depth & 507 & 507 & 507 & 507 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 99.24 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.849 & 99.965 & 99.849 \\ \hline
Duplication ratio & 1.000 & 1.000 & {\bf 0.999} & 1.001 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 162.58 & {\bf 0.00} & 1334.26 \\ \hline
\# indels per 100 kbp & {\bf 23.19} & 23.23 & 34.82 & 406.08 \\ \hline
Largest alignment & {\bf 8624} & 8611 & 8615 & 8619 \\ \hline
Total aligned length & {\bf 8624} & 8611 & 8615 & 8619 \\ \hline
NA50 & {\bf 8624} & 8611 & 8615 & 8619 \\ \hline
NGA50 & {\bf 8624} & 8611 & 8615 & 8619 \\ \hline
NA90 & {\bf 8624} & 8611 & 8615 & 8619 \\ \hline
NGA90 & {\bf 8624} & 8611 & 8615 & 8619 \\ \hline
auNA & {\bf 8624.0} & 8598.0 & 8615.0 & 8606.0 \\ \hline
auNGA & {\bf 8618.0} & 8592.0 & 8600.0 & 8608.0 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
