\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8703} & 8636 & 1420 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8703} & 8636 & 1420 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8703} & 8636 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8703} & 8636 & 1420 & 8625 \\ \hline
Total length & {\bf 8703} & 8636 & 1420 & 8625 \\ \hline
Reference length & 8736 & 8736 & 8736 & 8736 \\ \hline
GC (\%) & 40.56 & 41.25 & 40.88 & 41.36 \\ \hline
Reference GC (\%) & 40.59 & 40.59 & 40.59 & 40.59 \\ \hline
N50 & {\bf 8703} & 8636 & 1420 & 8625 \\ \hline
NG50 & {\bf 8703} & 8636 & - & 8625 \\ \hline
N75 & {\bf 8703} & 8636 & 1420 & 8625 \\ \hline
NG75 & {\bf 8703} & 8636 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 380000 & 380000 & 380000 & 380000 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 94.36} & 51.04 & 10.39 & 16.06 \\ \hline
Reference mapped (\%) & 94.31 & 94.31 & 94.31 & 94.31 \\ \hline
Properly paired (\%) & {\bf 90.41} & 37.38 & 1.82 & 5.51 \\ \hline
Reference properly paired (\%) & 90.36 & 90.36 & 90.36 & 90.36 \\ \hline
Avg. coverage depth & {\bf 10247} & 5487 & 6653 & 1669 \\ \hline
Reference avg. coverage depth & 10235 & 10235 & 10235 & 10235 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 60.46 & 81.83 & 54.38 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 5 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 3295 & {\bf 0} & 7686 \\ \hline
Genome fraction (\%) & {\bf 99.943} & 61.337 & 13.313 & 10.786 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 1.225 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 18169.01 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 22.99} & 486.89 & 2502.16 & 2129.93 \\ \hline
\# indels per 100 kbp & 114.93 & {\bf 93.63} & 172.56 & 745.47 \\ \hline
Largest alignment & {\bf 8698} & 5341 & 1159 & 730 \\ \hline
Total aligned length & {\bf 8698} & 5341 & 1159 & 939 \\ \hline
NA50 & {\bf 8698} & 5341 & 1159 & - \\ \hline
NGA50 & {\bf 8698} & 5341 & - & - \\ \hline
NA75 & {\bf 8698} & - & 1159 & - \\ \hline
NGA75 & {\bf 8698} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & {\bf 1} & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
