\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8683 & 8683 & 8683 & 0 & 0 & 0 & 1 & 8683 & 8683 & 8707 & 40.25 & 40.25 & 8683 & 8683 & 8683 & 8683 & 8683.0 & 8659.1 & 1 & 1 & 1 & 1 & 108029 & 190000 & 190000 & 331.15 & 330.59 & 316.56 & 316.03 & 10267 & 10259 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 69.10 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683.0 & 8659.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8707 & 40.98 & 40.25 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8569.5 & 1 & 1 & 1 & 1 & 225942 & 190000 & 190000 & 83.93 & 330.59 & 62.93 & 316.03 & 5386 & 10259 & 57.79 & 100.0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 & 0 & 0 + 1 part & 3501 & 59.162 & 1.000 & 0.00 & 19.47 & 77.87 & 5137 & 5137 & 5137 & 5137 & - & - & 3055.0 & 3030.8 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 5293 & 5293 & 5293 & 0 & 0 & 0 & 1 & 5293 & 5293 & 8707 & 39.59 & 40.25 & 5293 & 5293 & 5293 & - & 5293.0 & 3217.6 & 1 & 1 & 1 & - & 255525 & 190000 & 190000 & 57.76 & 330.59 & 33.27 & 316.03 & 6798 & 10259 & 74.44 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 47.011 & 1.000 & 22841.49 & 269.48 & 24.50 & 4082 & 4082 & 4082 & - & - & - & 3148.1 & 1913.7 & 1 & - & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8707 & 41.29 & 40.25 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8541.8 & 1 & 1 & 1 & 1 & 314368 & 190000 & 190000 & 24.67 & 330.59 & 10.56 & 316.03 & 2134 & 10259 & 57.1 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7364 & 14.511 & 1.000 & 0.00 & 2539.68 & 317.46 & 851 & 1260 & - & - & - & - & 93.7 & 92.8 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
