\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8577 & 8577 & 8577 & 0 & 0 & 0 & 1 & 8577 & 8577 & 8619 & 40.34 & 40.34 & 8577 & 8577 & 8577 & 8577 & 8577.0 & 8535.2 & 1 & 1 & 1 & 1 & 15755 & 8800 & 8800 & 110.19 & 110.17 & 108.66 & 108.64 & 503 & 503 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 0.00 & 23.32 & 104.93 & 8577 & 8577 & 8577 & 8577 & 8577 & 8577 & 8577.0 & 8535.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8595 & 8595 & 8595 & 0 & 0 & 0 & 1 & 8595 & 8595 & 8619 & 40.23 & 40.34 & 8595 & 8595 & 8595 & 8595 & 8595.0 & 8571.1 & 1 & 1 & 1 & 1 & 15752 & 8800 & 8800 & 110.11 & 110.17 & 108.49 & 108.64 & 503 & 503 & 99.86 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.453 & 1.001 & 0.00 & 374.36 & 81.89 & 8548 & 8548 & 8548 & 8548 & 8548 & 8548 & 8501.3 & 8477.6 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7595 & 7595 & 7595 & 0 & 0 & 0 & 1 & 7595 & 7595 & 8619 & 39.98 & 40.34 & 7595 & 7595 & 7595 & - & 7595.0 & 6692.7 & 1 & 1 & 1 & - & 15364 & 8800 & 8800 & 78.16 & 110.17 & 55.95 & 108.64 & 384 & 503 & 87.14 & 100.0 & 0 & 0 & 0 & 0 & 0 & 5 & 0 & 0 & 0 + 0 part & 0 & 78.170 & 0.999 & 13903.88 & 2012.52 & 44.72 & 6708 & 6708 & 6708 & 6708 & - & - & 5924.6 & 5220.7 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8598 & 8598 & 8598 & 0 & 0 & 0 & 1 & 8598 & 8598 & 8619 & 40.74 & 40.34 & 8598 & 8598 & 8598 & 8598 & 8598.0 & 8577.1 & 1 & 1 & 1 & 1 & 15489 & 8800 & 8800 & 100.36 & 110.17 & 89.07 & 108.64 & 444 & 503 & 98.14 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 97.904 & 1.001 & 0.00 & 2934.54 & 605.92 & 8417 & 8417 & 8417 & 8417 & 8417 & 8417 & 8239.8 & 8219.7 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
