\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8611 & 8611 & 8611 & 0 & 0 & 0 & 1 & 8611 & 8611 & 8641 & 40.55 & 40.59 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8581.1 & 1 & 1 & 1 & 1 & 15426 & 8800 & 8800 & 113.49 & 113.4 & 112.89 & 112.8 & 507 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 499.36 & 116.13 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8581.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8611 & 8611 & 8611 & 0 & 0 & 0 & 1 & 8611 & 8611 & 8641 & 40.62 & 40.59 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8581.1 & 1 & 1 & 1 & 1 & 15429 & 8800 & 8800 & 113.33 & 113.4 & 112.59 & 112.8 & 506 & 506 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.106 & 1.000 & 0.00 & 703.07 & 105.46 & 8534 & 8534 & 8534 & 8534 & 8534 & 8534 & 8457.7 & 8428.3 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8614 & 8614 & 8614 & 0 & 0 & 0 & 1 & 8614 & 8614 & 8641 & 40.37 & 40.59 & 8614 & 8614 & 8614 & 8614 & 8614.0 & 8587.1 & 1 & 1 & 1 & 1 & 15053 & 8800 & 8800 & 79.14 & 113.4 & 54.04 & 112.8 & 336 & 506 & 76.35 & 100.0 & 0 & 0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 + 0 part & 0 & 76.361 & 1.000 & 24378.92 & 897.07 & 45.61 & 6577 & 6577 & 6577 & 6577 & - & - & 5021.7 & 5006.0 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8642 & 8642 & 8642 & 0 & 0 & 0 & 1 & 8642 & 8642 & 8641 & 40.96 & 40.59 & 8642 & 8642 & 8642 & 8642 & 8642.0 & 8643.0 & 1 & 1 & 1 & 1 & 15322 & 8800 & 8800 & 105.85 & 113.4 & 98.29 & 112.8 & 463 & 506 & 99.44 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.015 & 1.003 & 0.00 & 2632.51 & 755.52 & 8471 & 8471 & 8471 & 8471 & 8471 & 8471 & 8303.4 & 8304.3 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
