\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
Total length ($\geq$ 1000 bp) & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
Total length ($\geq$ 5000 bp) & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
Total length & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
Reference length & 8731 & 8731 & 8731 & 8731 \\ \hline
GC (\%) & 40.76 & 40.75 & 40.73 & 40.74 \\ \hline
Reference GC (\%) & 40.72 & 40.72 & 40.72 & 40.72 \\ \hline
N50 & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
NG50 & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
N90 & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
NG90 & 8714 & 8720 & 8726 & {\bf 8732} \\ \hline
auN & 8714.0 & 8720.0 & 8726.0 & {\bf 8732.0} \\ \hline
auNG & 8697.0 & 8709.0 & 8721.0 & {\bf 8733.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 117041 & 117098 & 117253 & 117263 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 306.53} & 306.4 & 305.9 & 305.93 \\ \hline
Reference mapped (\%) & 305.07 & 305.07 & 305.07 & 305.07 \\ \hline
Properly paired (\%) & {\bf 293.47} & 293.35 & 292.85 & 292.86 \\ \hline
Reference properly paired (\%) & 291.79 & 291.79 & 291.79 & 291.79 \\ \hline
Avg. coverage depth & {\bf 10255} & {\bf 10255} & 10249 & 10251 \\ \hline
Reference avg. coverage depth & 10256 & 10256 & 10256 & 10256 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 100.000 & 100.000 & 100.000 & 100.000 \\ \hline
Duplication ratio & {\bf 0.999} & 1.000 & 1.001 & 1.001 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 22.92 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & {\bf 0.00} & 436.28 & {\bf 0.00} \\ \hline
\# indels per 100 kbp & 103.47 & 91.91 & 80.37 & {\bf 57.37} \\ \hline
Largest alignment & 8698 & 8704 & 8710 & {\bf 8716} \\ \hline
Total aligned length & 8698 & 8704 & 8710 & {\bf 8716} \\ \hline
NA50 & 8698 & 8704 & 8710 & {\bf 8716} \\ \hline
NGA50 & 8698 & 8704 & 8710 & {\bf 8716} \\ \hline
NA90 & 8698 & 8704 & 8710 & {\bf 8716} \\ \hline
NGA90 & 8698 & 8704 & 8710 & {\bf 8716} \\ \hline
auNA & 8682.0 & 8688.0 & 8694.0 & {\bf 8700.0} \\ \hline
auNGA & 8665.1 & 8677.1 & 8689.1 & {\bf 8701.0} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
