\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
Total length ($\geq$ 1000 bp) & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
Total length ($\geq$ 5000 bp) & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
Total length & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
Reference length & 8633 & 8633 & 8633 & 8633 \\ \hline
GC (\%) & 40.07 & 40.04 & 39.75 & 39.96 \\ \hline
Reference GC (\%) & 40.08 & 40.08 & 40.08 & 40.08 \\ \hline
N50 & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
NG50 & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
N75 & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
NG75 & 8606 & 8606 & 8323 & {\bf 8612} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.48} & {\bf 99.48} & 87.31 & {\bf 99.48} \\ \hline
Reference mapped (\%) & 99.48 & 99.48 & 99.48 & 99.48 \\ \hline
Properly paired (\%) & {\bf 98.95} & {\bf 98.95} & 75.18 & {\bf 98.95} \\ \hline
Reference properly paired (\%) & 98.95 & 98.95 & 98.95 & 98.95 \\ \hline
Avg. coverage depth & {\bf 507} & {\bf 507} & 456 & {\bf 507} \\ \hline
Reference avg. coverage depth & 507 & 507 & 507 & 507 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 94.14 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & 525 & {\bf 0} \\ \hline
Genome fraction (\%) & {\bf 100.000} & {\bf 100.000} & 84.906 & {\bf 100.000} \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 1.067 & 1.001 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 6367.90 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 11.62} & 34.86 & 752.70 & 139.44 \\ \hline
\# indels per 100 kbp & 92.96 & 92.96 & 82.11 & {\bf 81.34} \\ \hline
Largest alignment & 8606 & 8606 & 7310 & {\bf 8612} \\ \hline
Total aligned length & 8606 & 8606 & 7310 & {\bf 8612} \\ \hline
NA50 & 8606 & 8606 & 7310 & {\bf 8612} \\ \hline
NGA50 & 8606 & 8606 & 7310 & {\bf 8612} \\ \hline
NA75 & 8606 & 8606 & 7310 & {\bf 8612} \\ \hline
NGA75 & 8606 & 8606 & 7310 & {\bf 8612} \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
