\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
Total length ($\geq$ 1000 bp) & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
Total length ($\geq$ 5000 bp) & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
Total length & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
Reference length & 8647 & 8647 & 8647 & 8647 \\ \hline
GC (\%) & 40.55 & 40.60 & 40.38 & 41.13 \\ \hline
Reference GC (\%) & 40.59 & 40.59 & 40.59 & 40.59 \\ \hline
N50 & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
NG50 & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
N75 & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
NG75 & 8632 & 8638 & 8629 & {\bf 8644} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.5} & 99.41 & 97.28 & 92.89 \\ \hline
Reference mapped (\%) & 99.5 & 99.5 & 99.5 & 99.5 \\ \hline
Properly paired (\%) & {\bf 99.0} & 98.82 & 94.57 & 87.19 \\ \hline
Reference properly paired (\%) & 99.0 & 99.0 & 99.0 & 99.0 \\ \hline
Avg. coverage depth & {\bf 506} & 505 & 492 & 467 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.91 & 99.05 & 99.24 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 2 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.722 & 99.930 & 91.725 \\ \hline
Duplication ratio & {\bf 1.000} & 1.004 & 1.001 & 1.092 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 2155.52 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 289.75} & 534.64 & 788.68 & 2160.73 \\ \hline
\# indels per 100 kbp & 57.95 & 46.49 & {\bf 46.39} & 151.63 \\ \hline
Largest alignment & {\bf 8632} & 8611 & 8629 & 7917 \\ \hline
Total aligned length & {\bf 8632} & 8611 & 8629 & 7917 \\ \hline
NA50 & {\bf 8632} & 8611 & 8629 & 7917 \\ \hline
NGA50 & {\bf 8632} & 8611 & 8629 & 7917 \\ \hline
NA75 & {\bf 8632} & 8611 & 8629 & 7917 \\ \hline
NGA75 & {\bf 8632} & 8611 & 8629 & 7917 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
