\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8583 & 8583 & 8583 & 0 & 0 & 0 & 1 & 8583 & 8583 & 8601 & 40.11 & 40.10 & 8583 & 8583 & 8583 & 8583 & 8583.0 & 8565.0 & 1 & 1 & 1 & 1 & 86724 & 175000 & 175000 & 398.28 & 398.18 & 392.98 & 392.88 & 10014 & 10021 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.999 & 0.00 & 11.65 & 34.95 & 8583 & 8583 & 8583 & 8583 & 8583 & 8583 & 8583.0 & 8565.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8574 & 8574 & 8574 & 0 & 0 & 0 & 1 & 8574 & 8574 & 8601 & 40.07 & 40.10 & 8574 & 8574 & 8574 & 8574 & 8574.0 & 8547.1 & 1 & 1 & 1 & 1 & 87567 & 175000 & 175000 & 392.96 & 398.18 & 386.23 & 392.88 & 9973 & 10021 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.942 & 0.998 & 0.00 & 70.02 & 35.01 & 8569 & 8569 & 8569 & 8569 & 8569 & 8569 & 8564.0 & 8537.1 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8479 & 8479 & 8479 & 0 & 0 & 0 & 1 & 8479 & 8479 & 8601 & 40.07 & 40.10 & 8479 & 8479 & 8479 & 8479 & 8479.0 & 8358.7 & 1 & 1 & 1 & 1 & 87756 & 175000 & 175000 & 390.81 & 398.18 & 382.78 & 392.88 & 10071 & 10021 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.650 & 1.000 & 1191.18 & 224.08 & 23.59 & 8479 & 8479 & 8479 & 8479 & 8479 & 8479 & 8479.0 & 8358.7 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8604 & 8604 & 8604 & 0 & 0 & 0 & 1 & 8604 & 8604 & 8601 & 40.48 & 40.10 & 8604 & 8604 & 8604 & 8604 & 8604.0 & 8607.0 & 1 & 1 & 1 & 1 & 98080 & 175000 & 175000 & 334.7 & 398.18 & 316.82 & 392.88 & 9413 & 10021 & 99.88 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.453 & 1.001 & 0.00 & 956.20 & 413.17 & 8471 & 8471 & 8471 & 8471 & 8471 & 8471 & 8340.1 & 8343.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
