\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
Total length ($\geq$ 1000 bp) & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
Total length ($\geq$ 5000 bp) & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
Total length & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
Reference length & 8646 & 8646 & 8646 & 8646 \\ \hline
GC (\%) & 40.29 & 40.31 & 40.41 & 40.75 \\ \hline
Reference GC (\%) & 40.40 & 40.40 & 40.40 & 40.40 \\ \hline
N50 & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
NG50 & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
N90 & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
NG90 & 8526 & 8571 & 8595 & {\bf 8628} \\ \hline
auN & 8526.0 & 8571.0 & 8595.0 & {\bf 8628.0} \\ \hline
auNG & 8407.7 & 8496.7 & 8544.3 & {\bf 8610.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 97293 & 94106 & 93312 & 102584 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & 347.42 & 365.18 & {\bf 370.18} & 320.95 \\ \hline
Reference mapped (\%) & 368.0 & 368.0 & 368.0 & 368.0 \\ \hline
Properly paired (\%) & 335.11 & 358.44 & {\bf 365.26} & 304.68 \\ \hline
Reference properly paired (\%) & 362.99 & 362.99 & 362.99 & 362.99 \\ \hline
Avg. coverage depth & 9862 & 9962 & {\bf 10023} & 9450 \\ \hline
Reference avg. coverage depth & 10001 & 10001 & 10001 & 10001 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 99.453 & {\bf 100.000} & 98.289 \\ \hline
Duplication ratio & {\bf 0.992} & 0.997 & 1.000 & 1.004 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 11.73} & 70.39 & 302.50 & 1002.24 \\ \hline
\# indels per 100 kbp & 164.20 & 187.71 & {\bf 162.89} & 436.27 \\ \hline
Largest alignment & 8526 & 8524 & {\bf 8595} & 8481 \\ \hline
Total aligned length & 8526 & 8524 & {\bf 8595} & 8481 \\ \hline
NA50 & 8526 & 8524 & {\bf 8595} & 8481 \\ \hline
NGA50 & 8526 & 8524 & {\bf 8595} & 8481 \\ \hline
NA90 & 8526 & 8524 & {\bf 8595} & 8481 \\ \hline
NGA90 & 8526 & 8524 & {\bf 8595} & 8481 \\ \hline
auNA & 8526.0 & 8477.3 & {\bf 8595.0} & 8336.5 \\ \hline
auNGA & 8407.7 & 8403.7 & {\bf 8544.3} & 8319.1 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
