\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
Total length ($\geq$ 1000 bp) & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
Total length ($\geq$ 5000 bp) & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
Total length & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
Reference length & 8662 & 8662 & 8662 & 8662 \\ \hline
GC (\%) & 40.38 & 40.37 & - & 40.83 \\ \hline
Reference GC (\%) & 40.39 & 40.39 & 40.39 & 40.39 \\ \hline
N50 & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
NG50 & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
N90 & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
NG90 & 8615 & 8605 & 8624 & {\bf 8638} \\ \hline
auN & 8615.0 & 8605.0 & 8624.0 & {\bf 8638.0} \\ \hline
auNG & 8568.3 & 8548.4 & 8586.2 & {\bf 8614.1} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 100738 & 101045 & 350000 & 113733 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & {\bf 345.78} & 343.99 & 0.0 & 284.95 \\ \hline
Reference mapped (\%) & 344.21 & 344.21 & 344.21 & 344.21 \\ \hline
Properly paired (\%) & {\bf 344.12} & 341.6 & 0.0 & 263.88 \\ \hline
Reference properly paired (\%) & 342.55 & 342.55 & 342.55 & 342.55 \\ \hline
Avg. coverage depth & {\bf 10078} & 10049 & 0 & 9242 \\ \hline
Reference avg. coverage depth & 10059 & 10059 & 10059 & 10059 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 0.0 & 99.53 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# misassembled contigs & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
Misassembled contigs length & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# structural variations & 0 & 0 & - & 0 \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & - & {\bf 0} \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 1 + 0 part & {\bf 0 + 0 part} \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & 8624 & {\bf 0} \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.931 & - & 98.827 \\ \hline
Duplication ratio & 1.001 & {\bf 0.999} & - & 1.003 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 100000.00 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 516.86 & - & 1686.97 \\ \hline
\# indels per 100 kbp & 162.51 & {\bf 152.71} & - & 574.04 \\ \hline
Largest alignment & {\bf 8615} & 8513 & - & 8536 \\ \hline
Total aligned length & {\bf 8615} & 8513 & - & 8536 \\ \hline
NA50 & {\bf 8615} & 8513 & - & 8536 \\ \hline
NGA50 & {\bf 8615} & 8513 & - & 8536 \\ \hline
NA90 & {\bf 8615} & 8513 & - & 8536 \\ \hline
NGA90 & {\bf 8615} & 8513 & - & 8536 \\ \hline
auNA & {\bf 8615.0} & 8422.0 & - & 8435.2 \\ \hline
auNGA & {\bf 8568.3} & 8366.6 & - & 8411.8 \\ \hline
LA50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LA90 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
LGA90 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
