\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8615 & 8615 & 8615 & 0 & 0 & 0 & 1 & 8615 & 8615 & 8662 & 40.38 & 40.39 & 8615 & 8615 & 8615 & 8615 & 8615.0 & 8568.3 & 1 & 1 & 1 & 1 & 100738 & 175000 & 175000 & 345.78 & 344.21 & 344.12 & 342.55 & 10078 & 10059 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.001 & 0.00 & 0.00 & 162.51 & 8615 & 8615 & 8615 & 8615 & 8615 & 8615 & 8615.0 & 8568.3 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8605 & 8605 & 8605 & 0 & 0 & 0 & 1 & 8605 & 8605 & 8662 & 40.37 & 40.39 & 8605 & 8605 & 8605 & 8605 & 8605.0 & 8548.4 & 1 & 1 & 1 & 1 & 101045 & 175000 & 175000 & 343.99 & 344.21 & 341.6 & 342.55 & 10049 & 10059 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.931 & 0.999 & 0.00 & 516.86 & 152.71 & 8513 & 8513 & 8513 & 8513 & 8513 & 8513 & 8422.0 & 8366.6 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8662 & - & 40.39 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8586.2 & 1 & 1 & 1 & 1 & 350000 & 175000 & 175000 & 0.0 & 344.21 & 0.0 & 342.55 & 0 & 10059 & 0.0 & 100.0 & - & - & - & - & - & - & - & - & 1 + 0 part & 8624 & - & - & 100000.00 & - & - & - & - & - & - & - & - & - & - & - & - & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8662 & 40.83 & 40.39 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8614.1 & 1 & 1 & 1 & 1 & 113733 & 175000 & 175000 & 284.95 & 344.21 & 263.88 & 342.55 & 9242 & 10059 & 99.53 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.827 & 1.003 & 0.00 & 1686.97 & 574.04 & 8536 & 8536 & 8536 & 8536 & 8536 & 8536 & 8435.2 & 8411.8 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
