\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8601 & 8600 & 5696 & {\bf 8605} \\ \hline
Total length ($\geq$ 1000 bp) & 8601 & 8600 & 5696 & {\bf 8605} \\ \hline
Total length ($\geq$ 5000 bp) & 8601 & 8600 & 5696 & {\bf 8605} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8601 & 8600 & 5696 & {\bf 8605} \\ \hline
Total length & 8601 & 8600 & 5696 & {\bf 8605} \\ \hline
Reference length & 8634 & 8634 & 8634 & 8634 \\ \hline
GC (\%) & 40.30 & 40.35 & 41.60 & 40.83 \\ \hline
Reference GC (\%) & 40.31 & 40.31 & 40.31 & 40.31 \\ \hline
N50 & 8601 & 8600 & 5696 & {\bf 8605} \\ \hline
NG50 & 8601 & 8600 & 5696 & {\bf 8605} \\ \hline
N75 & 8601 & 8600 & 5696 & {\bf 8605} \\ \hline
NG75 & 8601 & 8600 & - & {\bf 8605} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 98.8} & 98.64 & 52.5 & 90.18 \\ \hline
Reference mapped (\%) & 98.8 & 98.8 & 98.8 & 98.8 \\ \hline
Properly paired (\%) & {\bf 97.6} & 97.28 & 40.12 & 81.74 \\ \hline
Reference properly paired (\%) & 97.6 & 97.6 & 97.6 & 97.6 \\ \hline
Avg. coverage depth & {\bf 504} & 503 & 399 & 453 \\ \hline
Reference avg. coverage depth & 503 & 503 & 503 & 503 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 85.5 & 99.23 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.814 & 54.820 & 97.942 \\ \hline
Duplication ratio & {\bf 1.000} & 1.012 & 1.208 & 1.022 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 15291.43 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 174.44} & 411.91 & 381.84 & 2683.45 \\ \hline
\# indels per 100 kbp & {\bf 69.78} & 70.61 & 106.07 & 474.95 \\ \hline
Largest alignment & {\bf 8601} & 8499 & 4716 & 8430 \\ \hline
Total aligned length & {\bf 8601} & 8499 & 4716 & 8430 \\ \hline
NA50 & {\bf 8601} & 8499 & 4716 & 8430 \\ \hline
NGA50 & {\bf 8601} & 8499 & 4716 & 8430 \\ \hline
NA75 & {\bf 8601} & 8499 & 4716 & 8430 \\ \hline
NGA75 & {\bf 8601} & 8499 & - & 8430 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
