\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8684 & 8684 & 8684 & 0 & 0 & 0 & 1 & 8684 & 8684 & 8713 & 40.48 & 40.40 & 8684 & 8684 & 8684 & 8684 & 8684.0 & 8655.1 & 1 & 1 & 1 & 1 & 16855 & 9500 & 9500 & 106.35 & 106.3 & 101.9 & 101.86 & 513 & 513 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 126.67 & 149.70 & 8684 & 8684 & 8684 & 8684 & 8684 & 8684 & 8684.0 & 8655.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8635 & 8635 & 8635 & 0 & 0 & 0 & 1 & 8635 & 8635 & 8713 & 41.15 & 40.40 & 8635 & 8635 & 8635 & 8635 & 8635.0 & 8557.7 & 1 & 1 & 1 & 1 & 17234 & 9500 & 9500 & 55.69 & 106.3 & 41.74 & 101.86 & 272 & 513 & 57.66 & 100.0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 & 0 & 0 + 1 part & 3503 & 59.084 & 1.000 & 0.00 & 19.49 & 77.94 & 5132 & 5132 & 5132 & 5132 & - & - & 3050.1 & 3022.8 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 5280 & 5280 & 5280 & 0 & 0 & 0 & 1 & 5280 & 5280 & 8713 & 40.18 & 40.40 & 5280 & 5280 & 5280 & - & 5280.0 & 3199.6 & 1 & 1 & 1 & - & 17274 & 9500 & 9500 & 47.2 & 106.3 & 30.35 & 101.86 & 377 & 513 & 77.82 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 48.584 & 0.999 & 21382.58 & 213.57 & 118.65 & 4214 & 4214 & 4214 & - & - & - & 3363.2 & 2038.1 & 1 & - & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8713 & 41.19 & 40.40 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8535.9 & 1 & 1 & 1 & 1 & 18175 & 9500 & 9500 & 16.98 & 106.3 & 6.15 & 101.86 & 84 & 513 & 53.51 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 8005 & 7.126 & 1.000 & 0.00 & 323.10 & 0.00 & 413 & 619 & - & - & - & - & 24.7 & 24.4 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
