\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8641 & 8641 & 8641 & 0 & 0 & 0 & 1 & 8641 & 8641 & 8738 & 40.61 & 40.63 & 8641 & 8641 & 8641 & 8641 & 8641.0 & 8545.1 & 1 & 1 & 1 & 1 & 105330 & 190000 & 190000 & 333.73 & 354.66 & 312.62 & 339.88 & 10092 & 10244 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.989 & 0.991 & 0.00 & 46.29 & 81.01 & 8641 & 8641 & 8641 & 8641 & 8641 & 8641 & 8641.0 & 8545.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8723 & 8723 & 8723 & 0 & 0 & 0 & 1 & 8723 & 8723 & 8738 & 40.67 & 40.63 & 8723 & 8723 & 8723 & 8723 & 8723.0 & 8708.0 & 1 & 1 & 1 & 1 & 101042 & 190000 & 190000 & 355.08 & 354.66 & 340.27 & 339.88 & 10248 & 10244 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 103.18 & 45.86 & 8723 & 8723 & 8723 & 8723 & 8723 & 8723 & 8723.0 & 8708.0 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8722 & 8722 & 8722 & 0 & 0 & 0 & 1 & 8722 & 8722 & 8738 & 40.66 & 40.63 & 8722 & 8722 & 8722 & 8722 & 8722.0 & 8706.0 & 1 & 1 & 1 & 1 & 105411 & 190000 & 190000 & 332.87 & 354.66 & 311.18 & 339.88 & 9997 & 10244 & 99.09 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.989 & 1.000 & 905.76 & 435.68 & 45.86 & 8722 & 8722 & 8722 & 8722 & 8722 & 8722 & 8722.0 & 8706.0 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8722 & 8722 & 8722 & 0 & 0 & 0 & 1 & 8722 & 8722 & 8738 & 40.61 & 40.63 & 8722 & 8722 & 8722 & 8722 & 8722.0 & 8706.0 & 1 & 1 & 1 & 1 & 101053 & 190000 & 190000 & 355.03 & 354.66 & 340.21 & 339.88 & 10249 & 10244 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.989 & 1.000 & 0.00 & 34.40 & 45.86 & 8722 & 8722 & 8722 & 8722 & 8722 & 8722 & 8722.0 & 8706.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
