\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8713 & 40.71 & 40.71 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8659.1 & 1 & 1 & 1 & 1 & 110276 & 190000 & 190000 & 324.75 & 323.7 & 310.23 & 309.13 & 10280 & 10265 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 80.59 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8659.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8713 & 41.34 & 40.71 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8563.6 & 1 & 1 & 1 & 1 & 215372 & 190000 & 190000 & 95.81 & 323.7 & 70.1 & 309.13 & 5788 & 10265 & 65.98 & 100.0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 & 0 & 0 + 1 part & 2820 & 66.981 & 1.000 & 0.00 & 0.00 & 17.19 & 5818 & 5818 & 5818 & 5818 & - & - & 3918.6 & 3884.9 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7166 & 7166 & 7166 & 0 & 0 & 0 & 1 & 7166 & 7166 & 8713 & 41.03 & 40.71 & 7166 & 7166 & 7166 & - & 7166.0 & 5893.7 & 1 & 1 & 1 & - & 216158 & 190000 & 190000 & 96.91 & 323.7 & 75.89 & 309.13 & 7169 & 10265 & 75.27 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 63.804 & 0.999 & 22732.35 & 72.22 & 54.16 & 5539 & 5539 & 5539 & 5539 & - & - & 4281.4 & 3521.2 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8713 & 41.36 & 40.71 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8535.9 & 1 & 1 & 1 & 1 & 307723 & 190000 & 190000 & 27.81 & 323.7 & 12.18 & 309.13 & 2360 & 10265 & 60.19 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7312 & 15.105 & 1.000 & 0.00 & 1143.29 & 304.88 & 858 & 1312 & - & - & - & - & 97.4 & 96.4 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
