\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
Total length ($\geq$ 1000 bp) & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
Total length ($\geq$ 5000 bp) & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
Total length & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
Reference length & 8631 & 8631 & 8631 & 8631 \\ \hline
GC (\%) & 40.37 & 40.45 & 40.41 & 40.76 \\ \hline
Reference GC (\%) & 40.41 & 40.41 & 40.41 & 40.41 \\ \hline
N50 & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
NG50 & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
N90 & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
NG90 & 8579 & 8574 & 8595 & {\bf 8632} \\ \hline
auN & 8579.0 & 8574.0 & 8595.0 & {\bf 8632.0} \\ \hline
auNG & 8527.3 & 8517.4 & 8559.2 & {\bf 8633.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 15292 & 15256 & 15277 & 15196 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 112.92 & 113.29 & {\bf 113.78} & 106.36 \\ \hline
Reference mapped (\%) & 113.76 & 113.76 & 113.76 & 113.76 \\ \hline
Properly paired (\%) & 111.38 & 111.22 & {\bf 112.35} & 98.5 \\ \hline
Reference properly paired (\%) & 112.34 & 112.34 & 112.34 & 112.34 \\ \hline
Avg. coverage depth & 500 & 501 & {\bf 504} & 461 \\ \hline
Reference avg. coverage depth & 503 & 503 & 503 & 503 \\ \hline
Coverage $\geq$ 1x (\%) & 99.9 & 99.93 & 100.0 & 99.28 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & 99.930 & 98.883 & {\bf 100.000} & 98.057 \\ \hline
Duplication ratio & {\bf 0.998} & {\bf 0.998} & 1.000 & 1.004 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 0.00} & 23.59 & 11.63 & 1795.63 \\ \hline
\# indels per 100 kbp & 151.69 & 117.95 & {\bf 104.71} & 637.92 \\ \hline
Largest alignment & 8570 & 8478 & {\bf 8595} & 8465 \\ \hline
Total aligned length & 8570 & 8478 & {\bf 8595} & 8465 \\ \hline
NA50 & 8570 & 8478 & {\bf 8595} & 8465 \\ \hline
NGA50 & 8570 & 8478 & {\bf 8595} & 8465 \\ \hline
NA90 & 8570 & 8478 & {\bf 8595} & 8465 \\ \hline
NGA90 & 8570 & 8478 & {\bf 8595} & 8465 \\ \hline
auNA & 8561.0 & 8383.1 & {\bf 8595.0} & 8301.2 \\ \hline
auNGA & 8509.4 & 8327.7 & {\bf 8559.2} & 8302.2 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
