\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
Total length & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
Reference length & 8619 & 8619 & 8619 & 8619 \\ \hline
GC (\%) & - & 40.18 & 40.00 & 40.67 \\ \hline
Reference GC (\%) & 40.16 & 40.16 & 40.16 & 40.16 \\ \hline
N50 & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
NG50 & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
N90 & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
NG90 & {\bf 8624} & 8571 & 8594 & 8593 \\ \hline
auN & {\bf 8624.0} & 8571.0 & 8594.0 & 8593.0 \\ \hline
auNG & {\bf 8629.0} & 8523.3 & 8569.1 & 8567.1 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 15496 & 15374 & 15236 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 0.0 & {\bf 111.57} & 107.68 & 101.87 \\ \hline
Reference mapped (\%) & 111.85 & 111.85 & 111.85 & 111.85 \\ \hline
Properly paired (\%) & 0.0 & {\bf 109.56} & 100.87 & 90.29 \\ \hline
Reference properly paired (\%) & 110.26 & 110.26 & 110.26 & 110.26 \\ \hline
Avg. coverage depth & 0 & {\bf 501} & 478 & 444 \\ \hline
Reference avg. coverage depth & 503 & 503 & 503 & 503 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 100.0 & 98.96 & 98.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 0 & 0 \\ \hline
\# structural variations & - & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 1 + 0 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & 8624 & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & - & 100.000 & 100.000 & 98.009 \\ \hline
Duplication ratio & - & 0.998 & 1.000 & 1.000 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & 1908.31 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 245.01 & 1175.24 & 2362.86 \\ \hline
\# indels per 100 kbp & - & 105.01 & 93.09 & 736.17 \\ \hline
Largest alignment & - & 8571 & 8594 & 8422 \\ \hline
Total aligned length & - & 8571 & 8594 & 8422 \\ \hline
NA50 & - & 8571 & 8594 & 8422 \\ \hline
NGA50 & - & 8571 & 8594 & 8422 \\ \hline
NA90 & - & 8571 & 8594 & 8422 \\ \hline
NGA90 & - & 8571 & 8594 & 8422 \\ \hline
auNA & - & 8571.0 & 8594.0 & 8254.4 \\ \hline
auNGA & - & 8523.3 & 8569.1 & 8229.5 \\ \hline
LA50 & - & 1 & 1 & 1 \\ \hline
LGA50 & - & 1 & 1 & 1 \\ \hline
LA90 & - & 1 & 1 & 1 \\ \hline
LGA90 & - & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
