\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8716 & 40.72 & 40.72 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8656.1 & 1 & 1 & 1 & 1 & 16530 & 9500 & 9500 & 108.37 & 108.32 & 103.42 & 103.36 & 514 & 513 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 69.08 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8656.1 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8716 & 41.39 & 40.72 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8560.7 & 1 & 1 & 1 & 1 & 16981 & 9500 & 9500 & 59.7 & 108.32 & 43.83 & 103.36 & 286 & 513 & 63.13 & 100.0 & 0 & 0 & 0 & 6 & 0 & 0 & 0 & 0 & 0 + 1 part & 3032 & 64.541 & 1.000 & 0.00 & 0.00 & 53.51 & 5606 & 5606 & 5606 & 5606 & - & - & 3638.3 & 3605.7 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7161 & 7161 & 7161 & 0 & 0 & 0 & 1 & 7161 & 7161 & 8716 & 41.05 & 40.72 & 7161 & 7161 & 7161 & - & 7161.0 & 5883.4 & 1 & 1 & 1 & - & 17132 & 9500 & 9500 & 59.46 & 108.32 & 47.14 & 103.36 & 350 & 513 & 71.34 & 100.0 & 0 & 0 & 0 & 0 & 0 & 3 & 0 & 0 & 0 + 0 part & 0 & 60.292 & 1.000 & 26658.29 & 38.19 & 19.09 & 5237 & 5237 & 5237 & 5237 & - & - & 3829.9 & 3146.6 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8716 & 41.29 & 40.72 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8533.0 & 1 & 1 & 1 & 1 & 18043 & 9500 & 9500 & 20.66 & 108.32 & 9.27 & 103.36 & 102 & 513 & 54.1 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7685 & 10.810 & 1.000 & 0.00 & 1916.93 & 425.99 & 733 & 939 & - & - & - & - & 67.2 & 66.5 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
