\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8611 & 8611 & 8611 & 0 & 0 & 0 & 1 & 8611 & 8611 & 8635 & 40.37 & 40.37 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8587.1 & 1 & 1 & 1 & 1 & 90290 & 175000 & 175000 & 385.57 & 383.84 & 383.49 & 381.44 & 10088 & 10067 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 99.745 & 1.000 & 0.00 & 0.00 & 58.21 & 8589 & 8589 & 8589 & 8589 & 8589 & 8589 & 8567.1 & 8543.2 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8611 & 8611 & 8611 & 0 & 0 & 0 & 1 & 8611 & 8611 & 8635 & 40.46 & 40.37 & 8611 & 8611 & 8611 & 8611 & 8611.0 & 8587.1 & 1 & 1 & 1 & 1 & 90593 & 175000 & 175000 & 383.72 & 383.84 & 381.1 & 381.44 & 10074 & 10067 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.932 & 1.000 & 0.00 & 0.00 & 58.69 & 8519 & 8519 & 8519 & 8519 & 8519 & 8519 & 8428.0 & 8404.6 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8460 & 8460 & 8460 & 0 & 0 & 0 & 1 & 8460 & 8460 & 8635 & 40.39 & 40.37 & 8460 & 8460 & 8460 & 8460 & 8460.0 & 8288.5 & 1 & 1 & 1 & 1 & 91977 & 175000 & 175000 & 375.17 & 383.84 & 369.8 & 381.44 & 10131 & 10067 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.316 & 0.999 & 1513.00 & 1843.97 & 70.92 & 8460 & 8460 & 8460 & 8460 & 8460 & 8460 & 8460.0 & 8288.5 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8644 & 8644 & 8644 & 0 & 0 & 0 & 1 & 8644 & 8644 & 8635 & 40.93 & 40.37 & 8644 & 8644 & 8644 & 8644 & 8644.0 & 8653.0 & 1 & 1 & 1 & 1 & 103508 & 175000 & 175000 & 312.65 & 383.84 & 290.86 & 381.44 & 9250 & 10067 & 99.95 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 98.850 & 1.004 & 0.00 & 1977.76 & 608.54 & 8545 & 8545 & 8545 & 8545 & 8545 & 8545 & 8447.1 & 8455.9 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
