\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
Total length & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
Reference length & 8623 & 8623 & 8623 & 8623 \\ \hline
GC (\%) & - & 40.21 & 40.09 & 40.13 \\ \hline
Reference GC (\%) & 40.20 & 40.20 & 40.20 & 40.20 \\ \hline
N50 & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
NG50 & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
N90 & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
NG90 & {\bf 8624} & 8599 & 8593 & 8599 \\ \hline
auN & {\bf 8624.0} & 8599.0 & 8593.0 & 8599.0 \\ \hline
auNG & {\bf 8625.0} & 8575.1 & 8563.1 & 8575.1 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 15721 & 15392 & 15723 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & 0.0 & {\bf 111.39} & 102.0 & 111.36 \\ \hline
Reference mapped (\%) & 111.32 & 111.32 & 111.32 & 111.32 \\ \hline
Properly paired (\%) & 0.0 & {\bf 110.83} & 89.66 & 110.78 \\ \hline
Reference properly paired (\%) & 110.76 & 110.76 & 110.76 & 110.76 \\ \hline
Avg. coverage depth & 0 & {\bf 507} & 450 & {\bf 507} \\ \hline
Reference avg. coverage depth & 507 & 507 & 507 & 507 \\ \hline
Coverage $\geq$ 1x (\%) & 0.0 & 100.0 & 93.87 & 100.0 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & - & 0 & 0 & 0 \\ \hline
\# misassembled contigs & - & 0 & 0 & 0 \\ \hline
Misassembled contigs length & - & 0 & 0 & 0 \\ \hline
\# local misassemblies & - & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & - & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & - & 0 & 2 & 0 \\ \hline
\# structural variations & - & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & - & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 1 + 0 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & 8624 & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & - & 100.000 & 95.023 & 99.907 \\ \hline
Duplication ratio & - & 1.000 & 0.999 & 1.000 \\ \hline
\# N's per 100 kbp & 100000.00 & {\bf 0.00} & 7017.34 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & - & 34.89 & 232.70 & 186.24 \\ \hline
\# indels per 100 kbp & - & 93.03 & 110.23 & 93.12 \\ \hline
Largest alignment & - & 8599 & 8165 & 8591 \\ \hline
Total aligned length & - & 8599 & 8165 & 8591 \\ \hline
NA50 & - & 8599 & 8165 & 8591 \\ \hline
NGA50 & - & 8599 & 8165 & 8591 \\ \hline
NA90 & - & 8599 & 8165 & 8591 \\ \hline
NGA90 & - & 8599 & 8165 & 8591 \\ \hline
auNA & - & 8599.0 & 7758.3 & 8583.0 \\ \hline
auNGA & - & 8575.1 & 7731.3 & 8559.1 \\ \hline
LA50 & - & 1 & 1 & 1 \\ \hline
LGA50 & - & 1 & 1 & 1 \\ \hline
LA90 & - & 1 & 1 & 1 \\ \hline
LGA90 & - & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
