\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
Total length & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
Reference length & 8653 & 8653 & 8653 & 8653 \\ \hline
GC (\%) & 40.65 & 40.81 & 40.21 & 41.08 \\ \hline
Reference GC (\%) & 40.59 & 40.59 & 40.59 & 40.59 \\ \hline
N50 & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
NG50 & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
N75 & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
NG75 & {\bf 8623} & 8621 & 8526 & 8618 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 17600 & 17600 & 17600 & 17600 \\ \hline
\# left & 8800 & 8800 & 8800 & 8800 \\ \hline
\# right & 8800 & 8800 & 8800 & 8800 \\ \hline
Mapped (\%) & {\bf 99.58} & 99.28 & 89.27 & 88.63 \\ \hline
Reference mapped (\%) & 99.59 & 99.59 & 99.59 & 99.59 \\ \hline
Properly paired (\%) & {\bf 99.16} & 98.56 & 79.65 & 78.88 \\ \hline
Reference properly paired (\%) & 99.17 & 99.17 & 99.17 & 99.17 \\ \hline
Avg. coverage depth & {\bf 506} & 505 & 456 & 446 \\ \hline
Reference avg. coverage depth & 506 & 506 & 506 & 506 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.92 & 93.65 & 98.6 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & {\bf 0} & {\bf 0} & 4 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 2 & {\bf 0} \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.643 & 92.575 & 84.594 \\ \hline
Duplication ratio & {\bf 1.000} & 1.014 & 1.068 & 1.182 \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 6697.16 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 34.80} & 364.58 & 3120.30 & 2591.88 \\ \hline
\# indels per 100 kbp & 104.41 & 94.08 & {\bf 62.66} & 96.00 \\ \hline
Largest alignment & {\bf 8623} & 8506 & 7985 & 7298 \\ \hline
Total aligned length & {\bf 8623} & 8506 & 7985 & 7298 \\ \hline
NA50 & {\bf 8623} & 8506 & 7985 & 7298 \\ \hline
NGA50 & {\bf 8623} & 8506 & 7985 & 7298 \\ \hline
NA75 & {\bf 8623} & 8506 & 7985 & 7298 \\ \hline
NGA75 & {\bf 8623} & 8506 & 7985 & 7298 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA75 & 1 & 1 & 1 & 1 \\ \hline
LGA75 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
