\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8655 & 8655 & 8655 & 0 & 0 & 0 & 1 & 8655 & 8655 & 8739 & 40.58 & 40.58 & 8655 & 8655 & 8655 & 8655 & 8655.0 & 8571.8 & 1 & 1 & 1 & 1 & 113264 & 190000 & 190000 & 310.29 & 323.47 & 290.41 & 309.65 & 10090 & 10238 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 0.992 & 0.00 & 0.00 & 34.66 & 8655 & 8655 & 8655 & 8655 & 8655 & 8655 & 8655.0 & 8571.8 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8724 & 8724 & 8724 & 0 & 0 & 0 & 1 & 8724 & 8724 & 8739 & 40.54 & 40.58 & 8724 & 8724 & 8724 & 8724 & 8724.0 & 8709.0 & 1 & 1 & 1 & 1 & 110650 & 190000 & 190000 & 324.14 & 323.47 & 310.31 & 309.65 & 10241 & 10238 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 80.24 & 45.85 & 8724 & 8724 & 8724 & 8724 & 8724 & 8724 & 8724.0 & 8709.0 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8736 & 8736 & 8736 & 0 & 0 & 0 & 1 & 8736 & 8736 & 8739 & 40.58 & 40.58 & 8736 & 8736 & 8736 & 8736 & 8736.0 & 8733.0 & 1 & 1 & 1 & 1 & 115670 & 190000 & 190000 & 301.08 & 323.47 & 279.15 & 309.65 & 9862 & 10238 & 98.52 & 100.0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 + 0 part & 0 & 98.521 & 1.001 & 3434.07 & 1080.89 & 23.25 & 8604 & 8604 & 8604 & 8604 & 8604 & 8604 & 8474.0 & 8471.1 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8736 & 8736 & 8736 & 0 & 0 & 0 & 1 & 8736 & 8736 & 8739 & 40.57 & 40.58 & 8736 & 8736 & 8736 & 8736 & 8736.0 & 8733.0 & 1 & 1 & 1 & 1 & 110837 & 190000 & 190000 & 323.58 & 323.47 & 309.76 & 309.65 & 10238 & 10238 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.002 & 0.00 & 11.45 & 11.45 & 8736 & 8736 & 8736 & 8736 & 8736 & 8736 & 8736.0 & 8733.0 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
