\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 0 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & {\bf 8720} & 8641 & 3002 & 8625 \\ \hline
Total length ($\geq$ 1000 bp) & {\bf 8720} & 8641 & 3002 & 8625 \\ \hline
Total length ($\geq$ 5000 bp) & {\bf 8720} & 8641 & 0 & 8625 \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & {\bf 8720} & 8641 & 3002 & 8625 \\ \hline
Total length & {\bf 8720} & 8641 & 3002 & 8625 \\ \hline
Reference length & 8732 & 8732 & 8732 & 8732 \\ \hline
GC (\%) & 40.75 & 41.29 & 43.75 & 41.34 \\ \hline
Reference GC (\%) & 40.75 & 40.75 & 40.75 & 40.75 \\ \hline
N50 & {\bf 8720} & 8641 & 3002 & 8625 \\ \hline
NG50 & {\bf 8720} & 8641 & - & 8625 \\ \hline
N75 & {\bf 8720} & 8641 & 3002 & 8625 \\ \hline
NG75 & {\bf 8720} & 8641 & - & 8625 \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
L75 & 1 & 1 & 1 & 1 \\ \hline
LG75 & {\bf 1} & {\bf 1} & - & {\bf 1} \\ \hline
\# total reads & 380000 & 380000 & 380000 & 380000 \\ \hline
\# left & 190000 & 190000 & 190000 & 190000 \\ \hline
\# right & 190000 & 190000 & 190000 & 190000 \\ \hline
Mapped (\%) & {\bf 94.48} & 54.04 & 14.64 & 22.44 \\ \hline
Reference mapped (\%) & 94.48 & 94.48 & 94.48 & 94.48 \\ \hline
Properly paired (\%) & {\bf 90.49} & 39.49 & 6.35 & 9.78 \\ \hline
Reference properly paired (\%) & 90.49 & 90.49 & 90.49 & 90.49 \\ \hline
Avg. coverage depth & {\bf 10263} & 5760 & 4480 & 2357 \\ \hline
Reference avg. coverage depth & 10259 & 10259 & 10259 & 10259 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 65.79 & 50.87 & 59.97 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & {\bf 0} & 7 & {\bf 0} & {\bf 0} \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & {\bf 0} & 1 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & 0 + 1 part & {\bf 0 + 0 part} & 0 + 1 part \\ \hline
Unaligned length & {\bf 0} & 2818 & {\bf 0} & 7270 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 66.774 & 17.502 & 15.541 \\ \hline
Duplication ratio & {\bf 1.000} & {\bf 1.000} & 1.967 & {\bf 1.000} \\ \hline
\# N's per 100 kbp & {\bf 0.00} & {\bf 0.00} & 49133.91 & {\bf 0.00} \\ \hline
\# mismatches per 100 kbp & {\bf 22.94} & 137.41 & 131.06 & 1476.01 \\ \hline
\# indels per 100 kbp & 34.41 & {\bf 17.18} & 65.53 & 295.20 \\ \hline
Largest alignment & {\bf 8720} & 5823 & 1527 & 862 \\ \hline
Total aligned length & {\bf 8720} & 5823 & 1527 & 1355 \\ \hline
NA50 & {\bf 8720} & 5823 & 1527 & - \\ \hline
NGA50 & {\bf 8720} & 5823 & - & - \\ \hline
NA75 & {\bf 8720} & - & - & - \\ \hline
NGA75 & {\bf 8720} & - & - & - \\ \hline
LA50 & {\bf 1} & {\bf 1} & {\bf 1} & - \\ \hline
LGA50 & {\bf 1} & {\bf 1} & - & - \\ \hline
LA75 & {\bf 1} & - & - & - \\ \hline
LGA75 & {\bf 1} & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
