\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8685 & 8685 & 8685 & 0 & 0 & 0 & 1 & 8685 & 8685 & 8692 & 40.71 & 40.72 & 8685 & 8685 & 8685 & 8685 & 8685.0 & 8678.0 & 1 & 1 & 1 & 1 & 104136 & 190000 & 190000 & 343.87 & 343.64 & 328.55 & 328.35 & 10277 & 10275 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 0.00 & 34.54 & 8685 & 8685 & 8685 & 8685 & 8685 & 8685 & 8685.0 & 8678.0 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8638 & 8638 & 8638 & 0 & 0 & 0 & 1 & 8638 & 8638 & 8692 & 41.32 & 40.72 & 8638 & 8638 & 8638 & 8638 & 8638.0 & 8584.3 & 1 & 1 & 1 & 1 & 216290 & 190000 & 190000 & 93.18 & 343.64 & 67.67 & 328.35 & 5680 & 10275 & 66.11 & 100.0 & 0 & 0 & 0 & 7 & 0 & 0 & 0 & 0 & 0 + 1 part & 2836 & 66.797 & 1.000 & 0.00 & 0.00 & 17.24 & 5802 & 5802 & 5802 & 5802 & - & - & 3897.1 & 3872.9 & 1 & 1 & - & - \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 7163 & 7163 & 7163 & 0 & 0 & 0 & 1 & 7163 & 7163 & 8692 & 41.02 & 40.72 & 7163 & 7163 & 7163 & - & 7163.0 & 5903.0 & 1 & 1 & 1 & - & 213715 & 190000 & 190000 & 97.11 & 343.64 & 74.85 & 328.35 & 7101 & 10275 & 74.61 & 100.0 & 0 & 0 & 0 & 0 & 0 & 4 & 0 & 0 & 0 + 0 part & 0 & 63.251 & 0.999 & 23398.02 & 163.99 & 36.44 & 5488 & 5488 & 5488 & 5488 & - & - & 4204.7 & 3465.0 & 1 & 1 & - & - \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8624 & 8624 & 8624 & 0 & 0 & 0 & 1 & 8624 & 8624 & 8692 & 41.29 & 40.72 & 8624 & 8624 & 8624 & 8624 & 8624.0 & 8556.5 & 1 & 1 & 1 & 1 & 309205 & 190000 & 190000 & 26.91 & 343.64 & 12.0 & 328.35 & 2287 & 10275 & 57.61 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 + 1 part & 7379 & 14.333 & 1.000 & 0.00 & 1445.78 & 401.61 & 822 & 1245 & - & - & - & - & 88.7 & 88.0 & - & - & - & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
