\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{4}{|r}|}
\hline
Assembly & shiver\_genome & smaltalign\_genome & viralngs\_genome & vpipe\_genome \\ \hline
\# contigs ($\geq$ 0 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 1000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 5000 bp) & 1 & 1 & 1 & 1 \\ \hline
\# contigs ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 0 bp) & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
Total length ($\geq$ 1000 bp) & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
Total length ($\geq$ 5000 bp) & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
Total length ($\geq$ 10000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 25000 bp) & 0 & 0 & 0 & 0 \\ \hline
Total length ($\geq$ 50000 bp) & 0 & 0 & 0 & 0 \\ \hline
\# contigs & 1 & 1 & 1 & 1 \\ \hline
Largest contig & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
Total length & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
Reference length & 8650 & 8650 & 8650 & 8650 \\ \hline
GC (\%) & 40.08 & 40.18 & 39.93 & 40.58 \\ \hline
Reference GC (\%) & 40.06 & 40.06 & 40.06 & 40.06 \\ \hline
N50 & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
NG50 & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
N90 & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
NG90 & 8616 & 8614 & 8585 & {\bf 8650} \\ \hline
auN & 8616.0 & 8614.0 & 8585.0 & {\bf 8650.0} \\ \hline
auNG & 8582.1 & 8578.1 & 8520.5 & {\bf 8650.0} \\ \hline
L50 & 1 & 1 & 1 & 1 \\ \hline
LG50 & 1 & 1 & 1 & 1 \\ \hline
L90 & 1 & 1 & 1 & 1 \\ \hline
LG90 & 1 & 1 & 1 & 1 \\ \hline
\# total reads & 89857 & 90103 & 89828 & 106872 \\ \hline
\# left & 175000 & 175000 & 175000 & 175000 \\ \hline
\# right & 175000 & 175000 & 175000 & 175000 \\ \hline
Mapped (\%) & 387.67 & 385.85 & {\bf 387.8} & 300.03 \\ \hline
Reference mapped (\%) & 386.71 & 386.71 & 386.71 & 386.71 \\ \hline
Properly paired (\%) & 385.84 & 383.25 & {\bf 385.96} & 277.74 \\ \hline
Reference properly paired (\%) & 384.88 & 384.88 & 384.88 & 384.88 \\ \hline
Avg. coverage depth & 10085 & 10068 & {\bf 10098} & 9162 \\ \hline
Reference avg. coverage depth & 10069 & 10069 & 10069 & 10069 \\ \hline
Coverage $\geq$ 1x (\%) & 100.0 & 99.9 & 100.0 & 99.9 \\ \hline
Reference coverage $\geq$ 1x (\%) & 100.0 & 100.0 & 100.0 & 100.0 \\ \hline
\# misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# misassembled contigs & 0 & 0 & 0 & 0 \\ \hline
Misassembled contigs length & 0 & 0 & 0 & 0 \\ \hline
\# local misassemblies & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 \\ \hline
\# structural variations & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & 0 + 0 part & 0 + 0 part & 0 + 0 part & 0 + 0 part \\ \hline
Unaligned length & 0 & 0 & 0 & 0 \\ \hline
Genome fraction (\%) & {\bf 100.000} & 98.932 & 99.803 & 98.015 \\ \hline
Duplication ratio & 1.000 & 1.000 & {\bf 0.999} & 1.004 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & {\bf 11.61} & 46.94 & 419.34 & 1851.63 \\ \hline
\# indels per 100 kbp & 81.24 & {\bf 70.41} & 81.54 & 625.07 \\ \hline
Largest alignment & {\bf 8616} & 8522 & 8585 & 8479 \\ \hline
Total aligned length & {\bf 8616} & 8522 & 8585 & 8479 \\ \hline
NA50 & {\bf 8616} & 8522 & 8585 & 8479 \\ \hline
NGA50 & {\bf 8616} & 8522 & 8585 & 8479 \\ \hline
NA90 & {\bf 8616} & 8522 & 8585 & 8479 \\ \hline
NGA90 & {\bf 8616} & 8522 & 8585 & 8479 \\ \hline
auNA & {\bf 8616.0} & 8431.0 & 8585.0 & 8311.4 \\ \hline
auNGA & {\bf 8582.1} & 8395.9 & 8520.5 & 8311.4 \\ \hline
LA50 & 1 & 1 & 1 & 1 \\ \hline
LGA50 & 1 & 1 & 1 & 1 \\ \hline
LA90 & 1 & 1 & 1 & 1 \\ \hline
LGA90 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
