\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 100 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{66}{|r}|}
\hline
Assembly & \# contigs ($\geq$ 0 bp) & \# contigs ($\geq$ 1000 bp) & \# contigs ($\geq$ 5000 bp) & \# contigs ($\geq$ 10000 bp) & \# contigs ($\geq$ 25000 bp) & \# contigs ($\geq$ 50000 bp) & Total length ($\geq$ 0 bp) & Total length ($\geq$ 1000 bp) & Total length ($\geq$ 5000 bp) & Total length ($\geq$ 10000 bp) & Total length ($\geq$ 25000 bp) & Total length ($\geq$ 50000 bp) & \# contigs & Largest contig & Total length & Reference length & GC (\%) & Reference GC (\%) & N50 & NG50 & N90 & NG90 & auN & auNG & L50 & LG50 & L90 & LG90 & \# total reads & \# left & \# right & Mapped (\%) & Reference mapped (\%) & Properly paired (\%) & Reference properly paired (\%) & Avg. coverage depth & Reference avg. coverage depth & Coverage $\geq$ 1x (\%) & Reference coverage $\geq$ 1x (\%) & \# misassemblies & \# misassembled contigs & Misassembled contigs length & \# local misassemblies & \# scaffold gap ext. mis. & \# scaffold gap loc. mis. & \# structural variations & \# unaligned mis. contigs & \# unaligned contigs & Unaligned length & Genome fraction (\%) & Duplication ratio & \# N's per 100 kbp & \# mismatches per 100 kbp & \# indels per 100 kbp & Largest alignment & Total aligned length & NA50 & NGA50 & NA90 & NGA90 & auNA & auNGA & LA50 & LGA50 & LA90 & LGA90 \\ \hline
shiver\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8734 & 40.58 & 40.58 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8638.3 & 1 & 1 & 1 & 1 & 107087 & 190000 & 190000 & 334.53 & 333.59 & 319.93 & 319.04 & 10281 & 10253 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 23.03 & 103.62 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8638.3 & 1 & 1 & 1 & 1 \\ \hline
smaltalign\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8686 & 8686 & 8686 & 0 & 0 & 0 & 1 & 8686 & 8686 & 8734 & 40.59 & 40.58 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8638.3 & 1 & 1 & 1 & 1 & 107089 & 190000 & 190000 & 334.52 & 333.59 & 319.93 & 319.04 & 10281 & 10253 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 46.05 & 103.62 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686 & 8686.0 & 8638.3 & 1 & 1 & 1 & 1 \\ \hline
viralngs\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8683 & 8683 & 8683 & 0 & 0 & 0 & 1 & 8683 & 8683 & 8734 & 40.34 & 40.58 & 8683 & 8683 & 8683 & 8683 & 8683.0 & 8632.3 & 1 & 1 & 1 & 1 & 107094 & 190000 & 190000 & 334.48 & 333.59 & 319.9 & 319.04 & 10259 & 10253 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 955.89 & 1001.96 & 103.65 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683 & 8683.0 & 8632.3 & 1 & 1 & 1 & 1 \\ \hline
vpipe\_genome & 1 & 1 & 1 & 0 & 0 & 0 & 8689 & 8689 & 8689 & 0 & 0 & 0 & 1 & 8689 & 8689 & 8734 & 40.60 & 40.58 & 8689 & 8689 & 8689 & 8689 & 8689.0 & 8644.2 & 1 & 1 & 1 & 1 & 107111 & 190000 & 190000 & 334.45 & 333.59 & 319.86 & 319.04 & 10280 & 10253 & 100.0 & 100.0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 + 0 part & 0 & 100.000 & 1.000 & 0.00 & 11.51 & 92.07 & 8689 & 8689 & 8689 & 8689 & 8689 & 8689 & 8689.0 & 8644.2 & 1 & 1 & 1 & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
